<!DOCTYPE html>
<html xmlns="http://www.w3.org/1999/xhtml" lang="en-US" xml:lang="en-US">
<head>
  <meta charset="utf-8" />
  <meta name="generator" content="pandoc" />
  <meta name="viewport" content="width=device-width, initial-scale=1.0, user-scalable=yes" />
  <meta name="author" content="David N. Nicholson" />
  <meta name="author" content="Vincent Rubinetti" />
  <meta name="author" content="Dongbo Hu" />
  <meta name="author" content="Marvin Thielk" />
  <meta name="author" content="Lawrence E. Hunter" />
  <meta name="author" content="Casey S. Greene" />
  <meta name="dcterms.date" content="2021-11-03" />
  <meta name="keywords" content="biorxiv, preprints, pubmed-central, natural-language-processing, descriptive-linguistics" />
  <title>How things change: examining linguistic shifts between preprints and publications</title>
  <style>
    code{white-space: pre-wrap;}
    span.smallcaps{font-variant: small-caps;}
    span.underline{text-decoration: underline;}
    div.column{display: inline-block; vertical-align: top; width: 50%;}
    div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
    ul.task-list{list-style: none;}
    pre > code.sourceCode { white-space: pre; position: relative; }
    pre > code.sourceCode > span { display: inline-block; line-height: 1.25; }
    pre > code.sourceCode > span:empty { height: 1.2em; }
    .sourceCode { overflow: visible; }
    code.sourceCode > span { color: inherit; text-decoration: inherit; }
    div.sourceCode { margin: 1em 0; }
    pre.sourceCode { margin: 0; }
    @media screen {
    div.sourceCode { overflow: auto; }
    }
    @media print {
    pre > code.sourceCode { white-space: pre-wrap; }
    pre > code.sourceCode > span { text-indent: -5em; padding-left: 5em; }
    }
    pre.numberSource code
      { counter-reset: source-line 0; }
    pre.numberSource code > span
      { position: relative; left: -4em; counter-increment: source-line; }
    pre.numberSource code > span > a:first-child::before
      { content: counter(source-line);
        position: relative; left: -1em; text-align: right; vertical-align: baseline;
        border: none; display: inline-block;
        -webkit-touch-callout: none; -webkit-user-select: none;
        -khtml-user-select: none; -moz-user-select: none;
        -ms-user-select: none; user-select: none;
        padding: 0 4px; width: 4em;
        color: #aaaaaa;
      }
    pre.numberSource { margin-left: 3em; border-left: 1px solid #aaaaaa;  padding-left: 4px; }
    div.sourceCode
      {   }
    @media screen {
    pre > code.sourceCode > span > a:first-child::before { text-decoration: underline; }
    }
    code span.al { color: #ff0000; font-weight: bold; } /* Alert */
    code span.an { color: #60a0b0; font-weight: bold; font-style: italic; } /* Annotation */
    code span.at { color: #7d9029; } /* Attribute */
    code span.bn { color: #40a070; } /* BaseN */
    code span.bu { } /* BuiltIn */
    code span.cf { color: #007020; font-weight: bold; } /* ControlFlow */
    code span.ch { color: #4070a0; } /* Char */
    code span.cn { color: #880000; } /* Constant */
    code span.co { color: #60a0b0; font-style: italic; } /* Comment */
    code span.cv { color: #60a0b0; font-weight: bold; font-style: italic; } /* CommentVar */
    code span.do { color: #ba2121; font-style: italic; } /* Documentation */
    code span.dt { color: #902000; } /* DataType */
    code span.dv { color: #40a070; } /* DecVal */
    code span.er { color: #ff0000; font-weight: bold; } /* Error */
    code span.ex { } /* Extension */
    code span.fl { color: #40a070; } /* Float */
    code span.fu { color: #06287e; } /* Function */
    code span.im { } /* Import */
    code span.in { color: #60a0b0; font-weight: bold; font-style: italic; } /* Information */
    code span.kw { color: #007020; font-weight: bold; } /* Keyword */
    code span.op { color: #666666; } /* Operator */
    code span.ot { color: #007020; } /* Other */
    code span.pp { color: #bc7a00; } /* Preprocessor */
    code span.sc { color: #4070a0; } /* SpecialChar */
    code span.ss { color: #bb6688; } /* SpecialString */
    code span.st { color: #4070a0; } /* String */
    code span.va { color: #19177c; } /* Variable */
    code span.vs { color: #4070a0; } /* VerbatimString */
    code span.wa { color: #60a0b0; font-weight: bold; font-style: italic; } /* Warning */
    div.csl-bib-body { }
    div.csl-entry {
      clear: both;
    }
    .hanging div.csl-entry {
      margin-left:2em;
      text-indent:-2em;
    }
    div.csl-left-margin {
      min-width:2em;
      float:left;
    }
    div.csl-right-inline {
      margin-left:2em;
      padding-left:1em;
    }
    div.csl-indent {
      margin-left: 2em;
    }
  </style>
  <!--[if lt IE 9]>
    <script src="//cdnjs.cloudflare.com/ajax/libs/html5shiv/3.7.3/html5shiv-printshiv.min.js"></script>
  <![endif]-->
  <!--
  Manubot generated metadata rendered from header-includes-template.html.
  Suggest improvements at https://github.com/manubot/manubot/blob/main/manubot/process/header-includes-template.html
  -->
  <meta name="dc.format" content="text/html" />
  <meta name="dc.title" content="How things change: examining linguistic shifts between preprints and publications" />
  <meta name="citation_title" content="How things change: examining linguistic shifts between preprints and publications" />
  <meta property="og:title" content="How things change: examining linguistic shifts between preprints and publications" />
  <meta property="twitter:title" content="How things change: examining linguistic shifts between preprints and publications" />
  <meta name="dc.date" content="2021-11-03" />
  <meta name="citation_publication_date" content="2021-11-03" />
  <meta name="dc.language" content="en-US" />
  <meta name="citation_language" content="en-US" />
  <meta name="dc.relation.ispartof" content="Manubot" />
  <meta name="dc.publisher" content="Manubot" />
  <meta name="citation_journal_title" content="Manubot" />
  <meta name="citation_technical_report_institution" content="Manubot" />
  <meta name="citation_author" content="David N. Nicholson" />
  <meta name="citation_author_institution" content="Department of Systems Pharmacology and Translational Therapeutics, Perelman School of Medicine University of Pennsylvania, Philadelphia PA, USA" />
  <meta name="citation_author_orcid" content="0000-0003-0002-5761" />
  <meta name="twitter:creator" content="@dnicholson329" />
  <meta name="citation_author" content="Vincent Rubinetti" />
  <meta name="citation_author_institution" content="Department of Systems Pharmacology and Translational Therapeutics, Perelman School of Medicine University of Pennsylvania, Philadelphia PA, USA" />
  <meta name="citation_author_institution" content="Center for Health AI, University of Colorado School of Medicine, Aurora, CO, USA" />
  <meta name="citation_author_orcid" content="None" />
  <meta name="twitter:creator" content="@vincerubinetti" />
  <meta name="citation_author" content="Dongbo Hu" />
  <meta name="citation_author_institution" content="Department of Systems Pharmacology and Translational Therapeutics, Perelman School of Medicine University of Pennsylvania, Philadelphia PA, USA" />
  <meta name="citation_author_orcid" content="None" />
  <meta name="twitter:creator" content="@None" />
  <meta name="citation_author" content="Marvin Thielk" />
  <meta name="citation_author_institution" content="Elsevier, Philadelphia PA, USA" />
  <meta name="citation_author_orcid" content="0000-0002-0751-3664" />
  <meta name="twitter:creator" content="@TheNeuralCoder" />
  <meta name="citation_author" content="Lawrence E. Hunter" />
  <meta name="citation_author_institution" content="Center for Computational Pharmacology, University of Colorado School of Medicine, Aurora CO, USA" />
  <meta name="citation_author_orcid" content="0000-0003-1455-3370" />
  <meta name="twitter:creator" content="@ProfLHunter" />
  <meta name="citation_author" content="Casey S. Greene" />
  <meta name="citation_author_institution" content="Department of Systems Pharmacology and Translational Therapeutics, Perelman School of Medicine University of Pennsylvania, Philadelphia PA, USA" />
  <meta name="citation_author_institution" content="Department of Biochemistry and Molecular Genetics, University of Colorado School of Medicine, Aurora CO, USA" />
  <meta name="citation_author_institution" content="Center for Health AI, University of Colorado School of Medicine, Aurora, CO, USA" />
  <meta name="citation_author_orcid" content="0000-0001-8713-9213" />
  <meta name="twitter:creator" content="@greenescientist" />
  <link rel="canonical" href="https://greenelab.github.io/annorxiver_manuscript/" />
  <meta property="og:url" content="https://greenelab.github.io/annorxiver_manuscript/" />
  <meta property="twitter:url" content="https://greenelab.github.io/annorxiver_manuscript/" />
  <meta name="citation_fulltext_html_url" content="https://greenelab.github.io/annorxiver_manuscript/" />
  <meta name="citation_pdf_url" content="https://greenelab.github.io/annorxiver_manuscript/manuscript.pdf" />
  <link rel="alternate" type="application/pdf" href="https://greenelab.github.io/annorxiver_manuscript/manuscript.pdf" />
  <link rel="alternate" type="text/html" href="https://greenelab.github.io/annorxiver_manuscript/v/70cd00ac6309d56567b58053b04468e071519f87/" />
  <meta name="manubot_html_url_versioned" content="https://greenelab.github.io/annorxiver_manuscript/v/70cd00ac6309d56567b58053b04468e071519f87/" />
  <meta name="manubot_pdf_url_versioned" content="https://greenelab.github.io/annorxiver_manuscript/v/70cd00ac6309d56567b58053b04468e071519f87/manuscript.pdf" />
  <meta property="og:type" content="article" />
  <meta property="twitter:card" content="summary_large_image" />
  <link rel="icon" type="image/png" sizes="192x192" href="https://manubot.org/favicon-192x192.png" />
  <link rel="mask-icon" href="https://manubot.org/safari-pinned-tab.svg" color="#ad1457" />
  <meta name="theme-color" content="#ad1457" />
  <!-- end Manubot generated metadata -->
</head>
<body>
<header id="title-block-header">
<h1 class="title">How things change: examining linguistic shifts between preprints and publications</h1>
</header>
<p><small><em>
This manuscript
(<a href="https://greenelab.github.io/annorxiver_manuscript/v/70cd00ac6309d56567b58053b04468e071519f87/">permalink</a>)
was automatically generated
from <a href="https://github.com/greenelab/annorxiver_manuscript/tree/70cd00ac6309d56567b58053b04468e071519f87">greenelab/annorxiver_manuscript@70cd00a</a>
on November 3, 2021.
</em></small></p>
<h2 id="authors">Authors</h2>
<ul>
<li><p><strong>David N. Nicholson</strong><br>
<img src="images/orcid.svg" class="inline_icon" width="16" height="16" alt="ORCID icon" />
<a href="https://orcid.org/0000-0003-0002-5761">0000-0003-0002-5761</a>
· <img src="images/github.svg" class="inline_icon" width="16" height="16" alt="GitHub icon" />
<a href="https://github.com/danich1">danich1</a>
· <img src="images/twitter.svg" class="inline_icon" width="16" height="16" alt="Twitter icon" />
<a href="https://twitter.com/dnicholson329">dnicholson329</a><br>
<small>
Department of Systems Pharmacology and Translational Therapeutics, Perelman School of Medicine University of Pennsylvania, Philadelphia PA, USA
· Funded by The Gordon and Betty Moore Foundation (GBMF4552); The National Institutes of Health (T32 HG000046)
</small></p></li>
<li><p><strong>Vincent Rubinetti</strong><br>
· <img src="images/github.svg" class="inline_icon" width="16" height="16" alt="GitHub icon" />
<a href="https://github.com/vincerubinetti">vincerubinetti</a>
· <img src="images/twitter.svg" class="inline_icon" width="16" height="16" alt="Twitter icon" />
<a href="https://twitter.com/vincerubinetti">vincerubinetti</a><br>
<small>
Department of Systems Pharmacology and Translational Therapeutics, Perelman School of Medicine University of Pennsylvania, Philadelphia PA, USA; Center for Health AI, University of Colorado School of Medicine, Aurora, CO, USA
· Funded by The Gordon and Betty Moore Foundation (GBMF4552); The National Institutes of Health (R01 HG010067)
</small></p></li>
<li><p><strong>Dongbo Hu</strong><br>
· <img src="images/github.svg" class="inline_icon" width="16" height="16" alt="GitHub icon" />
<a href="https://github.com/dongbohu">dongbohu</a><br>
<small>
Department of Systems Pharmacology and Translational Therapeutics, Perelman School of Medicine University of Pennsylvania, Philadelphia PA, USA
· Funded by The Gordon and Betty Moore Foundation (GBMF4552); The National Institutes of Health (R01 HG010067)
</small></p></li>
<li><p><strong>Marvin Thielk</strong><br>
<img src="images/orcid.svg" class="inline_icon" width="16" height="16" alt="ORCID icon" />
<a href="https://orcid.org/0000-0002-0751-3664">0000-0002-0751-3664</a>
· <img src="images/github.svg" class="inline_icon" width="16" height="16" alt="GitHub icon" />
<a href="https://github.com/MarvinT">MarvinT</a>
· <img src="images/twitter.svg" class="inline_icon" width="16" height="16" alt="Twitter icon" />
<a href="https://twitter.com/TheNeuralCoder">TheNeuralCoder</a><br>
<small>
Elsevier, Philadelphia PA, USA
</small></p></li>
<li><p><strong>Lawrence E. Hunter</strong><br>
<img src="images/orcid.svg" class="inline_icon" width="16" height="16" alt="ORCID icon" />
<a href="https://orcid.org/0000-0003-1455-3370">0000-0003-1455-3370</a>
· <img src="images/github.svg" class="inline_icon" width="16" height="16" alt="GitHub icon" />
<a href="https://github.com/LEHunter">LEHunter</a>
· <img src="images/twitter.svg" class="inline_icon" width="16" height="16" alt="Twitter icon" />
<a href="https://twitter.com/ProfLHunter">ProfLHunter</a><br>
<small>
Center for Computational Pharmacology, University of Colorado School of Medicine, Aurora CO, USA
· Funded by The Gordon and Betty Moore Foundation (GBMF4552)
</small></p></li>
<li><p><strong>Casey S. Greene</strong><br>
<img src="images/orcid.svg" class="inline_icon" width="16" height="16" alt="ORCID icon" />
<a href="https://orcid.org/0000-0001-8713-9213">0000-0001-8713-9213</a>
· <img src="images/github.svg" class="inline_icon" width="16" height="16" alt="GitHub icon" />
<a href="https://github.com/cgreene">cgreene</a>
· <img src="images/twitter.svg" class="inline_icon" width="16" height="16" alt="Twitter icon" />
<a href="https://twitter.com/greenescientist">greenescientist</a><br>
<small>
Department of Systems Pharmacology and Translational Therapeutics, Perelman School of Medicine University of Pennsylvania, Philadelphia PA, USA; Department of Biochemistry and Molecular Genetics, University of Colorado School of Medicine, Aurora CO, USA; Center for Health AI, University of Colorado School of Medicine, Aurora, CO, USA
· Funded by The Gordon and Betty Moore Foundation (GBMF4552); The National Institutes of Health (R01 HG010067)
</small></p></li>
</ul>
<h2 class="page_break_before" id="abstract">Abstract</h2>
<p>Preprints allow researchers to make their findings available to the scientific community before they have undergone peer review.
Studies on preprints within bioRxiv have been largely focused on article metadata and how often these preprints are downloaded, cited, published, and discussed online.
A missing element that has yet to be examined is the language contained within the bioRxiv preprint repository.
We sought to compare and contrast linguistic features within bioRxiv preprints to published biomedical text as a whole as this is an excellent opportunity to examine how peer review changes these documents.
The most prevalent features that changed appear to be associated with typesetting and mentions of supplementary sections or additional files.
In addition to text comparison, we created document embeddings derived from a preprint-trained word2vec model.
We found that these embeddings are able to parse out different scientific approaches and concepts, link unannotated preprint-peer reviewed article pairs, and identify journals that publish linguistically similar papers to a given preprint.
We also used these embeddings to examine factors associated with the time elapsed between the posting of a first preprint and the appearance of a peer reviewed publication.
We found that preprints with more versions posted and more textual changes took longer to publish.
Lastly, we constructed a web application (https://greenelab.github.io/preprint-similarity-search/) that allows users to identify which journals and articles that are most linguistically similar to a bioRxiv or medRxiv preprint as well as observe where the preprint would be positioned within a published article landscape.</p>
<h2 id="introduction">Introduction</h2>
<p>The dissemination of research findings is key to science.
Initially, much of this communication happened orally <span class="citation" data-cites="TQgSqYqx">[<a href="#ref-TQgSqYqx" role="doc-biblioref">1</a>]</span>.
During the 17th century, the predominant form of communication shifted to personal letters shared from one scientist to another <span class="citation" data-cites="TQgSqYqx">[<a href="#ref-TQgSqYqx" role="doc-biblioref">1</a>]</span>.
Scientific journals didn’t become a predominant mode of communication until the 19th and 20th centuries when the first journal was created <span class="citation" data-cites="TQgSqYqx u6LNiV2V es2rrG6B">[<a href="#ref-TQgSqYqx" role="doc-biblioref">1</a>,<a href="#ref-u6LNiV2V" role="doc-biblioref">2</a>,<a href="#ref-es2rrG6B" role="doc-biblioref">3</a>]</span>.
Although scientific journals became the primary method of communication, they added high maintenance costs and long publication times to scientific discourse <span class="citation" data-cites="u6LNiV2V es2rrG6B">[<a href="#ref-u6LNiV2V" role="doc-biblioref">2</a>,<a href="#ref-es2rrG6B" role="doc-biblioref">3</a>]</span>.
Some scientists’ solutions to these issues have been to communicate through preprints, which are scholarly works that have yet to undergo peer review process <span class="citation" data-cites="ZdEaAXLe ePmdRbmq">[<a href="#ref-ZdEaAXLe" role="doc-biblioref">4</a>,<a href="#ref-ePmdRbmq" role="doc-biblioref">5</a>]</span>.</p>
<p>Preprints are commonly hosted on online repositories, where users have open and easy access to these works.
Notable repositories include arXiv <span class="citation" data-cites="ZTyQYlDz">[<a href="#ref-ZTyQYlDz" role="doc-biblioref">6</a>]</span>, bioRxiv <span class="citation" data-cites="i9BKPuqb">[<a href="#ref-i9BKPuqb" role="doc-biblioref">7</a>]</span> and medRxiv <span class="citation" data-cites="NKHNs9j3">[<a href="#ref-NKHNs9j3" role="doc-biblioref">8</a>/]</span>; however, there are over 60 different repositories available <span class="citation" data-cites="lMj4YzIP">[<a href="#ref-lMj4YzIP" role="doc-biblioref">9</a>/]</span>.
The burgeoning uptake of preprints in life sciences has been examined through research focused on metadata from the bioRxiv repository.
For example, life science preprints are being posted at an increasing rate <span class="citation" data-cites="1DCEzEMkZ">[<a href="#ref-1DCEzEMkZ" role="doc-biblioref">10</a>]</span>.
Furthermore, these preprints are being rapidly shared on social media, routinely downloaded, and cited <span class="citation" data-cites="PJwR7lcq">[<a href="#ref-PJwR7lcq" role="doc-biblioref">11</a>]</span>.
Some preprint categories are shared on social media by both scientists and non-scientists <span class="citation" data-cites="QM87Cg3q">[<a href="#ref-QM87Cg3q" role="doc-biblioref">12</a>]</span>.
About two-thirds to three-quarters of preprints are eventually published <span class="citation" data-cites="iPnRRJya LcHjf487">[<a href="#ref-iPnRRJya" role="doc-biblioref">13</a>,<a href="#ref-LcHjf487" role="doc-biblioref">14</a>]</span> and life science articles that have a corresponding preprint version are cited and discussed more often than articles without them <span class="citation" data-cites="13dwNbYs7 49GeQtua WaEbLm5M">[<a href="#ref-13dwNbYs7" role="doc-biblioref">15</a>,<a href="#ref-49GeQtua" role="doc-biblioref">16</a>,<a href="#ref-WaEbLm5M" role="doc-biblioref">17</a>]</span>.
Preprints take an average of 160 days to be published in the peer-reviewed literature <span class="citation" data-cites="tTgEr7g1">[<a href="#ref-tTgEr7g1" role="doc-biblioref">18</a>]</span>, and those with multiple versions take longer to publish<span class="citation" data-cites="tTgEr7g1">[<a href="#ref-tTgEr7g1" role="doc-biblioref">18</a>]</span>.</p>
<p>The rapid uptake of preprints in the life sciences also poses challenges.
Preprint repositories receive a growing number of submissions <span class="citation" data-cites="TYlEZ3Dw">[<a href="#ref-TYlEZ3Dw" role="doc-biblioref">19</a>]</span>.
Linking preprints with their published counterparts is vital to maintaining scholarly discourse consistency, but this task is challenging to perform manually <span class="citation" data-cites="PmF48i62 49GeQtua vW9wl0tz">[<a href="#ref-49GeQtua" role="doc-biblioref">16</a>,<a href="#ref-PmF48i62" role="doc-biblioref">20</a>,<a href="#ref-vW9wl0tz" role="doc-biblioref">21</a>]</span>.
Errors and omissions in linkage result in missing links and consequently erroneous metadata.
Furthermore, repositories based on standard publishing tools are not designed to show how the textual content of preprints is altered due to the peer review process <span class="citation" data-cites="TYlEZ3Dw">[<a href="#ref-TYlEZ3Dw" role="doc-biblioref">19</a>]</span>.
Certain scientists have expressed concern that competitors could scoop them by making results available before publication <span class="citation" data-cites="TYlEZ3Dw jXP4z24U">[<a href="#ref-TYlEZ3Dw" role="doc-biblioref">19</a>,<a href="#ref-jXP4z24U" role="doc-biblioref">22</a>]</span>.
Preprint repositories by definition do not perform in-depth peer review, which can result in posted preprints containing inconsistent results or conclusions <span class="citation" data-cites="WaEbLm5M 2aMrUMfz PmF48i62 1EwmaUwf9">[<a href="#ref-WaEbLm5M" role="doc-biblioref">17</a>,<a href="#ref-PmF48i62" role="doc-biblioref">20</a>,<a href="#ref-2aMrUMfz" role="doc-biblioref">23</a>,<a href="#ref-1EwmaUwf9" role="doc-biblioref">24</a>]</span>; however, an analysis of preprints posted at the beginning of 2020 revealed that over 50% underwent minor changes in the abstract text as they were published, but over 70% did not change or only had simple rearrangements to panels and tables <span class="citation" data-cites="DgTUHUNP">[<a href="#ref-DgTUHUNP" role="doc-biblioref">25</a>]</span>.
Despite a growing emphasis on using preprints to examine the publishing process within life sciences, how these findings relate to the text of all documents in bioRxiv has yet to be examined.</p>
<p>Textual analysis uses linguistic, statistical, and machine learning techniques to analyze and extract information from text <span class="citation" data-cites="Ibrx6ppY nmSCIhkg">[<a href="#ref-Ibrx6ppY" role="doc-biblioref">26</a>,<a href="#ref-nmSCIhkg" role="doc-biblioref">27</a>]</span>.
For instance, scientists analyzed linguistic similarities and differences of biomedical corpora [<span class="citation" data-cites="1H7sXN2v7">[<a href="#ref-1H7sXN2v7" role="doc-biblioref">28</a>]</span>;10.1186/1471-2105-9-S3-S6; <span class="citation" data-cites="1Ec63crER">[<a href="#ref-1Ec63crER" role="doc-biblioref">29</a>]</span>].
Scientists have provided the community with a number of tools that aide future text mining systems <span class="citation" data-cites="qGYQ9hj7 18ZyyTcTe XXeZHguV">[<a href="#ref-qGYQ9hj7" role="doc-biblioref">30</a>,<a href="#ref-18ZyyTcTe" role="doc-biblioref">31</a>,<a href="#ref-XXeZHguV" role="doc-biblioref">32</a>]</span> as well as advice on how to train and test future text processing systems <span class="citation" data-cites="vU15xGGa 1CW99irQj YhMeTgyl">[<a href="#ref-vU15xGGa" role="doc-biblioref">33</a>,<a href="#ref-1CW99irQj" role="doc-biblioref">34</a>,<a href="#ref-YhMeTgyl" role="doc-biblioref">35</a>]</span>.
Here, we use textual analysis to examine the bioRxiv repository, placing a particular emphasis on understanding the extent to which full-text research can address hypotheses derived from the study of metadata alone.</p>
<p>To understand how preprints relate to the traditional publishing ecosystem, we examine the linguistic similarities and differences between preprints and peer-reviewed text and observe how linguistic features change during the peer review and publishing process.
We hypothesize that preprints and biomedical text will appear to have similar characteristics, especially when controlling for the differential uptake of preprints across fields.
Furthermore, we hypothesize that document embeddings <span class="citation" data-cites="1GhHIDxuW SdcIDSwR">[<a href="#ref-1GhHIDxuW" role="doc-biblioref">36</a>,<a href="#ref-SdcIDSwR" role="doc-biblioref">37</a>]</span> provide a versatile way to disentangle linguistic features along with serving as a suitable medium for improving preprint repository functionality.
We test this hypothesis by producing a linguistic landscape of bioRxiv preprints, detecting preprints that change substantially during publication, and identify journals that publish manuscripts that are linguistically similar to a target preprint.
We encapsulate our findings through a web app that projects a user-selected preprint onto this landscape and suggests journals and articles that are linguistically similar.
Our work reveals how linguistically similar and dissimilar preprints are to peer-reviewed text, quantifies linguistic changes that occur during the peer review process, and highlights the feasibility of document embeddings concerning preprint repository functionality and peer review’s effect on publication time.</p>
<h2 id="materials-and-methods">Materials and Methods</h2>
<h3 id="corpora-examined">Corpora Examined</h3>
<p>Text analytics is generally comparative in nature, so we selected three relevant text corpora for analysis: the BioRxiv corpus, which is the target of the investigation; the PubMedCentral Open Access corpus, which represents the peer-reviewed biomedical literature; and the New York Times Annotated Corpus, which is used a representative of general English text.</p>
<h4 id="biorxiv-corpus">BioRxiv Corpus</h4>
<p>BioRxiv <span class="citation" data-cites="i9BKPuqb">[<a href="#ref-i9BKPuqb" role="doc-biblioref">7</a>]</span> is a repository for life sciences preprints.
We downloaded an XML snapshot of this repository on February 3rd, 2020, from bioRxiv’s Amazon S3 bucket <span class="citation" data-cites="142YFczMe">[<a href="#ref-142YFczMe" role="doc-biblioref">38</a>]</span>.
This snapshot contained the full text and image content of 98,023 preprints.
Preprints on bioRxiv are versioned, and in our snapshot, 26,905 out of 98,023 contained more than one version.
When preprints had multiple versions, we used the latest one unless otherwise noted.
Authors submitting preprints to bioRxiv can select one of twenty-nine different categories and tag the type of article: a new result, confirmatory finding, or contradictory finding.
A few preprints in this snapshot were later withdrawn from bioRxiv; when withdrawn, their content is replaced with the reason for withdrawal.
We encountered a total of 72 withdrawn preprints within our snapshot.
After removal, we were left with 97,951 preprints for our downstream analyses.</p>
<h4 id="pubmed-central-open-access-corpus">PubMed Central Open Access Corpus</h4>
<p>PubMed Central (PMC) is a digital archive for the United States National Institute of Health’s Library of Medicine (NIH/NLM) that contains full text biomedical and life science articles <span class="citation" data-cites="u294RvPz">[<a href="#ref-u294RvPz" role="doc-biblioref">39</a>]</span>.
Paper availability within PMC is mainly dependent on the journal’s participation level <span class="citation" data-cites="lBh7pyRc">[<a href="#ref-lBh7pyRc" role="doc-biblioref">40</a>/]</span>.
Articles appear in PMC as either accepted author manuscripts (Green Open Access) or via open access publishing at the journal (Gold Open Access <span class="citation" data-cites="iyblQx2M">[<a href="#ref-iyblQx2M" role="doc-biblioref">41</a>]</span>).
Individual journals have the option to fully participate in submitting articles to PMC, selectively participate sending only a few papers to PMC, only submit papers according to NIH’s public access policy <span class="citation" data-cites="vGaAJBqg">[<a href="#ref-vGaAJBqg" role="doc-biblioref">42</a>]</span>, or not participate at all; however, individual articles published with the CC BY license may be incorporated.
As of September 2019, PMC had 5,725,819 articles available <span class="citation" data-cites="rzmpIYmn">[<a href="#ref-rzmpIYmn" role="doc-biblioref">43</a>/]</span>.
Out of these 5 million articles, about 3 million were open access (PMCOA) and available for text processing systems <span class="citation" data-cites="rURwqquJ 18ZyyTcTe">[<a href="#ref-18ZyyTcTe" role="doc-biblioref">31</a>,<a href="#ref-rURwqquJ" role="doc-biblioref">44</a>]</span>.
PMC also contains a resource that holds author manuscripts that have already passed the peer review process <span class="citation" data-cites="12qFW0Wyn">[<a href="#ref-12qFW0Wyn" role="doc-biblioref">45</a>/]</span>.
Since these manuscripts have already been peer-reviewed, we excluded them from our analysis as the scope of our work is focused on examining the beginning and end of a preprint’s life cycle.
We downloaded a snapshot of the PMCOA corpus on January 31st, 2020.
This snapshot contained many types of articles: literature reviews, book reviews, editorials, case reports, research articles, and more.
We used only research articles, which align with the intended role of bioRxiv, and we refer to these articles as the PMCOA corpus.</p>
<h4 id="the-new-york-times-annotated-corpus">The New York Times Annotated Corpus</h4>
<p>The New York Times Annotated Corpus (NYTAC) is <span class="citation" data-cites="18JcY2okY">[<a href="#ref-18JcY2okY" role="doc-biblioref">46</a>]</span> is a collection of newspaper articles from the New York Times dating from January 1st, 1987, to June 19th, 2007.
This collection contains over 1.8 million articles where 1.5 million of those articles have undergone manual entity tagging by library scientists <span class="citation" data-cites="18JcY2okY">[<a href="#ref-18JcY2okY" role="doc-biblioref">46</a>]</span>.
We downloaded this collection on August 3rd, 2020, from the Linguistic Data Consortium (see Software and Data Availability section) and used the entire collection as a negative control for our corpora comparison analysis.</p>
<h3 id="mapping-biorxiv-preprints-to-their-published-counterparts">Mapping bioRxiv preprints to their published counterparts</h3>
<p>We used CrossRef <span class="citation" data-cites="XnHb341s">[<a href="#ref-XnHb341s" role="doc-biblioref">47</a>]</span> to identify bioRxiv preprints linked to a corresponding published article.
We accessed CrossRef on July 7th, 2020, and successfully linked 23,271 preprints to their published counterparts.
Out of those 23,271 preprint-published pairs, only 17,952 pairs had a published version present within the PMCOA corpus.
For our analyses that involved published links, we only focused on this subset of preprints-published pairs.</p>
<h3 id="comparing-corpora">Comparing Corpora</h3>
<p>We compared the bioRxiv, PMCOA, and NYTAC corpora to assess the similarities and differences between them.
We used the NYTAC corpus as a negative control to assess the similarity between two life sciences repositories compared with non-life sciences text.
All corpora contain multiple words that do not have any meaning (e.g. conjunctions, prepositions, etc.) or occur with a high frequency.
These words are termed stopwords and are often removed to improve text processing pipelines.
Along with stopwords, all corpora contain both words and non-word entities (e.g., numbers or symbols like <span class="math inline">\(\pm\)</span>), which we refer to together as tokens to avoid confusion.
We calculated the following characteristic metrics for each corpus: the number of documents, the number of sentences, the total number of tokens, the number of stopwords, the average length of a document, the average length of a sentence, the number of negations, the number of coordinating conjunctions, the number of pronouns and the number of past tense verbs.
SpaCy is a lightweight and easy-to-use python package designed to preprocess and filter text <span class="citation" data-cites="q2fFAZTG">[<a href="#ref-q2fFAZTG" role="doc-biblioref">48</a>]</span>.
We used spaCy’s “en_core_web_sm” model <span class="citation" data-cites="q2fFAZTG">[<a href="#ref-q2fFAZTG" role="doc-biblioref">48</a>]</span> (version 2.2.3) to preprocess all corpora and filter out 326 stopwords using spaCy’s default settings.</p>
<p>Following that cleaning process, we calculated the frequency of every token across all corpora.
Because many tokens were unique to one set or the other and observed at low frequency, we focused on the union of the top 0.05% (~100) most frequently occurring tokens within each corpus.
We generated a contingency table for each token in this union and calculated the odds ratio along with the 95% confidence interval <span class="citation" data-cites="Lgm5q6mO">[<a href="#ref-Lgm5q6mO" role="doc-biblioref">49</a>/]</span>.
We measured corpora similarity by calculating the Kullback–Leibler (KL) divergence across all corpora along with token enrichment analysis.
KL divergence is a metric that measures the extent to which two distributions differ from each other.
A low value of KL divergence implicates that two distributions are similar and vice versa for high values.
The optimal number of tokens used to calculate the KL divergence is unknown, so we calculated this metric using a range of the 100 most frequently occurring tokens between two corpora to the 5000 most frequently occurring tokens.</p>
<h3 id="constructing-a-document-representation-for-life-sciences-text">Constructing a Document Representation for Life Sciences Text</h3>
<p>We sought to build a language model to quantify linguistic similarities of biomedical preprints and articles.
Word2vec is a suite of neural networks designed to model linguistic features of tokens based on their appearance in the text.
These models are trained to either predict a token based on its sentence context, called a continuous bag of words (CBOW) model, or predict the context based on a given token, called a skipgram model <span class="citation" data-cites="1GhHIDxuW">[<a href="#ref-1GhHIDxuW" role="doc-biblioref">36</a>]</span>.
Through these prediction tasks, both networks learn latent linguistic features which are helpful for downstream tasks, such as identifying similar tokens.
We used gensim <span class="citation" data-cites="HUWvSbrt">[<a href="#ref-HUWvSbrt" role="doc-biblioref">50</a>]</span> (version 3.8.1) to train a CBOW <span class="citation" data-cites="1GhHIDxuW">[<a href="#ref-1GhHIDxuW" role="doc-biblioref">36</a>]</span> model over all the main text within each preprint in the bioRxiv corpus.
Determining the best number of dimensions for token embeddings can be a non-trivial task; however, it has been shown that optimal performance is between 100-1000 dimensions <span class="citation" data-cites="Km2ytzVD">[<a href="#ref-Km2ytzVD" role="doc-biblioref">51</a>]</span>.
We chose to train the CBOW model using 300 hidden nodes, a batch size of 10000 tokens, and for 20 epochs.
We set a fixed random seed and used gensim’s default settings for all other hyperparameters.
Once trained, every token present within the CBOW model is associated with a dense vector representing latent features captured by the network.
We used these token vectors to generate a document representation for every article within the bioRxiv and PMCOA corpora.
We used spaCy to lemmatize each token for each document and then took the average of every lemmatized token present within the CBOW model and the individual document <span class="citation" data-cites="SdcIDSwR">[<a href="#ref-SdcIDSwR" role="doc-biblioref">37</a>]</span>.
Any token present within the document but not in the CBOW model is ignored during this calculation process.</p>
<h3 id="visualizing-and-characterizing-preprint-representations">Visualizing and Characterizing Preprint Representations</h3>
<p>We sought to visualize the landscape of preprints and determine the extent to which their representation as document vectors corresponded to author-supplied document labels.
We used principal component analysis (PCA) <span class="citation" data-cites="qkjVrvqt">[<a href="#ref-qkjVrvqt" role="doc-biblioref">52</a>]</span> to project bioRxiv document vectors into a low-dimensional space.
We trained this model using scikit-learn’s <span class="citation" data-cites="vHIKEjNs">[<a href="#ref-vHIKEjNs" role="doc-biblioref">53</a>]</span> implementation of a randomized solver <span class="citation" data-cites="7tMTyalf">[<a href="#ref-7tMTyalf" role="doc-biblioref">54</a>]</span> with a random seed of 100, an output of 50 principal components (PCs), and default settings for all other hyperparameters.
After training the model, every preprint within the bioRxiv corpus receives a score for each generated PC.
We sought to uncover concepts captured within generated PCs and used the cosine similarity metric to examine these concepts.
This metric takes two vectors as input and outputs a score between -1 (most dissimilar) and 1 (most similar).
We used this metric to score the similarity between all generated PCs and every token within our CBOW model for our use case.
We report the top 100 positive and negative scoring tokens as word clouds.
The size of each word corresponds to the magnitude of similarity, and color represents a positive (orange) or negative (blue) association.</p>
<h3 id="discovering-unannotated-preprint-publication-relationships">Discovering Unannotated Preprint-Publication Relationships</h3>
<p>The bioRxiv maintainers have automated procedures to link preprints to peer-reviewed versions, and many journals require authors to update preprints with a link to the published version.
However, this automation is primarily based on the exact matching of specific preprint attributes.
If authors change the title between a preprint and published version (e.g., <span class="citation" data-cites="1Bpwk7s3x">[<a href="#ref-1Bpwk7s3x" role="doc-biblioref">55</a>]</span> and <span class="citation" data-cites="iBFkqblV">[<a href="#ref-iBFkqblV" role="doc-biblioref">56</a>]</span>), then this change will prevent bioRxiv from automatically establishing a link.
Furthermore, if the authors do not report the publication to bioRxiv, the preprint and its corresponding published version are treated as distinct entities despite representing the same underlying research.
We hypothesize that close proximity in the document embedding space could match preprints with their corresponding published version.
If this finding holds, we could use this embedding space to fill in links missed by existing automated processes.
We used the subset of paper-preprint pairs annotated in CrossRef as described above to calculate the distribution of available preprint to published distances.
We calculated this distribution by taking the Euclidean distance between the preprint’s embedding coordinates and the coordinates of its corresponding published version.
We also calculated a background distribution, which consisted of the distance between each preprint with an annotated publication and a randomly selected article from the same journal.
We compared both distributions to determine if there was a difference between both groups as a significant difference would indicate that this embedding method can parse preprint-published pairs apart.
After comparing the two distributions, we calculated distances between preprints without a published version link with PMCOA articles that weren’t matched with a corresponding preprint.
We filtered any potential links with distances greater than the minimum value of the background distribution as we considered these pairs to be true negatives.
Lastly, we binned the remaining pairs based on percentiles from the annotated pairs distribution at the [0,25th percentile), [25th percentile, 50th percentile), [50th percentile, 75th percentile), and [75th percentile, minimum background distance).
We randomly sampled 50 articles from each bin and shuffled these four sets to produce a list of 200 potential preprint-published pairs with a randomized order.
We supplied these pairs to two co-authors to manually determine if each link between a preprint and a putative matched version was correct or incorrect.
After the curation process, we encountered eight disagreements between the reviewers.
We supplied these pairs to a third scientist, who carefully reviewed each case and made a final decision.
Using this curated set, we evaluated the extent to which distance in the embedding space revealed valid but unannotated links between preprints and their published versions.</p>
<h3 id="measuring-time-duration-for-preprint-publication-process">Measuring Time Duration for Preprint Publication Process</h3>
<p>Preprints can take varying amounts of time to be published.
We sought to measure the time required for preprints to be published in the peer-reviewed literature and compared this time measurement across author-selected preprint categories as well as individual preprints.
First, we queried bioRxiv’s application programming interface (API) to obtain the date a preprint was posted onto bioRxiv as well as the date a preprint was accepted for publication.
We did not include preprint matches found by our paper matching approach (see ‘Discovering Unannotated Preprint-Publication Relationships’).
We measured time elapsed as the difference between the date a preprint was first posted on bioRxiv and its publication date.
Along with calculating the time elapsed, we also recorded the number of different preprint versions posted onto bioRxiv.</p>
<p>We used this captured data to apply the Kaplan-Meier estimator <span class="citation" data-cites="PQlBxj25">[<a href="#ref-PQlBxj25" role="doc-biblioref">57</a>]</span> via the KaplanMeierFitter function from the lifelines <span class="citation" data-cites="4TsSA7hy">[<a href="#ref-4TsSA7hy" role="doc-biblioref">58</a>]</span> (version 0.25.6) python package to calculate the half-life of preprints across all preprint categories within bioRxiv.
We considered survival events as preprints that have yet to be published.
We encountered 123 cases where the preprint posting date was subsequent to the publication date, resulting in a negative time difference, as previously reported <span class="citation" data-cites="bN8m3HfR">[<a href="#ref-bN8m3HfR" role="doc-biblioref">59</a>]</span>.
We removed these preprints for this analysis as they were incompatible with the rules of the bioRxiv repository.</p>
<p>We measured the textual difference between preprints and their corresponding published version after our half-life calculation by calculating the Euclidean distance for their respective embedding representation.
This metric can be difficult to understand within the context of textual differences, so we sought to contextualize the meaning of a distance unit.
We first randomly sampled with replacement a pair of preprints from the Bioinformatics topic area as this was well represented within bioRxiv and contains a diverse set of research articles.
Next, we calculated the distance between two preprints 1000 times and reported the mean.
We repeated the above procedure using every preprint within bioRxiv as a whole.
These two means serve as normalized benchmarks to compare against as distance units are only meaningful when compared to other distances within the same space.
Following our contextualization approach, we performed linear regression to model the relationship between preprint version count with a preprint’s time to publication.
We also performed linear regression to measure the relationship between document embedding distance and a preprint’s time to publication.
For this analysis, we retained preprints with negative time within our linear regression model, and we observed that these preprints had minimal impact on results.
We visualize our version count regression model as a violin plot and our document embeddings regression model as a square bin plot.</p>
<h3 id="building-classifiers-to-detect-linguistically-similar-journal-venues-and-published-articles">Building Classifiers to Detect Linguistically Similar Journal Venues and Published Articles</h3>
<p>Preprints are more likely to be published in journals that publish articles with similar content.
We assessed this claim by building classifiers based on document and journal representations.
First, we removed all journals that had fewer than 100 papers in the PMC corpus.
We held our preprint-published subset (see above section ‘Mapping bioRxiv preprints to their published counterparts’) and treated it as a gold standard test set.
We used the remainder of the PMCOA corpus for training and initial evaluation for our models.</p>
<p>Training models to identify which journal publishes similar articles is challenging as not all journals are the same.
Some journals have a publication rate of at most hundreds of papers per year, while others publish at a rate of at least ten thousand papers per year.
Furthermore, some journals focus on publishing articles within a concentrated topic area, while others cover many dispersive topics.
Therefore, we designed two approaches to account for these characteristics.
Our first approach focuses on articles that account for a journal’s variation of publication topics.
This approach allows for topically similar papers to be retrieved independently of their respective journal.
Our second approach is centered on journals to account for varying publication rates.
This approach allows more selective or less popular journals to have equal representation to their high publishing counterparts.</p>
<p>Our article-based approach identifies most similar manuscripts to the preprint query, and we evaluated the journals that published these identified manuscripts.
We embedded each query article into the space defined by the word2vec model (see above section ‘Constructing a Document Representation for Life Sciences Text’).
Once embedded, we selected manuscripts close to the query via Euclidean distance in the embedding space.
Once identified, we return articles along with journals that published these identified articles.</p>
<p>We constructed a journal-based approach to accompany the article-based classifier while accounting for the overrepresentation of these high publishing frequency journals.
We identified the most similar journals for this approach by constructing a journal representation in the same embedding space.
We computed this representation by taking the average embedding of all published papers within a given journal.
We then projected a query article into the same space and returned journals closest to the query using the same distance calculation described above.</p>
<p>Both models were constructed using the scikit-learn k-Nearest Neighbors implementation <span class="citation" data-cites="vHIKEjNs">[<a href="#ref-vHIKEjNs" role="doc-biblioref">53</a>]</span> with the number of neighbors set to 10 as this is an appropriate number for our use case.
We consider a prediction to be a true positive if the correct journal appears within our reported list of neighbors and evaluate our performance using 10-fold cross-validation on the training set along with test set evaluation.</p>
<h3 id="web-application-for-discovering-similar-preprints-and-journals">Web Application for Discovering Similar Preprints and Journals</h3>
<p>We developed a web application that places any bioRxiv or medRxiv preprint into the overall document landscape and identifies topically similar papers and journals (similar to <span class="citation" data-cites="mZvqlKk6">[<a href="#ref-mZvqlKk6" role="doc-biblioref">60</a>]</span>).
Our application attempts to download the full text xml version of any preprint hosted on the bioRxiv or medRxiv server and uses the lxml package (version num) to extract text.
If the xml version isn’t available our application defaults to downloading the pdf version and uses PyMuPDF <span class="citation" data-cites="icndlpmW">[<a href="#ref-icndlpmW" role="doc-biblioref">61</a>]</span> to extract text from the pdf.
The extracted text is fed into our CBOW model to construct a document embedding representation.
We pass this representation onto our journal and article classifiers to identify journals based on the ten closest neighbors of individual papers and journal centroids.
We implemented this search using the scikit-learn implementation of k-d trees.
To run it more cost-effectively in a cloud computing environment with limited available memory, we sharded the k-d trees into four trees.</p>
<p>The app provides a visualization of the article’s position within our training data to illustrate the local publication landscape,
We used SAUCIE <span class="citation" data-cites="LxhnWcbN">[<a href="#ref-LxhnWcbN" role="doc-biblioref">62</a>]</span>, an autoencoder designed to cluster single-cell RNA-seq data, to build a two-dimensional embedding space that could be applied to newly generated preprints without retraining, a limitation of other approaches that we explored for visualizing entities expected to lie on a nonlinear manifold.
We trained this model on document embeddings of PMC articles that did not contain a matching preprint version.
We used the following parameters to train the model: a hidden size of 2, a learning rate of 0.001, lambda_b of 0, lambda_c of 0.001, and lambda_d of 0.001 for 5000 iterations.
When a user requests a new document, we can then project that document onto our generated two-dimensional space; thereby, allowing the user to see where their preprint falls along the landscape.
We illustrate our recommendations as a shortlist and provide access to our network visualization at our website (<a href="https://greenelab.github.io/preprint-similarity-search/">https://greenelab.github.io/preprint-similarity-search/</a>).</p>
<h3 id="analysis-of-the-preprints-in-motion-collection">Analysis of the Preprints in Motion Collection</h3>
<p>Our manuscript describes the large-scale analysis of bioRxiv.
Concurrent with our work, another set of authors performed a detailed curation and analysis of a subset of bioRxiv <span class="citation" data-cites="DgTUHUNP">[<a href="#ref-DgTUHUNP" role="doc-biblioref">25</a>]</span> that was focused on preprints posted during the initial stages of the COVID-19 pandemic.
The curated analysis was designed to examine preprints at a time of increased readership <span class="citation" data-cites="f22ESynO">[<a href="#ref-f22ESynO" role="doc-biblioref">63</a>]</span> and includes certain preprints posted from January 1st, 2020 to April 30th, 2020 <span class="citation" data-cites="DgTUHUNP">[<a href="#ref-DgTUHUNP" role="doc-biblioref">25</a>]</span>.
We sought to contextualize this subset, which we term “Preprints in Motion” after the title of the preprint <span class="citation" data-cites="DgTUHUNP">[<a href="#ref-DgTUHUNP" role="doc-biblioref">25</a>]</span>, within our global picture of the bioRxiv preprint landscape.
We extracted all preprints from the set reported in Preprints in Motion <span class="citation" data-cites="DgTUHUNP">[<a href="#ref-DgTUHUNP" role="doc-biblioref">25</a>]</span> and retained any entries in the bioRxiv repository.
We manually downloaded the XML version of these preprints and mapped them to their published counterparts as described above.
We used Pubmed Central’s DOI converter <span class="citation" data-cites="KyVRyag">[<a href="#ref-KyVRyag" role="doc-biblioref">64</a>/]</span> to map the published article DOIs with their respective PubMed Central IDs.
We retained articles that were included in the PMCOA corpus and performed a token analysis as described to compare these preprints with their published versions.
As above, we generated document embeddings for every obtained preprint and published article.
We projected these preprint embeddings onto our publication landscape to visually observe the dispersion of this subset.
We performed a time analysis that paralleled our approach for the full set of preprint-publication pairs to examine relationships between linguistic changes and the time to publication.
The “Preprints in Motion” subset includes recent papers, and the longest time to publish in that set was 195 days; however, our bioRxiv snapshot contains both older preprint-published pairs and many with publication times longer than this timepoint.
The optimum comparison would be to consider only preprints posted on the same days as preprints with the “Preprints in Motion” collection.
However, based on our results examining publication rate over time, these preprints may not have made it entirely through the publication process.
We performed a secondary analysis to control for the time since posting, where we filtered the bioRxiv snapshot to only contain publication pairs with publication time of less than or equal to 195 days.</p>
<h2 id="results">Results</h2>
<h3 id="comparing-biorxiv-to-other-corpora">Comparing bioRxiv to other corpora</h3>
<h4 id="biorxiv-metadata-statistics">bioRxiv Metadata Statistics</h4>
<p>The preprint landscape is rapidly changing, and the number of bioRxiv preprints in our data download (71,118) was nearly double that of a recent study that reported on a snapshot with 37,648 preprints <span class="citation" data-cites="iPnRRJya">[<a href="#ref-iPnRRJya" role="doc-biblioref">13</a>]</span>.
Because the rate of change is rapid, we first analyzed category data and compared our results with previous findings.
As in previous reports <span class="citation" data-cites="iPnRRJya">[<a href="#ref-iPnRRJya" role="doc-biblioref">13</a>]</span>, neuroscience remains the most common category of preprints, followed by bioinformatics (Supplemental Figure <a href="#fig:biorxiv_categories">S2</a>).
Microbiology, which was fifth in the most recent report <span class="citation" data-cites="iPnRRJya">[<a href="#ref-iPnRRJya" role="doc-biblioref">13</a>]</span>, has now surpassed evolutionary biology and genomics to move into third.
When authors upload their preprints, they select from three result category types: new results, confirmatory results, or contradictory results.
We found that nearly all preprints (97.5%) were categorized as new results, consistent with reports on a smaller set <span class="citation" data-cites="mTE5h91z">[<a href="#ref-mTE5h91z" role="doc-biblioref">65</a>]</span>.
The results taken together suggest that while bioRxiv has experienced dramatic growth, how it is being used appears to have remained consistent in recent years.</p>
<h4 id="global-analysis-reveals-similarities-and-differences-between-biorxiv-and-pmc">Global analysis reveals similarities and differences between bioRxiv and PMC</h4>
<div id="tbl:corpora_stats" class="tablenos">
<table id="tbl:corpora_stats">
<caption><span>Table 1:</span> Summary statistics for the bioRxiv, PMC, and NYTAC corpora. </caption>
<thead>
<tr class="header">
<th>Metric</th>
<th>bioRxiv</th>
<th>PMC</th>
<th>NYTAC</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td>document count</td>
<td>71,118</td>
<td>1,977,647</td>
<td>1,855,658</td>
</tr>
<tr class="even">
<td>sentence count</td>
<td>22,195,739</td>
<td>480,489,811</td>
<td>72,171,037</td>
</tr>
<tr class="odd">
<td>token count</td>
<td>420,969,930</td>
<td>8,597,101,167</td>
<td>1,218,673,384</td>
</tr>
<tr class="even">
<td>stopword count</td>
<td>158,429,441</td>
<td>3,153,077,263</td>
<td>559,391,073</td>
</tr>
<tr class="odd">
<td>avg. document length</td>
<td>312.10</td>
<td>242.96</td>
<td>38.89</td>
</tr>
<tr class="even">
<td>avg. sentence length</td>
<td>22.71</td>
<td>21.46</td>
<td>19.89</td>
</tr>
<tr class="odd">
<td>negatives</td>
<td>1,148,382</td>
<td>24,928,801</td>
<td>7,272,401</td>
</tr>
<tr class="even">
<td>coordinating conjunctions</td>
<td>14,295,736</td>
<td>307,082,313</td>
<td>38,730,053</td>
</tr>
<tr class="odd">
<td>coordinating conjunctions%</td>
<td>3.40%</td>
<td>3.57%</td>
<td>3.18%</td>
</tr>
<tr class="even">
<td>pronouns</td>
<td>4,604,432</td>
<td>74,994,125</td>
<td>46,712,553</td>
</tr>
<tr class="odd">
<td>pronouns%</td>
<td>1.09%</td>
<td>0.87%</td>
<td>3.83%</td>
</tr>
<tr class="even">
<td>passives</td>
<td>15,012,441</td>
<td>342,407,363</td>
<td>19,472,053</td>
</tr>
<tr class="odd">
<td>passive%</td>
<td>3.57%</td>
<td>3.98%</td>
<td>1.60%</td>
</tr>
</tbody>
</table>
</div>
<div id="fig:corpora_comparison_panels" class="fignos">
<figure>
<img src="https://raw.githubusercontent.com/danich1/annorxiver/65ee4a556ab69f2308e5e4d9192905e8cfec3728/figure_generation/output/Figure_1.png" style="width:100.0%" alt="Figure 1: A. The Kullback–Leibler divergence measures the extent to which the distributions, not specific tokens, differ from each other. The token distribution of bioRxiv and PMC corpora is more similar than these biomedical corpora are to the NYTAC one. B. The significant differences in token frequencies for the corpora appear to be driven by the fields with the highest uptake of bioRxiv, as terms from neuroscience and genomics are relatively more abundant in bioRxiv. We plotted the 95% confidence interval for each reported token. C. Of the tokens that differ between bioRxiv and PMC, the most abundant in bioRxiv are “et” and “al” while the most abundant in PMC is “study.” D. The significant differences in token frequencies for preprints and their corresponding published version often appear to be associated with typesetting and supplementary or additional materials. We plotted the 95% confidence interval for each reported token. E. The tokens with the largest absolute differences in abundance appear to be stylistic. Data for the information depicted in this figure are available at https://github.com/greenelab/annorxiver/blob/master/FIGURE_DATA_SOURCE.md#figure-one." /><figcaption aria-hidden="true"><span>Figure 1:</span> <strong>A.</strong> The Kullback–Leibler divergence measures the extent to which the distributions, not specific tokens, differ from each other.
The token distribution of bioRxiv and PMC corpora is more similar than these biomedical corpora are to the NYTAC one.
<strong>B.</strong> The significant differences in token frequencies for the corpora appear to be driven by the fields with the highest uptake of bioRxiv, as terms from neuroscience and genomics are relatively more abundant in bioRxiv.
We plotted the 95% confidence interval for each reported token.
<strong>C.</strong> Of the tokens that differ between bioRxiv and PMC, the most abundant in bioRxiv are “et” and “al” while the most abundant in PMC is “study.”
<strong>D.</strong> The significant differences in token frequencies for preprints and their corresponding published version often appear to be associated with typesetting and supplementary or additional materials.
We plotted the 95% confidence interval for each reported token.
<strong>E.</strong> The tokens with the largest absolute differences in abundance appear to be stylistic.
Data for the information depicted in this figure are available at <a href="https://github.com/greenelab/annorxiver/blob/master/FIGURE_DATA_SOURCE.md#figure-one">https://github.com/greenelab/annorxiver/blob/master/FIGURE_DATA_SOURCE.md#figure-one</a>.</figcaption>
</figure>
</div>
<p>Documents within bioRxiv were slightly longer than those within PMCOA, but both were much longer than those from the control (NYTAC) (Table <a href="#tbl:corpora_stats">1</a>).
The average sentence length, the fraction of pronouns, and the use of the passive voice were all more similar between bioRxiv and PMC than they were to NYTAC(Table <a href="#tbl:corpora_stats">1</a>).
The Kullback–Leibler (KL) divergence of term frequency distributions between bioRxiv and PMCOA were low, especially among the top few hundred tokens (Figure <a href="#fig:corpora_comparison_panels">1</a>A).
As more tokens were incorporated, the KL divergence started to increase but remained much lower than the biomedical corpora compared against NYTAC.
We provide a listing of the top 100 most frequently occurring tokens from all three corpora in our supplement (Supplemental Table <a href="#tbl:token_table">S4</a>).
These findings support our notion that bioRxiv is linguistically similar to the PMCOA repository.</p>
<p>The terms “neurons”, “genome”, and “genetic”, which are common in genomics and neuroscience, were more common in bioRxiv than PMCOA while others associated with clinical research, such as “clinical” “patients” and “treatment” were more common in PMCOA (Figure <a href="#fig:corpora_comparison_panels">1</a>B, <a href="#fig:corpora_comparison_panels">1</a>C and Supplementary Figure <a href="#fig:corpora_comparison_panels_bio_v_pmid">S3</a>).
When controlling for the differences in the body of documents to identify textual changes associated with the publication process, we found that tokens such as “et” “al” were enriched for bioRxiv while “<span class="math inline">\(\pm\)</span>”, “–” were enriched for PMCOA (Figure <a href="#fig:corpora_comparison_panels">1</a>D, <a href="#fig:corpora_comparison_panels">1</a>E).
When removing special and single-character tokens, data availability and presentation related terms “fle”, “supplementary”, “fig” appeared enriched for published articles, and research related terms “mice”, “activity”, “neurons” appeared enriched for bioRxiv (Supplementary Figure <a href="#fig:corpora_comparison_panels_pre_v_pub">S4</a>).
Furthermore, we found that specific changes appeared to be related to journal styles: “figure” was more common in bioRxiv while “fig” was relatively more common in PMCOA.
Other changes appeared to be associated with an increasing reference to content external to the manuscript itself: the tokens “supplementary”, “additional” and “file” were all more common in PMCOA than bioRxiv, suggesting that journals are not simply replacing one token with another but that there are more mentions of such content after peer review.</p>
<p>These results suggest that the text structure within preprints on bioRxiv is similar to published articles within PMCOA.
The differences in uptake across fields are supported by the authors’ categorization of their articles and the text within the articles themselves.
At the level of individual manuscripts, the most change terms appear to be associated with typesetting, journal style, and an increasing reliance on additional materials after peer review.</p>
<p>Following our analysis of tokens, we examined the principal components of document embeddings derived from bioRxiv.
We found that the top principal components separated methodological approaches and research fields.
Preprints from certain topic areas that spanned approaches from informatics-related to cell biology could be distinguished using these principal components (see Supplementary Results).</p>
<h3 id="document-embedding-similarities-reveal-unannotated-preprint-publication-pairs">Document embedding similarities reveal unannotated preprint-publication pairs</h3>
<div id="fig:preprint_links_panels" class="fignos">
<figure>
<img src="https://raw.githubusercontent.com/danich1/annorxiver/65ee4a556ab69f2308e5e4d9192905e8cfec3728/figure_generation/output/Figure_2.png" style="width:100.0%" alt="Figure 2: A. Preprints are closer in document embedding space to their corresponding peer-reviewed publication than they are to random papers published in the same journal. B. Potential preprint-publication pairs that are unannotated but within the 50th percentile of all preprint-publication pairs in the document embedding space are likely to represent true preprint-publication pairs. We depict the fraction of true positives over the total number of pairs in each bin. Accuracy is derived from the curation of a randomized list of 200 potential pairs (50 per quantile) performed in duplicate with a third rater used in the case of disagreement. C. Most preprints are eventually published. We show the publication rate of preprints since bioRxiv first started. The x-axis represents months since bioRxiv started, and the y-axis represents the proportion of preprints published given the month they were posted. The light blue line represents the publication rate previously estimated by Abdill et al. [13]. The dark blue line represents the updated publication rate using only CrossRef-derived annotations, while the dark green line includes annotations derived from our embedding space approach. The horizontal lines represent the overall proportion of preprints published as of the time of the annotated snapshot. The dashed horizontal line represents the overall proportion published preprints for preprints posted before 2019. Data for the information depicted in this figure are available at https://github.com/greenelab/annorxiver/blob/master/FIGURE_DATA_SOURCE.md#figure-two." /><figcaption aria-hidden="true"><span>Figure 2:</span> <strong>A.</strong> Preprints are closer in document embedding space to their corresponding peer-reviewed publication than they are to random papers published in the same journal.
<strong>B.</strong> Potential preprint-publication pairs that are unannotated but within the 50th percentile of all preprint-publication pairs in the document embedding space are likely to represent true preprint-publication pairs.
We depict the fraction of true positives over the total number of pairs in each bin.
Accuracy is derived from the curation of a randomized list of 200 potential pairs (50 per quantile) performed in duplicate with a third rater used in the case of disagreement.
<strong>C.</strong> Most preprints are eventually published.
We show the publication rate of preprints since bioRxiv first started.
The x-axis represents months since bioRxiv started, and the y-axis represents the proportion of preprints published given the month they were posted.
The light blue line represents the publication rate previously estimated by Abdill et al. <span class="citation" data-cites="iPnRRJya">[<a href="#ref-iPnRRJya" role="doc-biblioref">13</a>]</span>.
The dark blue line represents the updated publication rate using only CrossRef-derived annotations, while the dark green line includes annotations derived from our embedding space approach.
The horizontal lines represent the overall proportion of preprints published as of the time of the annotated snapshot.
The dashed horizontal line represents the overall proportion published preprints for preprints posted before 2019.
Data for the information depicted in this figure are available at <a href="https://github.com/greenelab/annorxiver/blob/master/FIGURE_DATA_SOURCE.md#figure-two">https://github.com/greenelab/annorxiver/blob/master/FIGURE_DATA_SOURCE.md#figure-two</a>.</figcaption>
</figure>
</div>
<p>Distances between preprints and their corresponding published versions were nearly always lower than preprints paired with a random article published in the same journal (Figure <a href="#fig:preprint_links_panels">2</a>A).
This suggested that embedding distances may predict the published form of preprints.
We directly tested this by selecting low-distance but unannotated preprint-publication pairs and curating the extent to which they represented matching documents.
Approximately 98% of our 200 pairs with an embedding distance in the 0-25th and 25th-50th percentile bins were successfully matched with their published counterpart (Figure <a href="#fig:preprint_links_panels">2</a>B).
These two bins contained 1,542 preprint-article pairs, suggesting that many preprints may have been published but not previously connected with their published versions.
There is a particular enrichment for preprints published but unlinked within the 2017-2018 interval (Figure <a href="#fig:preprint_links_panels">2</a>C).
We expected a higher proportion of such preprints before 2019 (many of which may not have been published yet); however, observing relatively few missed annotations before 2017 was against our expectations.
There are several possible explanations for this increasing fraction of missed annotations.
As the number of preprints posted on bioRxiv grows, it may be harder for bioRxiv to establish a link between preprints and their published counterparts simply due to the scale of the challenge.
It is possible that the set of authors participating in the preprint ecosystem is changing and that new participants may be less likely to report missed publications to bioRxiv.
Finally, as familiarity with preprinting grows, it is possible that authors are posting preprints earlier in the process and that metadata fields that bioRxiv uses to establish a link may be less stable.</p>
<h3 id="preprints-with-more-versions-or-more-text-changes-relative-to-their-published-counterpart-took-longer-to-publish">Preprints with more versions or more text changes relative to their published counterpart took longer to publish</h3>
<div id="fig:publication_delay_panels" class="fignos">
<figure>
<img src="https://raw.githubusercontent.com/danich1/annorxiver/65ee4a556ab69f2308e5e4d9192905e8cfec3728/figure_generation/output/Figure_3.png" style="width:100.0%" alt="Figure 3: A. Author-selected categories were associated with modest differences in the median time to publish. Author-selected preprint categories are shown on the y-axis, while the x-axis shows the median time-to-publish for each category. Error bars represent 95% confidence intervals for each median measurement. B. Preprints with more versions were associated with a longer time to publish. The x-axis shows the number of versions of a preprint posted on bioRxiv. The y-axis indicates the number of days that elapsed between the first version of a preprint posted on bioRxiv and the date at which the peer-reviewed publication appeared. The density of observations is depicted in the violin plot with an embedded boxplot. C. Preprints with more substantial text changes took longer to be published. The x-axis shows the Euclidean distance between document representations of the first version of a preprint and its peer-reviewed form. The y-axis shows the number of days elapsed between the first version of a preprint posted on bioRxiv and when a preprint is published. The color bar on the right represents the density of each hexbin in this plot, where more dense regions are shown in a brighter color. Data for the information depicted in this figure are available at https://github.com/greenelab/annorxiver/blob/master/FIGURE_DATA_SOURCE.md#figure-three." /><figcaption aria-hidden="true"><span>Figure 3:</span> <strong>A.</strong> Author-selected categories were associated with modest differences in the median time to publish.
Author-selected preprint categories are shown on the y-axis, while the x-axis shows the median time-to-publish for each category.
Error bars represent 95% confidence intervals for each median measurement.
<strong>B.</strong> Preprints with more versions were associated with a longer time to publish.
The x-axis shows the number of versions of a preprint posted on bioRxiv.
The y-axis indicates the number of days that elapsed between the first version of a preprint posted on bioRxiv and the date at which the peer-reviewed publication appeared.
The density of observations is depicted in the violin plot with an embedded boxplot.
<strong>C.</strong> Preprints with more substantial text changes took longer to be published.
The x-axis shows the Euclidean distance between document representations of the first version of a preprint and its peer-reviewed form.
The y-axis shows the number of days elapsed between the first version of a preprint posted on bioRxiv and when a preprint is published.
The color bar on the right represents the density of each hexbin in this plot, where more dense regions are shown in a brighter color.
Data for the information depicted in this figure are available at <a href="https://github.com/greenelab/annorxiver/blob/master/FIGURE_DATA_SOURCE.md#figure-three">https://github.com/greenelab/annorxiver/blob/master/FIGURE_DATA_SOURCE.md#figure-three</a>.</figcaption>
</figure>
</div>
<p>The process of peer review includes several steps, which take variable amounts of time <span class="citation" data-cites="TMxQYqVE">[<a href="#ref-TMxQYqVE" role="doc-biblioref">66</a>]</span>, and we sought to measure if there is a difference in publication time between author-selected categories of preprints (Figure <a href="#fig:publication_delay_panels">3</a>A).
Of the most abundant preprint categories microbiology was the fastest to publish (140 days, (137, 145 days) [95% CI]) and genomics was the slowest (190 days, (185, 195 days) [95% CI]) (Figure <a href="#fig:publication_delay_panels">3</a>A).
We did observe category-specific differences; however, these differences were generally modest, suggesting that the peer review process did not differ dramatically between preprint categories.
One exception was the Scientific Communication and Education category, which took substantially longer to be peer-reviewed and published (373 days, (373, 398 days) [95% CI]).
This hints that there may be differences in the publication or peer review process or culture that apply to preprints in this category.</p>
<p>Examining peer review’s effect on individual preprints, we found a positive correlation between preprints with multiple versions and the time elapsed until publication (Figure <a href="#fig:publication_delay_panels">3</a>B).
Every additional preprint version was associated with an increase of 51 days before a preprint was published.
This time duration seems broadly compatible with the amount of time it would take to receive reviews and revise a manuscript, suggesting that many authors may be updating their preprints in response to peer reviews or other external feedback.
The embedding space allows us to compare preprint and published documents to determine if the level of change that documents undergo relates to the time it takes them to be published.
Distances in this space are arbitrary and must be compared to reference distances.
We found that the average distance of two randomly selected papers from the bioinformatics category was 4.470, while the average distance of two randomly selected papers from bioRxiv was 5.343.
Preprints with large embedding space distances from their corresponding peer-reviewed publication took longer to publish (Figure <a href="#fig:publication_delay_panels">3</a>C): each additional unit of distance corresponded to roughly forty-three additional days.</p>
<p>Overall, our findings support a model where preprints are reviewed multiple times or require more extensive revisions take longer to publish.</p>
<h3 id="preprints-with-similar-document-embeddings-share-publication-venues">Preprints with similar document embeddings share publication venues</h3>
<p>We developed an online application that returns a listing of published papers and journals closest to a query preprint in document embedding space.
This application uses two k-nearest neighbor classifiers that achieved better performance than our baseline model (Supplemental Figure <a href="#fig:knn_auc">S5</a>) to identify these entities.
Users supply our app with digital object identifiers (DOIs) from bioRxiv or medRxiv, and the corresponding preprint is downloaded from the repository.
Next, the preprint’s PDF is converted to text, and this text is used to construct a document embedding representation.
This representation is supplied to our classifiers to generate a listing of the ten papers and journals with the most similar representations in the embedding space (Figures <a href="#fig:journal_rec_workflow">4</a>A, <a href="#fig:journal_rec_workflow">4</a>B and <a href="#fig:journal_rec_workflow">4</a>C).
Furthermore, the user-requested preprint’s location in this embedding space is then displayed on our interactive map, and users can select regions to identify the terms most associated with those regions (Figures <a href="#fig:journal_rec_workflow">4</a>D and <a href="#fig:journal_rec_workflow">4</a>E).
Users can also explore the terms associated with the top 50 PCs derived from the document embeddings, and those PCs vary across the document landscape.
You can access this application using the following url: <a href="https://greenelab.github.io/preprint-similarity-search/">https://greenelab.github.io/preprint-similarity-search/</a></p>
<div id="fig:journal_rec_workflow" class="fignos">
<figure>
<img src="images/Figure_4.png" style="width:100.0%" alt="Figure 4: The preprint-similarity-search app workflow allows users to examine where an individual preprint falls in the overall document landscape. A. Starting with the home screen, users can paste in a bioRxiv or medRxiv DOI, which sends a request to bioRxiv or medRxiv. Next, the app preprocesses the requested preprint and returns a listing of (B) the top ten most similar papers and (C) the ten closest journals. D. The app also displays the location of the query preprint in PMC. E. Users can select a square within the landscape to examine statistics associated with the square, including the top journals by article count in that square and the odds ratio of tokens." /><figcaption aria-hidden="true"><span>Figure 4:</span> The preprint-similarity-search app workflow allows users to examine where an individual preprint falls in the overall document landscape.
<strong>A.</strong> Starting with the home screen, users can paste in a bioRxiv or medRxiv DOI, which sends a request to bioRxiv or medRxiv.
Next, the app preprocesses the requested preprint and returns a listing of (<strong>B</strong>) the top ten most similar papers and (<strong>C</strong>) the ten closest journals.
<strong>D.</strong> The app also displays the location of the query preprint in PMC.
<strong>E.</strong> Users can select a square within the landscape to examine statistics associated with the square, including the top journals by article count in that square and the odds ratio of tokens.</figcaption>
</figure>
</div>
<h3 id="contextualizing-the-preprints-in-motion-collection">Contextualizing the Preprints in Motion Collection</h3>
<div id="fig:covid_19_preprints" class="fignos">
<figure>
<img src="https://raw.githubusercontent.com/danich1/annorxiver/65ee4a556ab69f2308e5e4d9192905e8cfec3728/figure_generation/output/Figure_5.png" style="width:100.0%" alt="Figure 5: The Preprints in Motion Collection results are similar to all preprint results, except that their time to publication was independent of the number of preprint versions and amount of linguistic change. A. Tokens that differed included those associated with typesetting and those related to the nomenclature of the virus that causes COVID-19. Error bars show 95% confidence intervals for each token. B. Of the tokens that differ between Preprints in Motion and their published counterparts, the most abundant were associated with the nomenclature of the virus. C. The Preprints in Motion collection fall across the landscape of PMCOA with respect to linguistic properties. This square bin plot depicts the binning of all published papers within the PMCOA corpus. High-density regions are depicted in yellow, while low-density regions are in dark blue. Red dots represent the Preprints in Motion Collection. D. The Preprints in Motion collection were published faster than other bioRxiv preprints, and the number of versions was not associated with an increase in time to publication. The x-axis shows the number of versions of a preprint posted on bioRxiv. The y-axis indicates the number of days that elapsed between the first version of a preprint posted on bioRxiv and the date at which the peer-reviewed publication appeared. The density of observations is depicted in the violin plot with an embedded boxplot. The red dots and red regression line represent Preprints in Motion. E. The Preprints in Motion collection were published faster than other bioRxiv preprints, and no dependence between the amount of linguistic change and time to publish was observed. The x-axis shows the Euclidean distance between document representations of the first version of a preprint and its peer-reviewed form. The y-axis shows the number of days elapsed between the first version of a preprint posted on bioRxiv and when a preprint is published. The color bar on the right represents the density of each hexbin in this plot, where more dense regions are shown in a brighter color. The red dots and red regression line represent Preprints in Motion. Data for the information depicted in this figure are available at https://github.com/greenelab/annorxiver/blob/master/FIGURE_DATA_SOURCE.md#figure-five." /><figcaption aria-hidden="true"><span>Figure 5:</span> The Preprints in Motion Collection results are similar to all preprint results, except that their time to publication was independent of the number of preprint versions and amount of linguistic change.
<strong>A.</strong> Tokens that differed included those associated with typesetting and those related to the nomenclature of the virus that causes COVID-19.
Error bars show 95% confidence intervals for each token.
<strong>B.</strong> Of the tokens that differ between Preprints in Motion and their published counterparts, the most abundant were associated with the nomenclature of the virus.
<strong>C.</strong> The Preprints in Motion collection fall across the landscape of PMCOA with respect to linguistic properties.
This square bin plot depicts the binning of all published papers within the PMCOA corpus.
High-density regions are depicted in yellow, while low-density regions are in dark blue.
Red dots represent the Preprints in Motion Collection.
<strong>D.</strong> The Preprints in Motion collection were published faster than other bioRxiv preprints, and the number of versions was not associated with an increase in time to publication.
The x-axis shows the number of versions of a preprint posted on bioRxiv.
The y-axis indicates the number of days that elapsed between the first version of a preprint posted on bioRxiv and the date at which the peer-reviewed publication appeared.
The density of observations is depicted in the violin plot with an embedded boxplot.
The red dots and red regression line represent Preprints in Motion.
<strong>E.</strong> The Preprints in Motion collection were published faster than other bioRxiv preprints, and no dependence between the amount of linguistic change and time to publish was observed.
The x-axis shows the Euclidean distance between document representations of the first version of a preprint and its peer-reviewed form.
The y-axis shows the number of days elapsed between the first version of a preprint posted on bioRxiv and when a preprint is published.
The color bar on the right represents the density of each hexbin in this plot, where more dense regions are shown in a brighter color.
The red dots and red regression line represent Preprints in Motion.
Data for the information depicted in this figure are available at <a href="https://github.com/greenelab/annorxiver/blob/master/FIGURE_DATA_SOURCE.md#figure-five">https://github.com/greenelab/annorxiver/blob/master/FIGURE_DATA_SOURCE.md#figure-five</a>.</figcaption>
</figure>
</div>
<p>The Preprints in Motion collection included a set of preprints posted during the first four months of 2020.
We examined the extent to which preprints in this set were representative of the patterns that we identified from our analysis on all of bioRxiv.
As with all of bioRxiv, typesetting tokens changed between preprints and their paired publications.
Our token-level analysis identified certain patterns consistent with our findings across bioRxiv (Figure <a href="#fig:covid_19_preprints">5</a>A and <a href="#fig:covid_19_preprints">5</a>B).
However, in this set, we also observe changes likely associated with the fast-moving nature of COVID-19 research: the token “2019-ncov” became less frequently represented while “sars” and “cov-2” became more represented, likely due to a shift in nomenclature from “2019-nCoV” to “SARS-CoV-2”.
The Preprints in Motion were not strongly colocalized in the linguistic landscape, suggesting that the collection covers a diverse set of research approaches (Figure <a href="#fig:covid_19_preprints">5</a>C).
Preprints in this collection were published faster than the broader set of bioRxiv preprints (Figure <a href="#fig:covid_19_preprints">5</a>D and <a href="#fig:covid_19_preprints">5</a>E).
We see the same trend when filtering the broader bioRxiv set to only contain preprints published within the same timeframe as this collection (Supplemental Figures <a href="#fig:filterd_polka_analysis">S6</a>A and <a href="#fig:filterd_polka_analysis">S6</a>B).
The relationship between time to publication and the number of versions (Figure <a href="#fig:covid_19_preprints">5</a>D and Supplemental Figure <a href="#fig:filterd_polka_analysis">S6</a>A) and the relationship between time to publication and the amount of linguistic change (Figure <a href="#fig:covid_19_preprints">5</a>E and Supplemental Figure <a href="#fig:filterd_polka_analysis">S6</a>B) were both lost in the Preprints in Motion set.
Our findings suggest that Preprints in Motion changed during publication in ways aligned with changes in the full preprint set but that peer review was accelerated in ways that broke the time dependencies observed with the full bioRxiv set.</p>
<h2 id="discussion-and-conclusions">Discussion and Conclusions</h2>
<p>BioRxiv is a constantly growing repository that contains life science preprints.
Over 77% of bioRxiv preprints with a corresponding publication in our snapshot were successfully detected within Pubmed Central’s Open Access Corpus (PMCOA).
This suggests that most work from groups participating in the preprint ecosystem is now available in final form for literature mining and other applications.
Most research on bioRxiv preprints has examined their metadata; we examine the text content as well.
Throughout this work, we sought to analyze the language within these preprints and understand how it changes in response to peer review.</p>
<p>Our global corpora analysis found that writing within bioRxiv is consistent with the biomedical literature in the PMCOA repository, suggesting that bioRxiv is linguistically similar to PMCOA.
Token-level analyses between bioRxiv and PMCOA suggested that research fields drive significant differences; e.g., more patient-related research is prevalent in PMCOA than bioRxiv.
This observation is expected as preprints focused on medicine are supported by the complementary medRxiv repository <span class="citation" data-cites="NKHNs9j3">[<a href="#ref-NKHNs9j3" role="doc-biblioref">8</a>/]</span>.
Token-level analyses for preprints and their corresponding published version suggest that peer review may focus on data availability and incorporating extra sections for published papers; however, future studies are needed to ascertain individual token level changes as preprints venture through the publication process.
One future avenue of research could examine the differences between only preprints and accepted author manuscripts within Pubmed Central to identify changes prior to journal publication.</p>
<p>Document embeddings are a versatile way to examine language contained within preprints, understanding peer review’s effect on preprints, and provide extra functionality for preprint repositories.
Our approach to generate document embeddings was focused on interpretability instead of predictive performance; however, using more advanced strategies to generate document vectors such as Doc2Vec <span class="citation" data-cites="SdcIDSwR">[<a href="#ref-SdcIDSwR" role="doc-biblioref">37</a>]</span> or BERT <span class="citation" data-cites="SkPpj3hW">[<a href="#ref-SkPpj3hW" role="doc-biblioref">67</a>]</span> should increase predictive performance.
Examining linguistic variance within document embeddings of life science preprints revealed that the largest source of variability was informatics.
This observation bisects the majority of life science research categories that have integrated preprints within their publication workflow.
This embedding space could also be used to quantify sentiment trends or other linguistic features.
Furthermore, methodologies for uncovering latent scientific knowledge <span class="citation" data-cites="kPI0iXkU">[<a href="#ref-kPI0iXkU" role="doc-biblioref">68</a>]</span> may be applicable in this embedding space.</p>
<p>Preprints are typically linked with their published articles via bioRxiv manually establishing links or authors self-reporting that their preprint has been published; however, gaps can occur as preprints change their appearance through multiple versions or authors do not notify bioRxiv.
Our work suggests that document embeddings can help fill in missing links within bioRxiv.<br />
Furthermore, our analysis reveals that the publication rate for preprints is higher than previously estimated, even though our analysis can only account for published open access papers.
Our results raise the lower bound of the total preprint publication fraction; however, the true fraction is necessarily higher.
Future work, especially that which aims to assess the fraction of preprints that are eventually published, should account for the possibility of missed annotations.</p>
<p>Preprints take a variable amount of time to become published, and we examined factors that influence a preprint’s time to publication.
Our half-life analysis on preprint categories revealed that preprints in most bioRxiv categories take similar amounts of time to be published.
An apparent exception is the scientific communication and education category, which contained preprints that took much longer to publish.
Regarding individual preprints, each new version adds several weeks to a preprints time to publication, which is roughly aligned with authors making changes after a round of peer review; furthermore, preprints that undergo substantial changes take longer to publish.
Overall, these results illustrate that bioRxiv is a practical resource for obtaining insight into the peer-review process.</p>
<p>Lastly, we found that document embeddings were associated with the eventual journal at which the work was published.
We trained two machine learning models to identify which journals publish linguistically similar papers towards a query preprint.
Our models achieved a considerably higher fold change over the baseline model, so we constructed a web application that makes our models available to the public and returns a list of the papers and journals that are linguistically similar to a bioRxiv or medRxiv preprint.</p>
<h2 id="software-and-data-availability">Software and Data Availability</h2>
<p>An online version of this manuscript is available under a Creative Commons Attribution License at <a href="https://greenelab.github.io/annorxiver_manuscript/">https://greenelab.github.io/annorxiver_manuscript/</a>.
Source for the research portions of this project is dual licensed under the BSD 3-Clause and Creative Commons Public Domain Dedication Licenses at <a href="https://github.com/greenelab/annorxiver">https://github.com/greenelab/annorxiver</a>.
All corresponding data for every figure in this manuscript is available at <a href="https://github.com/greenelab/annorxiver/blob/master/FIGURE_DATA_SOURCE.md">https://github.com/greenelab/annorxiver/blob/master/FIGURE_DATA_SOURCE.md</a>.
The preprint similarity search website can be found at <a href="https://greenelab.github.io/preprint-similarity-search/">https://greenelab.github.io/preprint-similarity-search/</a>, and code for the website is available under a BSD-2-Clause Plus Patent License at <a href="https://github.com/greenelab/preprint-similarity-search">https://github.com/greenelab/preprint-similarity-search</a>.
Full text access for the bioRxiv repository is available at <a href="https://www.biorxiv.org/tdm">https://www.biorxiv.org/tdm</a>.
Access to PubMed Central’s Open Access subset is available on NCBI’s FTP server at <a href="https://www.ncbi.nlm.nih.gov/pmc/tools/ftp/">https://www.ncbi.nlm.nih.gov/pmc/tools/ftp/</a>.
New York Times Annotated Corpus (NYTAC) can be accessed from the Linguistic Data Consortium at <a href="https://catalog.ldc.upenn.edu/LDC2008T19">https://catalog.ldc.upenn.edu/LDC2008T19</a> where there may be a $150 to $300 fee depending on membership status.</p>
<h2 id="acknowledgments">Acknowledgments</h2>
<p>The authors would like to thank Ariel Hippen Anderson for evaluating potential missing preprint to published version links.
We also would like to thank Richard Sever and the bioRxiv team for their assistance with access to and support with questions about preprint full text downloaded from bioRxiv.</p>
<h2 id="funding">Funding</h2>
<p>This work was supported by grants from the Gordon Betty Moore Foundation (GBMF4552) and the National Institutes of Health’s National Human Genome Research Institute (NHGRI) under award R01 HG010067 to CSG and the National Institutes of Health’s NHGRI under award T32 HG00046 to DNN.
The funders had no role in study design, data collection and analysis, decision to publish, or preparation of the manuscript.</p>
<h2 id="competing-interests">Competing Interests</h2>
<p>I have read the journal’s policy and the authors of this manuscript have the following competing interests: Marvin Thielk receives a salary from Elsevier Inc. where he contributes NLP expertise to health content operations.
Elsevier did not restrict the results or interpretations that could be published in this manuscript.
The opinions expressed here do not reflect the official policy or positions of Elsevier Inc.</p>
<h2 class="page_break_before" id="references">References</h2>
<!-- Explicitly insert bibliography here -->
<div id="refs" class="references csl-bib-body" role="doc-bibliography">
<div id="ref-TQgSqYqx" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1. </div><div class="csl-right-inline"><strong>Scientific communication pathways: an overview and introduction to a symposium</strong> <div class="csl-block">David F Zaye, WV Metanomski</div> <em>Journal of Chemical Information and Computer Sciences</em> (2002-05-01) <a href="https://doi.org/bwsxhg">https://doi.org/bwsxhg</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1021/ci00050a001">10.1021/ci00050a001</a></div></div>
</div>
<div id="ref-u6LNiV2V" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">2. </div><div class="csl-right-inline"><strong>The trouble with medical journals</strong> <div class="csl-block">Richard Smith</div> <em>Journal of the Royal Society of Medicine</em> (2006)</div>
</div>
<div id="ref-es2rrG6B" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">3. </div><div class="csl-right-inline"><strong>The Transition from Paper to Electronic Journals</strong> <div class="csl-block">Hak Joon Kim</div> <em>The Serials Librarian</em> (2001-11-19) <a href="https://doi.org/d7rnh2">https://doi.org/d7rnh2</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1300/j123v41n01_04">10.1300/j123v41n01_04</a></div></div>
</div>
<div id="ref-ZdEaAXLe" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">4. </div><div class="csl-right-inline"><strong>Preprints: What Role Do These Have in Communicating Scientific Results?</strong> <div class="csl-block">Susan A Elmore</div> <em>Toxicologic Pathology</em> (2018-04-08) <a href="https://doi.org/ghdd7c">https://doi.org/ghdd7c</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1177/0192623318767322">10.1177/0192623318767322</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/29628000">29628000</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5999550">PMC5999550</a></div></div>
</div>
<div id="ref-ePmdRbmq" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">5. </div><div class="csl-right-inline"><strong>The prehistory of biology preprints: A forgotten experiment from the 1960s</strong> <div class="csl-block">Matthew Cobb</div> <em>PLOS Biology</em> (2017-11-16) <a href="https://doi.org/c6wv">https://doi.org/c6wv</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1371/journal.pbio.2003995">10.1371/journal.pbio.2003995</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/29145518">29145518</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5690419">PMC5690419</a></div></div>
</div>
<div id="ref-ZTyQYlDz" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">6. </div><div class="csl-right-inline"><strong>arXiv.org: the Los Alamos National Laboratory e‐print server</strong> <div class="csl-block">Gerry McKiernan</div> <em>International Journal on Grey Literature</em> (2000-09) <a href="https://doi.org/fg8pw7">https://doi.org/fg8pw7</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1108/14666180010345564">10.1108/14666180010345564</a></div></div>
</div>
<div id="ref-i9BKPuqb" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">7. </div><div class="csl-right-inline"><strong>bioRxiv: the preprint server for biology</strong> <div class="csl-block">Richard Sever, Ted Roeder, Samantha Hindle, Linda Sussman, Kevin-John Black, Janet Argentine, Wayne Manos, John R Inglis</div> <em>Cold Spring Harbor Laboratory</em> (2019-11-06) <a href="https://doi.org/ggc46z">https://doi.org/ggc46z</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1101/833400">10.1101/833400</a></div></div>
</div>
<div id="ref-NKHNs9j3" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">8. </div><div class="csl-right-inline"><strong>medRxiv.org - the preprint server for Health Sciences</strong> <a href="https://www.medrxiv.org/">https://www.medrxiv.org/</a></div>
</div>
<div id="ref-lMj4YzIP" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">9. </div><div class="csl-right-inline"><strong>The Second Wave of Preprint Servers: How Can Publishers Keep Afloat?</strong> <div class="csl-block">By</div> <em>The Scholarly Kitchen</em> (2019-10-16) <a href="https://scholarlykitchen.sspnet.org/2019/10/16/the-second-wave-of-preprint-servers-how-can-publishers-keep-afloat/">https://scholarlykitchen.sspnet.org/2019/10/16/the-second-wave-of-preprint-servers-how-can-publishers-keep-afloat/</a></div>
</div>
<div id="ref-1DCEzEMkZ" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">10. </div><div class="csl-right-inline"><strong>Rxivist.org: Sorting biology preprints using social media and readership metrics</strong> <div class="csl-block">Richard J Abdill, Ran Blekhman</div> <em>PLOS Biology</em> (2019-05-21) <a href="https://doi.org/dm27">https://doi.org/dm27</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1371/journal.pbio.3000269">10.1371/journal.pbio.3000269</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/31112533">31112533</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6546241">PMC6546241</a></div></div>
</div>
<div id="ref-PJwR7lcq" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">11. </div><div class="csl-right-inline"><strong>How the Scientific Community Reacts to Newly Submitted Preprints: Article Downloads, Twitter Mentions, and Citations</strong> <div class="csl-block">Xin Shuai, Alberto Pepe, Johan Bollen</div> <em>PLoS ONE</em> (2012-11-01) <a href="https://doi.org/f4cw6r">https://doi.org/f4cw6r</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1371/journal.pone.0047523">10.1371/journal.pone.0047523</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/23133597">23133597</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3486871">PMC3486871</a></div></div>
</div>
<div id="ref-QM87Cg3q" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">12. </div><div class="csl-right-inline"><strong>Quantifying and contextualizing the impact of bioRxiv preprints through automated social media audience segmentation</strong> <div class="csl-block">Jedidiah Carlson, Kelley Harris</div> <em>PLOS Biology</em> (2020-09-22) <a href="https://doi.org/ghk53x">https://doi.org/ghk53x</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1371/journal.pbio.3000860">10.1371/journal.pbio.3000860</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32960891">32960891</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7508356">PMC7508356</a></div></div>
</div>
<div id="ref-iPnRRJya" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">13. </div><div class="csl-right-inline"><strong>Tracking the popularity and outcomes of all bioRxiv preprints</strong> <div class="csl-block">Richard J Abdill, Ran Blekhman</div> <em>eLife</em> (2019-04-24) <a href="https://doi.org/gf2str">https://doi.org/gf2str</a> <div class="csl-block">DOI: <a href="https://doi.org/10.7554/elife.45133">10.7554/elife.45133</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/31017570">31017570</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6510536">PMC6510536</a></div></div>
</div>
<div id="ref-LcHjf487" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">14. </div><div class="csl-right-inline"><strong>An analysis of published journals for papers posted on bioR &lt;sub&gt;X&lt;/sub&gt; iv</strong> <div class="csl-block">Hiroyuki Tsunoda, Yuan Sun, Masaki Nishizawa, Xiaomin Liu, Kou Amano</div> <em>Proceedings of the Association for Information Science and Technology</em> (2019-10-18) <a href="https://doi.org/ggz7f9">https://doi.org/ggz7f9</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1002/pra2.175">10.1002/pra2.175</a></div></div>
</div>
<div id="ref-13dwNbYs7" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">15. </div><div class="csl-right-inline"><strong>The relationship between bioRxiv preprints, citations and altmetrics</strong> <div class="csl-block">Nicholas Fraser, Fakhri Momeni, Philipp Mayr, Isabella Peters</div> <em>Quantitative Science Studies</em> (2020-04-01) <a href="https://doi.org/gg2cz3">https://doi.org/gg2cz3</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1162/qss_a_00043">10.1162/qss_a_00043</a></div></div>
</div>
<div id="ref-49GeQtua" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">16. </div><div class="csl-right-inline"><strong>Releasing a preprint is associated with more attention and citations for the peer-reviewed article</strong> <div class="csl-block">Darwin Y Fu, Jacob J Hughey</div> <em>eLife</em> (2019-12-06) <a href="https://doi.org/ghd3mv">https://doi.org/ghd3mv</a> <div class="csl-block">DOI: <a href="https://doi.org/10.7554/elife.52646">10.7554/elife.52646</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/31808742">31808742</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6914335">PMC6914335</a></div></div>
</div>
<div id="ref-WaEbLm5M" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">17. </div><div class="csl-right-inline"><strong>Preprints and Scholarly Communication: An Exploratory Qualitative Study of Adoption, Practices, Drivers and Barriers</strong> <div class="csl-block">Andrea Chiarelli, Rob Johnson, Stephen Pinfield, Emma Richens</div> <em>F1000Research</em> (2019-11-25) <a href="https://doi.org/ghp38z">https://doi.org/ghp38z</a> <div class="csl-block">DOI: <a href="https://doi.org/10.12688/f1000research.19619.2">10.12688/f1000research.19619.2</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32055396">32055396</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6961415">PMC6961415</a></div></div>
</div>
<div id="ref-tTgEr7g1" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">18. </div><div class="csl-right-inline"><strong>The Need for Speed: How Quickly Do Preprints Become Published Articles?</strong> <div class="csl-block">Rachel Herbert, Kate Gasson, Alex Ponsford</div> <em>SSRN Electronic Journal</em> (2019) <a href="https://doi.org/ghd3mt">https://doi.org/ghd3mt</a> <div class="csl-block">DOI: <a href="https://doi.org/10.2139/ssrn.3455146">10.2139/ssrn.3455146</a></div></div>
</div>
<div id="ref-TYlEZ3Dw" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">19. </div><div class="csl-right-inline"><strong>Technical and social issues influencing the adoption of preprints in the life sciences</strong> <div class="csl-block">Naomi C Penfold, Jessica K Polka</div> <em>PLOS Genetics</em> (2020-04-20) <a href="https://doi.org/dtt2">https://doi.org/dtt2</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1371/journal.pgen.1008565">10.1371/journal.pgen.1008565</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32310942">32310942</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7170218">PMC7170218</a></div></div>
</div>
<div id="ref-PmF48i62" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">20. </div><div class="csl-right-inline"><strong>Biologists urged to hug a preprint</strong> <div class="csl-block">Ewen Callaway, Kendall Powell</div> <em>Nature</em> (2016-02-16) <a href="https://doi.org/ghdd62">https://doi.org/ghdd62</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1038/530265a">10.1038/530265a</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/26887471">26887471</a></div></div>
</div>
<div id="ref-vW9wl0tz" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">21. </div><div class="csl-right-inline"><strong>Day-to-day discovery of preprint–publication links</strong> <div class="csl-block">Guillaume Cabanac, Theodora Oikonomidi, Isabelle Boutron</div> <em>Scientometrics</em> (2021-04-18) <a href="https://doi.org/gjr9k4">https://doi.org/gjr9k4</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1007/s11192-021-03900-7">10.1007/s11192-021-03900-7</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/33897069">33897069</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC8053368">PMC8053368</a></div></div>
</div>
<div id="ref-jXP4z24U" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">22. </div><div class="csl-right-inline"><strong>On the value of preprints: An early career researcher perspective</strong> <div class="csl-block">Sarvenaz Sarabipour, Humberto J Debat, Edward Emmott, Steven J Burgess, Benjamin Schwessinger, Zach Hensel</div> <em>PLOS Biology</em> (2019-02-21) <a href="https://doi.org/gfw8hd">https://doi.org/gfw8hd</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1371/journal.pbio.3000151">10.1371/journal.pbio.3000151</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/30789895">30789895</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6400415">PMC6400415</a></div></div>
</div>
<div id="ref-2aMrUMfz" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">23. </div><div class="csl-right-inline"><strong>Prepublication Communication of Research Results</strong> <div class="csl-block">Michael J Adams, Reid N Harris, Evan HC Grant, Matthew J Gray, M Camille Hopkins, Samuel A Iverson, Robert Likens, Mark Mandica, Deanna H Olson, Alex Shepack, Hardin Waddle</div> <em>EcoHealth</em> (2018-08-07) <a href="https://doi.org/ghn66s">https://doi.org/ghn66s</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1007/s10393-018-1352-3">10.1007/s10393-018-1352-3</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/30088185">30088185</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6245104">PMC6245104</a></div></div>
</div>
<div id="ref-1EwmaUwf9" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">24. </div><div class="csl-right-inline"><strong>Peer Review and bioRxiv</strong> <div class="csl-block">Leslie M Loew</div> <em>Biophysical Journal</em> (2016-08) <a href="https://doi.org/ghdd6x">https://doi.org/ghdd6x</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1016/j.bpj.2016.06.035">10.1016/j.bpj.2016.06.035</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/27508451">27508451</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4982934">PMC4982934</a></div></div>
</div>
<div id="ref-DgTUHUNP" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">25. </div><div class="csl-right-inline"><strong>Preprints in motion: tracking changes between preprint posting and journal publication during a pandemic</strong> <div class="csl-block">Liam Brierley, Federico Nanni, Jessica K Polka, Gautam Dey, Máté Pálfy, Nicholas Fraser, Jonathon Alexis Coates</div> <em>Cold Spring Harbor Laboratory</em> (2021-10-14) <a href="https://doi.org/gh5mhm">https://doi.org/gh5mhm</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1101/2021.02.20.432090">10.1101/2021.02.20.432090</a></div></div>
</div>
<div id="ref-Ibrx6ppY" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">26. </div><div class="csl-right-inline"><strong>Textual Analysis in Accounting and Finance: A Survey</strong> <div class="csl-block">TIM LOUGHRAN, BILL MCDONALD</div> <em>Journal of Accounting Research</em> (2016-09) <a href="https://doi.org/gc3hf7">https://doi.org/gc3hf7</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1111/1475-679x.12123">10.1111/1475-679x.12123</a></div></div>
</div>
<div id="ref-nmSCIhkg" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">27. </div><div class="csl-right-inline"><strong>SciReader: A Cloud-based Recommender System for Biomedical Literature</strong> <div class="csl-block">Priya Desai, Natalie Telis, Ben Lehmann, Keith Bettinger, Jonathan K Pritchard, Somalee Datta</div> <em>Cold Spring Harbor Laboratory</em> (2018-05-30) <a href="https://doi.org/gkw2zw">https://doi.org/gkw2zw</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1101/333922">10.1101/333922</a></div></div>
</div>
<div id="ref-1H7sXN2v7" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">28. </div><div class="csl-right-inline"><strong>The textual characteristics of traditional and Open Access scientific journals are similar</strong> <div class="csl-block">Karin Verspoor, KBretonnel Cohen, Lawrence Hunter</div> <em>BMC Bioinformatics</em> (2009-06-15) <a href="https://doi.org/b973tn">https://doi.org/b973tn</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1186/1471-2105-10-183">10.1186/1471-2105-10-183</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/19527520">19527520</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2714574">PMC2714574</a></div></div>
</div>
<div id="ref-1Ec63crER" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">29. </div><div class="csl-right-inline"><strong>Current findings from research on structured abstracts</strong> <div class="csl-block">James Hartley</div> <em>Journal of the Medical Library Association : JMLA</em> (2004-07) <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC442180/">https://www.ncbi.nlm.nih.gov/pmc/articles/PMC442180/</a> <div class="csl-block">PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/15243644">15243644</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC442180">PMC442180</a></div></div>
</div>
<div id="ref-qGYQ9hj7" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">30. </div><div class="csl-right-inline"><strong>A survey on annotation tools for the biomedical literature</strong> <div class="csl-block">M Neves, U Leser</div> <em>Briefings in Bioinformatics</em> (2012-12-18) <a href="https://doi.org/f5vzsj">https://doi.org/f5vzsj</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1093/bib/bbs084">10.1093/bib/bbs084</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/23255168">23255168</a></div></div>
</div>
<div id="ref-18ZyyTcTe" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">31. </div><div class="csl-right-inline"><strong>PubTator central: automated concept annotation for biomedical full text articles</strong> <div class="csl-block">Chih-Hsuan Wei, Alexis Allot, Robert Leaman, Zhiyong Lu</div> <em>Nucleic Acids Research</em> (2019-07-02) <a href="https://doi.org/ggzfsc">https://doi.org/ggzfsc</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1093/nar/gkz389">10.1093/nar/gkz389</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/31114887">31114887</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6602571">PMC6602571</a></div></div>
</div>
<div id="ref-XXeZHguV" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">32. </div><div class="csl-right-inline"><strong>Coreference annotation and resolution in the Colorado Richly Annotated Full Text (CRAFT) corpus of biomedical journal articles</strong> <div class="csl-block">KBretonnel Cohen, Arrick Lanfranchi, Miji Joo-young Choi, Michael Bada, William A Baumgartner, Natalya Panteleyeva, Karin Verspoor, Martha Palmer, Lawrence E Hunter</div> <em>BMC Bioinformatics</em> (2017-08-17) <a href="https://doi.org/ghmbw2">https://doi.org/ghmbw2</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1186/s12859-017-1775-9">10.1186/s12859-017-1775-9</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/28818042">28818042</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5561560">PMC5561560</a></div></div>
</div>
<div id="ref-vU15xGGa" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">33. </div><div class="csl-right-inline"><strong>The structural and content aspects of abstracts versus bodies of full text journal articles are different</strong> <div class="csl-block">KBretonnel Cohen, Helen L Johnson, Karin Verspoor, Christophe Roeder, Lawrence E Hunter</div> <em>BMC Bioinformatics</em> (2010-09-29) <a href="https://doi.org/b9f6rn">https://doi.org/b9f6rn</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1186/1471-2105-11-492">10.1186/1471-2105-11-492</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/20920264">20920264</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3098079">PMC3098079</a></div></div>
</div>
<div id="ref-1CW99irQj" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">34. </div><div class="csl-right-inline"><strong>A corpus of full-text journal articles is a robust evaluation tool for revealing differences in performance of biomedical natural language processing tools</strong> <div class="csl-block">Karin Verspoor, Kevin Bretonnel Cohen, Arrick Lanfranchi, Colin Warner, Helen L Johnson, Christophe Roeder, Jinho D Choi, Christopher Funk, Yuriy Malenkiy, Miriam Eckert, … Lawrence E Hunter</div> <em>BMC Bioinformatics</em> (2012-08-17) <a href="https://doi.org/gb8t7v">https://doi.org/gb8t7v</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1186/1471-2105-13-207">10.1186/1471-2105-13-207</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/22901054">22901054</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3483229">PMC3483229</a></div></div>
</div>
<div id="ref-YhMeTgyl" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">35. </div><div class="csl-right-inline"><strong>From POS tagging to dependency parsing for biomedical event extraction</strong> <div class="csl-block">Dat Quoc Nguyen, Karin Verspoor</div> <em>BMC Bioinformatics</em> (2019-02-12) <a href="https://doi.org/ggsrkw">https://doi.org/ggsrkw</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1186/s12859-019-2604-0">10.1186/s12859-019-2604-0</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/30755172">30755172</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6373122">PMC6373122</a></div></div>
</div>
<div id="ref-1GhHIDxuW" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">36. </div><div class="csl-right-inline"><strong>Efficient Estimation of Word Representations in Vector Space</strong> <div class="csl-block">Tomas Mikolov, Kai Chen, Greg Corrado, Jeffrey Dean</div> <em>arXiv</em> (2013-09-10) <a href="https://arxiv.org/abs/1301.3781">https://arxiv.org/abs/1301.3781</a></div>
</div>
<div id="ref-SdcIDSwR" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">37. </div><div class="csl-right-inline"><strong>Distributed Representations of Sentences and Documents</strong> <div class="csl-block">Quoc V Le, Tomas Mikolov</div> <em>arXiv</em> (2014-05-26) <a href="https://arxiv.org/abs/1405.4053">https://arxiv.org/abs/1405.4053</a></div>
</div>
<div id="ref-142YFczMe" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">38. </div><div class="csl-right-inline"><strong>BioRxiv Machine access and text/data mining resources</strong> <a href="https://www.biorxiv.org/tdm">https://www.biorxiv.org/tdm</a></div>
</div>
<div id="ref-u294RvPz" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">39. </div><div class="csl-right-inline"><strong>PubMed Central: The GenBank of the published literature</strong> <div class="csl-block">RJ Roberts</div> <em>Proceedings of the National Academy of Sciences</em> (2001-01-16) <a href="https://doi.org/bbn9k8">https://doi.org/bbn9k8</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1073/pnas.98.2.381">10.1073/pnas.98.2.381</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/11209037">11209037</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC33354">PMC33354</a></div></div>
</div>
<div id="ref-lBh7pyRc" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">40. </div><div class="csl-right-inline"><strong>How Papers Get Into PMC</strong> <a href="https://www.ncbi.nlm.nih.gov/pmc/about/submission-methods/">https://www.ncbi.nlm.nih.gov/pmc/about/submission-methods/</a></div>
</div>
<div id="ref-iyblQx2M" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">41. </div><div class="csl-right-inline"><strong>Gold open access: the best of both worlds</strong> <div class="csl-block">MAG van der Heyden, TAB van Veen</div> <em>Netherlands Heart Journal</em> (2017-12-01) <a href="https://doi.org/ggzfr9">https://doi.org/ggzfr9</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1007/s12471-017-1064-2">10.1007/s12471-017-1064-2</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/29196877">29196877</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5758455">PMC5758455</a></div></div>
</div>
<div id="ref-vGaAJBqg" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">42. </div><div class="csl-right-inline"><strong>8.2.2 NIH Public Access Policy</strong> <a href="https://grants.nih.gov/grants/policy/nihgps/html5/section_8/8.2.2_nih_public_access_policy.htm">https://grants.nih.gov/grants/policy/nihgps/html5/section_8/8.2.2_nih_public_access_policy.htm</a></div>
</div>
<div id="ref-rzmpIYmn" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">43. </div><div class="csl-right-inline"><strong>PMC Overview</strong> <a href="https://www.ncbi.nlm.nih.gov/pmc/about/intro/">https://www.ncbi.nlm.nih.gov/pmc/about/intro/</a></div>
</div>
<div id="ref-rURwqquJ" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">44. </div><div class="csl-right-inline"><strong>PMC text mining subset in BioC: about three million full-text articles and growing</strong> <div class="csl-block">Donald C Comeau, Chih-Hsuan Wei, Rezarta Islamaj Doğan, Zhiyong Lu</div> <em>Bioinformatics</em> (2019-09-15) <a href="https://doi.org/ggzfsb">https://doi.org/ggzfsb</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1093/bioinformatics/btz070">10.1093/bioinformatics/btz070</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/30715220">30715220</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6748740">PMC6748740</a></div></div>
</div>
<div id="ref-12qFW0Wyn" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">45. </div><div class="csl-right-inline"><strong>Author Manuscripts in PMC</strong> <a href="https://www.ncbi.nlm.nih.gov/pmc/about/authorms/">https://www.ncbi.nlm.nih.gov/pmc/about/authorms/</a></div>
</div>
<div id="ref-18JcY2okY" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">46. </div><div class="csl-right-inline"><strong>The new york times annotated corpus</strong> <div class="csl-block">Evan Sandhaus</div> <em>Linguistic Data Consortium, Philadelphia</em> (2008)</div>
</div>
<div id="ref-XnHb341s" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">47. </div><div class="csl-right-inline"><strong>CrossRef Text and Data Mining Services</strong> <div class="csl-block">Rachael Lammey</div> <em>Insights the UKSG journal</em> (2015-07-07) <a href="https://doi.org/gg4hp9">https://doi.org/gg4hp9</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1629/uksg.233">10.1629/uksg.233</a></div></div>
</div>
<div id="ref-q2fFAZTG" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">48. </div><div class="csl-right-inline"><strong>spaCy 2: Natural language understanding with Bloom embeddings, convolutional neural networks and incremental parsing</strong> <div class="csl-block">Matthew Honnibal, Ines Montani</div> (2017)</div>
</div>
<div id="ref-Lgm5q6mO" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">49. </div><div class="csl-right-inline"><strong>Odds Ratio</strong> <div class="csl-block">Steven Tenny, Mary R Hoffman</div> <em>StatPearls</em> (2021) <a href="http://www.ncbi.nlm.nih.gov/books/NBK431098/">http://www.ncbi.nlm.nih.gov/books/NBK431098/</a></div>
</div>
<div id="ref-HUWvSbrt" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">50. </div><div class="csl-right-inline"><strong>Gensim–python framework for vector space modelling</strong> <div class="csl-block">Radim Rehurek, Petr Sojka</div> <em>NLP Centre, Faculty of Informatics, Masaryk University, Brno, Czech Republic</em> (2011)</div>
</div>
<div id="ref-Km2ytzVD" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">51. </div><div class="csl-right-inline"><strong>On the Dimensionality of Word Embedding</strong> <div class="csl-block">Zi Yin, Yuanyuan Shen</div> <em>arXiv</em> (2018-12-12) <a href="https://arxiv.org/abs/1812.04224">https://arxiv.org/abs/1812.04224</a></div>
</div>
<div id="ref-qkjVrvqt" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">52. </div><div class="csl-right-inline"><strong>Probabilistic Principal Component Analysis</strong> <div class="csl-block">Michael E Tipping, Christopher M Bishop</div> <em>Journal of the Royal Statistical Society: Series B (Statistical Methodology)</em> (1999-08) <a href="https://doi.org/b3hjwt">https://doi.org/b3hjwt</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1111/1467-9868.00196">10.1111/1467-9868.00196</a></div></div>
</div>
<div id="ref-vHIKEjNs" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">53. </div><div class="csl-right-inline"><strong>Scikit-learn: Machine Learning in Python</strong> <div class="csl-block">Fabian Pedregosa, Gaël Varoquaux, Alexandre Gramfort, Vincent Michel, Bertrand Thirion, Olivier Grisel, Mathieu Blondel, Andreas Müller, Joel Nothman, Gilles Louppe, … Édouard Duchesnay</div> <em>arXiv</em> (2018-06-06) <a href="https://arxiv.org/abs/1201.0490">https://arxiv.org/abs/1201.0490</a></div>
</div>
<div id="ref-7tMTyalf" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">54. </div><div class="csl-right-inline"><strong>Finding structure with randomness: Probabilistic algorithms for constructing approximate matrix decompositions</strong> <div class="csl-block">Nathan Halko, Per-Gunnar Martinsson, Joel A Tropp</div> <em>arXiv</em> (2014-04-29) <a href="https://arxiv.org/abs/0909.4061">https://arxiv.org/abs/0909.4061</a></div>
</div>
<div id="ref-1Bpwk7s3x" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">55. </div><div class="csl-right-inline"><strong>The &lt;i&gt;Drosophila&lt;/i&gt; Cortactin Binding Protein 2 homolog, Nausicaa, regulates lamellipodial actin dynamics in a Cortactin-dependent manner</strong> <div class="csl-block">Meghan E O’Connell, Divya Sridharan, Tristan Driscoll, Ipsita Krishnamurthy, Wick G Perry, Derek A Applewhite</div> <em>Cold Spring Harbor Laboratory</em> (2018-07-24) <a href="https://doi.org/gg4hp7">https://doi.org/gg4hp7</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1101/376665">10.1101/376665</a></div></div>
</div>
<div id="ref-iBFkqblV" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">56. </div><div class="csl-right-inline"><strong>The &lt;i&gt;Drosophila&lt;/i&gt; protein, Nausicaa, regulates lamellipodial actin dynamics in a Cortactin-dependent manner</strong> <div class="csl-block">Meghan E O'Connell, Divya Sridharan, Tristan Driscoll, Ipsita Krishnamurthy, Wick G Perry, Derek A Applewhite</div> <em>Biology Open</em> (2019-01-01) <a href="https://doi.org/gg4hp8">https://doi.org/gg4hp8</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1242/bio.038232">10.1242/bio.038232</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/31164339">31164339</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6602326">PMC6602326</a></div></div>
</div>
<div id="ref-PQlBxj25" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">57. </div><div class="csl-right-inline"><strong>Understanding survival analysis: Kaplan-Meier estimate</strong> <div class="csl-block">Jugal Kishore, ManishKumar Goel, Pardeep Khanna</div> <em>International Journal of Ayurveda Research</em> (2010) <a href="https://doi.org/fdft75">https://doi.org/fdft75</a> <div class="csl-block">DOI: <a href="https://doi.org/10.4103/0974-7788.76794">10.4103/0974-7788.76794</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/21455458">21455458</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3059453">PMC3059453</a></div></div>
</div>
<div id="ref-4TsSA7hy" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">58. </div><div class="csl-right-inline"><strong>CamDavidsonPilon/lifelines: v0.25.6</strong> <div class="csl-block">Cameron Davidson-Pilon, Jonas Kalderstam, Noah Jacobson, Sean-Reed, Ben Kuhn, Paul Zivich, Mike Williamson, AbdealiJK, Deepyaman Datta, Andrew Fiore-Gartland, … Jlim13</div> <em>Zenodo</em> (2020-10-26) <a href="https://doi.org/ghh2d3">https://doi.org/ghh2d3</a> <div class="csl-block">DOI: <a href="https://doi.org/10.5281/zenodo.4136578">10.5281/zenodo.4136578</a></div></div>
</div>
<div id="ref-bN8m3HfR" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">59. </div><div class="csl-right-inline"><strong>The bioRxiv Wall of Shame</strong> <div class="csl-block">Jordan Anaya</div> (2018-08-03) <a href="https://medium.com/@OmnesRes/the-biorxiv-wall-of-shame-aa3d9cfc4cd7">https://medium.com/@OmnesRes/the-biorxiv-wall-of-shame-aa3d9cfc4cd7</a></div>
</div>
<div id="ref-mZvqlKk6" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">60. </div><div class="csl-right-inline"><strong>Journal/Author Name Estimator (JANE)</strong> <div class="csl-block">Carolann Lee Curry</div> <em>Journal of the Medical Library Association</em> (2019-01-04) <a href="https://doi.org/ghjw7j">https://doi.org/ghjw7j</a> <div class="csl-block">DOI: <a href="https://doi.org/10.5195/jmla.2019.598">10.5195/jmla.2019.598</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6300233">PMC6300233</a></div></div>
</div>
<div id="ref-icndlpmW" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">61. </div><div class="csl-right-inline"><strong>Introduction — PyMuPDF 1.19.1 documentation</strong> <a href="https://pymupdf.readthedocs.io/en/latest/intro.html">https://pymupdf.readthedocs.io/en/latest/intro.html</a></div>
</div>
<div id="ref-LxhnWcbN" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">62. </div><div class="csl-right-inline"><strong>Assessing the Heterogeneity of Cardiac Non-myocytes and the Effect of Cell Culture with Integrative Single Cell Analysis</strong> <div class="csl-block">Brian S Iskra, Logan Davis, Henry E Miller, Yu-Chiao Chiu, Alexander R Bishop, Yidong Chen, Gregory J Aune</div> <em>Cold Spring Harbor Laboratory</em> (2020-03-05) <a href="https://doi.org/gg9353">https://doi.org/gg9353</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1101/2020.03.04.975177">10.1101/2020.03.04.975177</a></div></div>
</div>
<div id="ref-f22ESynO" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">63. </div><div class="csl-right-inline"><strong>Preprinting the COVID-19 pandemic</strong> <div class="csl-block">Nicholas Fraser, Liam Brierley, Gautam Dey, Jessica K Polka, Máté Pálfy, Federico Nanni, Jonathon Alexis Coates</div> <em>Cold Spring Harbor Laboratory</em> (2021-02-05) <a href="https://doi.org/dxdb">https://doi.org/dxdb</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1101/2020.05.22.111294">10.1101/2020.05.22.111294</a></div></div>
</div>
<div id="ref-KyVRyag" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">64. </div><div class="csl-right-inline"><strong>PMCID - PMID - Manuscript ID - DOI Converter</strong> <a href="https://www.ncbi.nlm.nih.gov/pmc/pmctopmid/">https://www.ncbi.nlm.nih.gov/pmc/pmctopmid/</a></div>
</div>
<div id="ref-mTE5h91z" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">65. </div><div class="csl-right-inline"><strong>Altmetric Scores, Citations, and Publication of Studies Posted as Preprints</strong> <div class="csl-block">Stylianos Serghiou, John PA Ioannidis</div> <em>JAMA</em> (2018-01-23) <a href="https://doi.org/gftc69">https://doi.org/gftc69</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1001/jama.2017.21168">10.1001/jama.2017.21168</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/29362788">29362788</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5833561">PMC5833561</a></div></div>
</div>
<div id="ref-TMxQYqVE" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">66. </div><div class="csl-right-inline"><strong>Peer review and the publication process</strong> <div class="csl-block">Parveen Azam Ali, Roger Watson</div> <em>Nursing Open</em> (2016-03-16) <a href="https://doi.org/c4g8">https://doi.org/c4g8</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1002/nop2.51">10.1002/nop2.51</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/27708830">27708830</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5050543">PMC5050543</a></div></div>
</div>
<div id="ref-SkPpj3hW" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">67. </div><div class="csl-right-inline"><strong>BioBERT: a pre-trained biomedical language representation model for biomedical text mining</strong> <div class="csl-block">Jinhyuk Lee, Wonjin Yoon, Sungdong Kim, Donghyeon Kim, Sunkyu Kim, Chan Ho So, Jaewoo Kang</div> <em>Bioinformatics</em> (2019-09-10) <a href="https://doi.org/ggh5qq">https://doi.org/ggh5qq</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1093/bioinformatics/btz682">10.1093/bioinformatics/btz682</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/31501885">31501885</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7703786">PMC7703786</a></div></div>
</div>
<div id="ref-kPI0iXkU" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">68. </div><div class="csl-right-inline"><strong>Unsupervised word embeddings capture latent knowledge from materials science literature</strong> <div class="csl-block">Vahe Tshitoyan, John Dagdelen, Leigh Weston, Alexander Dunn, Ziqin Rong, Olga Kononova, Kristin A Persson, Gerbrand Ceder, Anubhav Jain</div> <em>Nature</em> (2019-07-03) <a href="https://doi.org/gf4kk5">https://doi.org/gf4kk5</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1038/s41586-019-1335-8">10.1038/s41586-019-1335-8</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/31270483">31270483</a></div></div>
</div>
<div id="ref-17IHBAq05" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">69. </div><div class="csl-right-inline"><strong>Efficient Vector Representation for Documents through Corruption</strong> <div class="csl-block">Minmin Chen</div> <em>arXiv</em> (2017-07-11) <a href="https://arxiv.org/abs/1707.02377">https://arxiv.org/abs/1707.02377</a></div>
</div>
<div id="ref-14k28UnQN" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">70. </div><div class="csl-right-inline"><strong>Document Network Projection in Pretrained Word Embedding Space</strong> <div class="csl-block">Antoine Gourru, Adrien Guille, Julien Velcin, Julien Jacques</div> <em>arXiv</em> (2020-01-17) <a href="https://arxiv.org/abs/2001.05727">https://arxiv.org/abs/2001.05727</a></div>
</div>
<div id="ref-ddvF5sXo" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">71. </div><div class="csl-right-inline"><strong>Conditional Robust Calibration (CRC): a new computational Bayesian methodology for model parameters estimation and identifiability analysis</strong> <div class="csl-block">Fortunato Bianconi, Chiara Antonini, Lorenzo Tomassoni, Paolo Valigi</div> <em>Cold Spring Harbor Laboratory</em> (2017-10-02) <a href="https://doi.org/gg9393">https://doi.org/gg9393</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1101/197400">10.1101/197400</a></div></div>
</div>
<div id="ref-18dTP7Wfc" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">72. </div><div class="csl-right-inline"><strong>Machine learning of stochastic gene network phenotypes</strong> <div class="csl-block">Kyemyung Park, Thorsten Prüstel, Yong Lu, John S Tsang</div> <em>Cold Spring Harbor Laboratory</em> (2019-10-31) <a href="https://doi.org/gg94bm">https://doi.org/gg94bm</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1101/825943">10.1101/825943</a></div></div>
</div>
<div id="ref-UmFNlo8W" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">73. </div><div class="csl-right-inline"><strong>Notions of similarity for computational biology models</strong> <div class="csl-block">Ron Henkel, Robert Hoehndorf, Tim Kacprowski, Christian Knüpfer, Wolfram Liebermeister, Dagmar Waltemath</div> <em>Cold Spring Harbor Laboratory</em> (2016-03-21) <a href="https://doi.org/gg939z">https://doi.org/gg939z</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1101/044818">10.1101/044818</a></div></div>
</div>
<div id="ref-16tWubqOj" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">74. </div><div class="csl-right-inline"><strong>GpABC: a Julia package for approximate Bayesian computation with Gaussian process emulation</strong> <div class="csl-block">Evgeny Tankhilevich, Jonathan Ish-Horowicz, Tara Hameed, Elisabeth Roesch, Istvan Kleijn, Michael PH Stumpf, Fei He</div> <em>Cold Spring Harbor Laboratory</em> (2019-09-18) <a href="https://doi.org/gg94bj">https://doi.org/gg94bj</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1101/769299">10.1101/769299</a></div></div>
</div>
<div id="ref-6gfAR2hM" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">75. </div><div class="csl-right-inline"><strong>SBpipe: a collection of pipelines for automating repetitive simulation and analysis tasks</strong> <div class="csl-block">Piero Dalle Pezze, Nicolas Le Novère</div> <em>Cold Spring Harbor Laboratory</em> (2017-02-09) <a href="https://doi.org/gg9392">https://doi.org/gg9392</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1101/107250">10.1101/107250</a></div></div>
</div>
<div id="ref-2V2BJL87" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">76. </div><div class="csl-right-inline"><strong>Spatiotemporal proteomics uncovers cathepsin-dependent host cell death during bacterial infection</strong> <div class="csl-block">Joel Selkrig, Nan Li, Jacob Bobonis, Annika Hausmann, Anna Sueki, Haruna Imamura, Bachir El Debs, Gianluca Sigismondo, Bogdan I Florea, Herman S Overkleeft, … Athanasios Typas</div> <em>Cold Spring Harbor Laboratory</em> (2018-11-07) <a href="https://doi.org/gg94bc">https://doi.org/gg94bc</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1101/455048">10.1101/455048</a></div></div>
</div>
<div id="ref-4BUSt1jK" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">77. </div><div class="csl-right-inline"><strong>Systems analysis by mass cytometry identifies susceptibility of latent HIV-infected T cells to targeting of p38 and mTOR pathways</strong> <div class="csl-block">Linda E Fong, Victor L Bass, Serena Spudich, Kathryn Miller-Jensen</div> <em>Cold Spring Harbor Laboratory</em> (2018-07-19) <a href="https://doi.org/gg9398">https://doi.org/gg9398</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1101/371922">10.1101/371922</a></div></div>
</div>
<div id="ref-nzDb3Lvn" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">78. </div><div class="csl-right-inline"><strong>NADPH consumption by L-cystine reduction creates a metabolic vulnerability upon glucose deprivation</strong> <div class="csl-block">James H Joly, Alireza Delfarah, Philip S Phung, Sydney Parrish, Nicholas A Graham</div> <em>Cold Spring Harbor Laboratory</em> (2019-08-13) <a href="https://doi.org/gg94bf">https://doi.org/gg94bf</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1101/733162">10.1101/733162</a></div></div>
</div>
<div id="ref-1CM9H3mFb" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">79. </div><div class="csl-right-inline"><strong>Inhibition of Bruton’s tyrosine kinase reduces NF-kB and NLRP3 inflammasome activity preventing insulin resistance and microvascular disease</strong> <div class="csl-block">Gareth SD Purvis, Massimo Collino, Haidee MA Tavio, Fausto Chiazza, Caroline E O’Riodan, Lynda Zeboudj, Nick Guisot, Peter Bunyard, David R Greaves, Christoph Thiemermann</div> <em>Cold Spring Harbor Laboratory</em> (2019-08-28) <a href="https://doi.org/gg94bg">https://doi.org/gg94bg</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1101/745943">10.1101/745943</a></div></div>
</div>
<div id="ref-jmFU2eta" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">80. </div><div class="csl-right-inline"><strong>AKT but not MYC promotes reactive oxygen species-mediated cell death in oxidative culture</strong> <div class="csl-block">Dongqing Zheng, Jonathan H Sussman, Matthew P Jeon, Sydney T Parrish, Alireza Delfarah, Nicholas A Graham</div> <em>Cold Spring Harbor Laboratory</em> (2019-09-01) <a href="https://doi.org/gg94bh">https://doi.org/gg94bh</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1101/754572">10.1101/754572</a></div></div>
</div>
<div id="ref-Maazaz8h" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">81. </div><div class="csl-right-inline"><strong>&lt;i&gt;FPtool&lt;/i&gt; a software tool to obtain &lt;i&gt;in silico&lt;/i&gt; genotype-phenotype signatures and fingerprints based on massive model simulations</strong> <div class="csl-block">Guido Santos, Julio Vera</div> <em>Cold Spring Harbor Laboratory</em> (2018-02-18) <a href="https://doi.org/gjr9m9">https://doi.org/gjr9m9</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1101/266775">10.1101/266775</a></div></div>
</div>
<div id="ref-NIw2ZKUa" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">82. </div><div class="csl-right-inline"><strong>Bromodomain inhibition reveals FGF15/19 as a target of epigenetic regulation and metabolic control</strong> <div class="csl-block">Chisayo Kozuka, Vicencia Sales, Soravis Osataphan, Yixing Yuchi, Jeremy Chimene-Weiss, Christopher Mulla, Elvira Isganaitis, Jessica Desmond, Suzuka Sanechika, Joji Kusuyama, … Mary-Elizabeth Patti</div> <em>Cold Spring Harbor Laboratory</em> (2019-12-12) <a href="https://doi.org/gjr9m8">https://doi.org/gjr9m8</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1101/2019.12.11.872887">10.1101/2019.12.11.872887</a></div></div>
</div>
</div>
<h2 id="supplemental-section">Supplemental Section</h2>
<h3 id="document-embeddings-derived-from-biorxiv-reveal-fields-and-subfields">Document embeddings derived from bioRxiv reveal fields and subfields</h3>
<div id="fig:topic_analysis_panels" class="fignos">
<figure>
<img src="https://raw.githubusercontent.com/danich1/annorxiver/65ee4a556ab69f2308e5e4d9192905e8cfec3728/figure_generation/output/Figure_S1.png" data-tag="S1" style="width:100.0%" alt="Figure S1: A. Principal components (PC) analysis of bioRxiv word2vec embeddings groups documents based on author-selected categories. We visualized documents from key categories on a scatterplot for the first two PCs. The first PC separated cell biology from informatics-related fields, and the second PC separated bioinformatics from neuroscience fields. B. A word cloud visualization of PC1. Each word cloud depicts the cosine similarity score between tokens and the first PC. Tokens in orange were most similar to the PC’s positive direction, while tokens in blue were most similar to the PC’s negative direction. The size of each token indicates the magnitude of the similarity. C. A word cloud visualization of PC2, which separated bioinformatics from neuroscience. Similar to the first PC, tokens in orange were most similar to the PC’s positive direction, while tokens in blue were most similar to the PC’s negative direction. The size of each token indicates the magnitude of the similarity. D. Examining PC1 values for each article by category created a continuum from informatics-related fields on the top through cell biology on the bottom. Specific article categories (neuroscience, genetics) were spread throughout PC1 values. E. Examining PC2 values for each article by category revealed fields like genomics, bioinformatics, and genetics on the top and neuroscience and behavior on the bottom. Data for the information depicted in this figure are available at https://github.com/greenelab/annorxiver/blob/master/FIGURE_DATA_SOURCE.md#figure-s1." /><figcaption aria-hidden="true"><span>Figure S1:</span> <strong>A.</strong> Principal components (PC) analysis of bioRxiv word2vec embeddings groups documents based on author-selected categories.
We visualized documents from key categories on a scatterplot for the first two PCs.
The first PC separated cell biology from informatics-related fields, and the second PC separated bioinformatics from neuroscience fields.
<strong>B.</strong> A word cloud visualization of PC1.
Each word cloud depicts the cosine similarity score between tokens and the first PC.
Tokens in orange were most similar to the PC’s positive direction, while tokens in blue were most similar to the PC’s negative direction.
The size of each token indicates the magnitude of the similarity.
<strong>C.</strong> A word cloud visualization of PC2, which separated bioinformatics from neuroscience.
Similar to the first PC, tokens in orange were most similar to the PC’s positive direction, while tokens in blue were most similar to the PC’s negative direction.
The size of each token indicates the magnitude of the similarity.
<strong>D.</strong> Examining PC1 values for each article by category created a continuum from informatics-related fields on the top through cell biology on the bottom.
Specific article categories (neuroscience, genetics) were spread throughout PC1 values.
<strong>E.</strong> Examining PC2 values for each article by category revealed fields like genomics, bioinformatics, and genetics on the top and neuroscience and behavior on the bottom.
Data for the information depicted in this figure are available at <a href="https://github.com/greenelab/annorxiver/blob/master/FIGURE_DATA_SOURCE.md#figure-s1">https://github.com/greenelab/annorxiver/blob/master/FIGURE_DATA_SOURCE.md#figure-s1</a>.</figcaption>
</figure>
</div>
<p>Document embeddings provide a means to categorize the language of documents in a way that takes into account the similarities between terms <span class="citation" data-cites="SdcIDSwR 17IHBAq05 14k28UnQN">[<a href="#ref-SdcIDSwR" role="doc-biblioref">37</a>,<a href="#ref-17IHBAq05" role="doc-biblioref">69</a>,<a href="#ref-14k28UnQN" role="doc-biblioref">70</a>]</span>.
We found that the first two PCs separated articles from different author-selected categories (Supplementary Figure <a href="#fig:topic_analysis_panels">S1</a>A).
Certain neuroscience papers appeared to be more associated with the cellular biology direction of PC1, while others seemed to be more associated with the informatics-related direction (Supplementary Figure <a href="#fig:topic_analysis_panels">S1</a>A).
This suggests that the concepts captured by PCs were not exclusively related to their field.</p>
<p>Visualizing token-PC similarity revealed tokens associated with certain research approaches (Supplementary Figures <a href="#fig:topic_analysis_panels">S1</a>B and <a href="#fig:topic_analysis_panels">S1</a>C).
Token association of PC1 shows the separation of cell biology and informatics-related fields through tokens: “empirical”, “estimates” and “statistics” depicted in orange and “cultured” and “overexpressing” shown in blue (Supplementary Figure <a href="#fig:topic_analysis_panels">S1</a>B and Supplementary Table <a href="#tbl:pc1_loadings_table">S2</a>).
Association for PC2 shows the separation of bioinformatics and neuroscience via tokens: “genomic”, “genome” and “genomes” depicted in orange and “evoked”, “stimulus” and “stimulation” shown in blue (Supplementary Figure <a href="#fig:topic_analysis_panels">S1</a>C and Supplementary Table <a href="#tbl:pc2_loadings_table">S3</a>).</p>
<p>Examining the value for PC1 across all author-selected categories revealed an ordering of fields from cell biology to informatics-related disciplines (Supplementary Figure <a href="#fig:topic_analysis_panels">S1</a>D).
These results suggest that a primary driver of the variability within the language used in bioRxiv could be the divide between informatics and cell biology approaches.
A similar analysis for PC2 suggested that neuroscience and bioinformatics present a similar language continuum (Supplementary Figure <a href="#fig:topic_analysis_panels">S1</a>E).
This result supports the notion that bioRxiv contains an influx of neuroscience and bioinformatics-related research results.
For both of the top two PCs, the submitter-selected category of systems biology preprints was near the middle of the distribution and had a relatively large interquartile range when compared with other categories (Supplementary Figures <a href="#fig:topic_analysis_panels">S1</a>D and <a href="#fig:topic_analysis_panels">S1</a>E), suggesting that systems biology is a broader subfield containing both informatics and cellular biology approaches.</p>
<p>Examining the top five highest-scoring and bottom five lowest-scoring systems biology preprints along PC1 reinforces its dichotomous theme (Supplementary Table <a href="#tbl:five_pc1_table">S1</a>).
Preprints with the highest values <span class="citation" data-cites="ddvF5sXo 18dTP7Wfc UmFNlo8W 16tWubqOj 6gfAR2hM">[<a href="#ref-ddvF5sXo" role="doc-biblioref">71</a>,<a href="#ref-18dTP7Wfc" role="doc-biblioref">72</a>,<a href="#ref-UmFNlo8W" role="doc-biblioref">73</a>,<a href="#ref-16tWubqOj" role="doc-biblioref">74</a>,<a href="#ref-6gfAR2hM" role="doc-biblioref">75</a>]</span> included software packages, machine learning analyses, and other computational biology manuscripts, while preprints with the lowest values <span class="citation" data-cites="2V2BJL87 4BUSt1jK nzDb3Lvn 1CM9H3mFb jmFU2eta">[<a href="#ref-2V2BJL87" role="doc-biblioref">76</a>,<a href="#ref-4BUSt1jK" role="doc-biblioref">77</a>,<a href="#ref-nzDb3Lvn" role="doc-biblioref">78</a>,<a href="#ref-1CM9H3mFb" role="doc-biblioref">79</a>,<a href="#ref-jmFU2eta" role="doc-biblioref">80</a>]</span> were focused on cellular signaling and protein activity.
We provide the rest of our 50 generated PCs in our online repository (see Software and Data Availability).</p>
<div id="tbl:five_pc1_table" class="tablenos">
<table id="tbl:five_pc1_table" style="width:100%;" data-tag="S1">
<caption><span>Table S1:</span> PC1 divided the author-selected category of systems biology preprints along an axis from computational to molecular approaches. </caption>
<colgroup>
<col style="width: 43%" />
<col style="width: 26%" />
<col style="width: 16%" />
<col style="width: 13%" />
</colgroup>
<thead>
<tr class="header">
<th>Title [citation]</th>
<th>PC1</th>
<th>License</th>
<th>Figure Thumbnail</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td>Conditional Robust Calibration (CRC): a new computational Bayesian methodology for model parameters estimation and identifiability analysis <span class="citation" data-cites="ddvF5sXo">[<a href="#ref-ddvF5sXo" role="doc-biblioref">71</a>]</span></td>
<td>4.522818390064091</td>
<td>None</td>
<td><img src="images/paper-thumbnails/pc1/197400_thumbnail.png" title="fig:" /></td>
</tr>
<tr class="even">
<td>FPtool a software tool to obtain in silico genotype-phenotype signatures and fingerprints based on massive model simulations <span class="citation" data-cites="Maazaz8h">[<a href="#ref-Maazaz8h" role="doc-biblioref">81</a>]</span></td>
<td>4.348956760251298</td>
<td>CC-BY</td>
<td><img src="images/paper-thumbnails/pc1/266775_thumbnail.png" title="fig:" /></td>
</tr>
<tr class="odd">
<td>GpABC: a Julia package for approximate Bayesian computation with Gaussian process emulation <span class="citation" data-cites="16tWubqOj">[<a href="#ref-16tWubqOj" role="doc-biblioref">74</a>]</span></td>
<td>4.259104249060651</td>
<td>CC-BY-NC-ND</td>
<td><img src="images/paper-thumbnails/pc1/769299_thumbnail.png" title="fig:" /></td>
</tr>
<tr class="even">
<td>Notions of similarity for computational biology models <span class="citation" data-cites="UmFNlo8W">[<a href="#ref-UmFNlo8W" role="doc-biblioref">73</a>]</span></td>
<td>4.079855550647664</td>
<td>CC-BY-NC-ND</td>
<td><img src="images/paper-thumbnails/pc1/044818_thumbnail.png" title="fig:" /></td>
</tr>
<tr class="odd">
<td>SBpipe: a collection of pipelines for automating repetitive simulation and analysis tasks <span class="citation" data-cites="6gfAR2hM">[<a href="#ref-6gfAR2hM" role="doc-biblioref">75</a>]</span></td>
<td>4.022240241143516</td>
<td>CC-BY-NC-ND</td>
<td><img src="images/paper-thumbnails/pc1/107250_thumbnail.png" title="fig:" /></td>
</tr>
<tr class="even">
<td></td>
<td></td>
<td></td>
<td></td>
</tr>
<tr class="odd">
<td>Bromodomain inhibition reveals FGF15/19 as a target of epigenetic regulation and metabolic control <span class="citation" data-cites="NIw2ZKUa">[<a href="#ref-NIw2ZKUa" role="doc-biblioref">82</a>]</span></td>
<td>-3.4783803547922414</td>
<td>None</td>
<td><img src="images/paper-thumbnails/pc1/2019.12.11.872887_thumbnail.png" title="fig:" /></td>
</tr>
<tr class="even">
<td>Inhibition of Bruton’s tyrosine kinase reduces NF-kB and NLRP3 inflammasome activity preventing insulin resistance and microvascular disease <span class="citation" data-cites="1CM9H3mFb">[<a href="#ref-1CM9H3mFb" role="doc-biblioref">79</a>]</span></td>
<td>-3.6926161167521476</td>
<td>None</td>
<td><img src="images/paper-thumbnails/pc1/745943_thumbnail.png" title="fig:" /></td>
</tr>
<tr class="odd">
<td>Spatiotemporal proteomics uncovers cathepsin-dependent host cell death during bacterial infection <span class="citation" data-cites="2V2BJL87">[<a href="#ref-2V2BJL87" role="doc-biblioref">76</a>]</span></td>
<td>-3.728443135960558</td>
<td>CC-BY-ND</td>
<td><img src="images/paper-thumbnails/pc1/455048_thumbnail.png" title="fig:" /></td>
</tr>
<tr class="even">
<td>NADPH consumption by L-cystine reduction creates a metabolic vulnerability upon glucose deprivation <span class="citation" data-cites="nzDb3Lvn">[<a href="#ref-nzDb3Lvn" role="doc-biblioref">78</a>]</span></td>
<td>-3.7363965062637288</td>
<td>None</td>
<td><img src="images/paper-thumbnails/pc1/733162_thumbnail.png" title="fig:" /></td>
</tr>
<tr class="odd">
<td>AKT but not MYC promotes reactive oxygen species-mediated cell death in oxidative culture <span class="citation" data-cites="jmFU2eta">[<a href="#ref-jmFU2eta" role="doc-biblioref">80</a>]</span></td>
<td>-3.8769231933681176</td>
<td>None</td>
<td><img src="images/paper-thumbnails/pc1/754572_thumbnail.png" title="fig:" /></td>
</tr>
</tbody>
</table>
</div>
<div id="tbl:pc1_loadings_table" class="tablenos">
<table id="tbl:pc1_loadings_table" data-tag="S2">
<caption><span>Table S2:</span> Top and bottom five cosine similarity scores between tokens and the PC1 axis. </caption>
<thead>
<tr class="header">
<th>Cosine Similarity (PC1, word)</th>
<th>word</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td>0.6399154807185836</td>
<td>empirical</td>
</tr>
<tr class="even">
<td>0.5995356000266072</td>
<td>estimates</td>
</tr>
<tr class="odd">
<td>0.5918321530159384</td>
<td>choice</td>
</tr>
<tr class="even">
<td>0.5905550757923625</td>
<td>statistics</td>
</tr>
<tr class="odd">
<td>0.5832932491448216</td>
<td>performance</td>
</tr>
<tr class="even">
<td>0.5803836474390357</td>
<td>accuracy</td>
</tr>
<tr class="odd">
<td>0.5757250459195589</td>
<td>weighting</td>
</tr>
<tr class="even">
<td>0.5753027342288192</td>
<td>estimation</td>
</tr>
<tr class="odd">
<td>0.5730092178610916</td>
<td>uncertainty</td>
</tr>
<tr class="even">
<td>0.5720493442813257</td>
<td>task</td>
</tr>
<tr class="odd">
<td></td>
<td></td>
</tr>
<tr class="even">
<td>-0.4484093198386865</td>
<td>abrogated</td>
</tr>
<tr class="odd">
<td>-0.4490583645152233</td>
<td>transfected</td>
</tr>
<tr class="even">
<td>-0.4500847285921068</td>
<td>incubating</td>
</tr>
<tr class="odd">
<td>-0.4531550791501111</td>
<td>inhibited</td>
</tr>
<tr class="even">
<td>-0.4585422153514687</td>
<td>co-incubated</td>
</tr>
<tr class="odd">
<td>-0.4774721756292901</td>
<td>pre-incubated</td>
</tr>
<tr class="even">
<td>-0.4793057689825842</td>
<td>overexpressing</td>
</tr>
<tr class="odd">
<td>-0.4839313193713342</td>
<td>purified</td>
</tr>
<tr class="even">
<td>-0.4869885872803974</td>
<td>incubated</td>
</tr>
<tr class="odd">
<td>-0.5040798110023075</td>
<td>cultured</td>
</tr>
</tbody>
</table>
</div>
<div id="tbl:pc2_loadings_table" class="tablenos">
<table id="tbl:pc2_loadings_table" data-tag="S3">
<caption><span>Table S3:</span> Top and bottom five cosine similarity scores between tokens and the PC2 axis. </caption>
<thead>
<tr class="header">
<th>Cosine Similarity (PC2, word)</th>
<th>word</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td>0.65930201597598</td>
<td>genomic</td>
</tr>
<tr class="even">
<td>0.6333515216782134</td>
<td>genome</td>
</tr>
<tr class="odd">
<td>0.5974018685580009</td>
<td>gene</td>
</tr>
<tr class="even">
<td>0.5796531207938461</td>
<td>genomes</td>
</tr>
<tr class="odd">
<td>0.5353687686155728</td>
<td>annotation</td>
</tr>
<tr class="even">
<td>0.5310140161149529</td>
<td>sequencing</td>
</tr>
<tr class="odd">
<td>0.5197350376908197</td>
<td>sequencesM.</td>
</tr>
<tr class="even">
<td>0.5181781615670665</td>
<td>genome,</td>
</tr>
<tr class="odd">
<td>0.5168781637087506</td>
<td>bioinformatic</td>
</tr>
<tr class="even">
<td>0.513853407439108</td>
<td>WGS</td>
</tr>
<tr class="odd">
<td></td>
<td></td>
</tr>
<tr class="even">
<td>-0.4589201401582101</td>
<td>duration</td>
</tr>
<tr class="odd">
<td>-0.4690482252758019</td>
<td>stimuli</td>
</tr>
<tr class="even">
<td>-0.4712875761979691</td>
<td>amplitudes</td>
</tr>
<tr class="odd">
<td>-0.4772723570301678</td>
<td>contralateral</td>
</tr>
<tr class="even">
<td>-0.4813219679071856</td>
<td>stimulation:</td>
</tr>
<tr class="odd">
<td>-0.4946709932017581</td>
<td>delay</td>
</tr>
<tr class="even">
<td>-0.5111990014804086</td>
<td>stimulus</td>
</tr>
<tr class="odd">
<td>-0.5251288188682695</td>
<td>amplitude</td>
</tr>
<tr class="even">
<td>-0.543586881182879</td>
<td>stimulation</td>
</tr>
<tr class="odd">
<td>-0.5467022203294039</td>
<td>evoked</td>
</tr>
</tbody>
</table>
</div>
<h2 id="supplemental-figures-and-tables">Supplemental Figures and Tables</h2>
<div id="fig:biorxiv_categories" class="fignos">
<figure>
<img src="https://raw.githubusercontent.com/danich1/annorxiver/65ee4a556ab69f2308e5e4d9192905e8cfec3728/figure_generation/output/Figure_S2.png" data-tag="S2" alt="Figure S2: Neuroscience and bioinformatics are the two most common author-selected topics for bioRxiv preprints. Data for the information depicted in this figure are available at https://github.com/greenelab/annorxiver/blob/master/FIGURE_DATA_SOURCE.md#figure-s2." /><figcaption aria-hidden="true"><span>Figure S2:</span> Neuroscience and bioinformatics are the two most common author-selected topics for bioRxiv preprints.
Data for the information depicted in this figure are available at <a href="https://github.com/greenelab/annorxiver/blob/master/FIGURE_DATA_SOURCE.md#figure-s2">https://github.com/greenelab/annorxiver/blob/master/FIGURE_DATA_SOURCE.md#figure-s2</a>.</figcaption>
</figure>
</div>
<div id="tbl:token_table" class="tablenos">
<table id="tbl:token_table" data-tag="S4">
<caption><span>Table S4:</span> The top 100 frequently occurring tokens across our three corpora. </caption>
<thead>
<tr class="header">
<th>bioRxiv Tokens</th>
<th>PMCOA Tokens</th>
<th>NYTAC Tokens</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td>‘et’</td>
<td>‘\\u2009’</td>
<td>‘said’</td>
</tr>
<tr class="even">
<td>‘al’</td>
<td>‘\xa0’</td>
<td>‘mr.’</td>
</tr>
<tr class="odd">
<td>‘cell’</td>
<td>‘\t\t\t\t’</td>
<td>’ ’</td>
</tr>
<tr class="even">
<td>‘cells’</td>
<td>‘et’</td>
<td>‘–’</td>
</tr>
<tr class="odd">
<td>‘1’</td>
<td>‘1’</td>
<td>‘new’</td>
</tr>
<tr class="even">
<td>‘different’</td>
<td>‘cells’</td>
<td>‘new’</td>
</tr>
<tr class="odd">
<td>‘2’</td>
<td>‘al’</td>
<td>‘like’</td>
</tr>
<tr class="even">
<td>‘high’</td>
<td>‘cell’</td>
<td>‘year’</td>
</tr>
<tr class="odd">
<td>‘genes’</td>
<td>‘patients’</td>
<td>‘years’</td>
</tr>
<tr class="even">
<td>‘gene’</td>
<td>‘study’</td>
<td>‘united’</td>
</tr>
<tr class="odd">
<td>‘3’</td>
<td>‘2’</td>
<td>‘ms.’</td>
</tr>
<tr class="even">
<td>‘specific’</td>
<td>‘different’</td>
<td>‘today’</td>
</tr>
<tr class="odd">
<td>‘figure’</td>
<td>‘high’</td>
<td>‘york’</td>
</tr>
<tr class="even">
<td>‘single’</td>
<td>‘3’</td>
<td>‘old’</td>
</tr>
<tr class="odd">
<td>‘non’</td>
<td>‘\\u2013’</td>
<td>‘american’</td>
</tr>
<tr class="even">
<td>‘5’</td>
<td>‘significant’</td>
<td>‘yesterday’</td>
</tr>
<tr class="odd">
<td>‘\\u201d’</td>
<td>‘10’</td>
<td>‘time’</td>
</tr>
<tr class="even">
<td>‘\\u201c’</td>
<td>‘5’</td>
<td>‘lead’</td>
</tr>
<tr class="odd">
<td>‘data’</td>
<td>‘significantly’</td>
<td>‘people’</td>
</tr>
<tr class="even">
<td>‘10’</td>
<td>‘group’</td>
<td>‘dr.’</td>
</tr>
<tr class="odd">
<td>‘4’</td>
<td>‘4’</td>
<td>‘years’</td>
</tr>
<tr class="even">
<td>‘significant’</td>
<td>‘non’</td>
<td>‘york’</td>
</tr>
<tr class="odd">
<td>‘\\u2019’</td>
<td>‘compared’</td>
<td>‘week’</td>
</tr>
<tr class="even">
<td>‘found’</td>
<td>‘\\u201c’</td>
<td>‘officials’</td>
</tr>
<tr class="odd">
<td>‘protein’</td>
<td>‘\\u201d’</td>
<td>‘ago’</td>
</tr>
<tr class="even">
<td>‘model’</td>
<td>‘found’</td>
<td>‘including’</td>
</tr>
<tr class="odd">
<td>‘performed’</td>
<td>‘performed’</td>
<td>‘10’</td>
</tr>
<tr class="even">
<td>‘figure’</td>
<td>‘specific’</td>
<td>‘people’</td>
</tr>
<tr class="odd">
<td>‘analysis’</td>
<td>‘respectively’</td>
<td>‘high’</td>
</tr>
<tr class="even">
<td>‘study’</td>
<td>‘\\u200a’</td>
<td>‘john’</td>
</tr>
<tr class="odd">
<td>‘genetic’</td>
<td>‘showed’</td>
<td>‘public’</td>
</tr>
<tr class="even">
<td>‘significantly’</td>
<td>‘analysis’</td>
<td>‘good’</td>
</tr>
<tr class="odd">
<td>‘species’</td>
<td>‘including’</td>
<td>‘political’</td>
</tr>
<tr class="even">
<td>‘low’</td>
<td>‘low’</td>
<td>‘1’</td>
</tr>
<tr class="odd">
<td>‘human’</td>
<td>‘higher’</td>
<td>‘said’</td>
</tr>
<tr class="even">
<td>‘time’</td>
<td>‘clinical’</td>
<td>‘president’</td>
</tr>
<tr class="odd">
<td>‘including’</td>
<td>‘results’</td>
<td>‘year’</td>
</tr>
<tr class="even">
<td>‘respectively’</td>
<td>‘groups’</td>
<td>‘national’</td>
</tr>
<tr class="odd">
<td>‘time’</td>
<td>‘shown’</td>
<td>‘second’</td>
</tr>
<tr class="even">
<td>‘compared’</td>
<td>‘time’</td>
<td>‘million’</td>
</tr>
<tr class="odd">
<td>‘previously’</td>
<td>‘\xb0’</td>
<td>‘university’</td>
</tr>
<tr class="even">
<td>‘results’</td>
<td>‘total’</td>
<td>‘recent’</td>
</tr>
<tr class="odd">
<td>‘shown’</td>
<td>‘treatment’</td>
<td>‘small’</td>
</tr>
<tr class="even">
<td>‘fig’</td>
<td>‘protein’</td>
<td>‘percent’</td>
</tr>
<tr class="odd">
<td>‘multiple’</td>
<td>‘additional’</td>
<td>‘2’</td>
</tr>
<tr class="even">
<td>‘large’</td>
<td>‘studies’</td>
<td>‘long’</td>
</tr>
<tr class="odd">
<td>‘similar’</td>
<td>‘genes’</td>
<td>‘far’</td>
</tr>
<tr class="even">
<td>‘\\u2013’</td>
<td>‘positive’</td>
<td>‘big’</td>
</tr>
<tr class="odd">
<td>‘higher’</td>
<td>‘figure’</td>
<td>‘major’</td>
</tr>
<tr class="even">
<td>‘expression’</td>
<td>‘cells’</td>
<td>‘later’</td>
</tr>
<tr class="odd">
<td>‘expression’</td>
<td>‘gene’</td>
<td>‘west’</td>
</tr>
<tr class="even">
<td>‘samples’</td>
<td>‘data’</td>
<td>‘great’</td>
</tr>
<tr class="odd">
<td>‘i.e.’</td>
<td>‘anti’</td>
<td>‘30’</td>
</tr>
<tr class="even">
<td>‘fig’</td>
<td>‘previous’</td>
<td>‘little’</td>
</tr>
<tr class="odd">
<td>‘individual’</td>
<td>‘data’</td>
<td>‘million’</td>
</tr>
<tr class="even">
<td>‘\xb0’</td>
<td>‘addition’</td>
<td>‘3’</td>
</tr>
<tr class="odd">
<td>‘dna’</td>
<td>‘human’</td>
<td>‘mrs.’</td>
</tr>
<tr class="even">
<td>‘average’</td>
<td>‘health’</td>
<td>‘states’</td>
</tr>
<tr class="odd">
<td>‘supplementary’</td>
<td>‘observed’</td>
<td>‘says’</td>
</tr>
<tr class="even">
<td>‘previous’</td>
<td>‘according’</td>
<td>‘according’</td>
</tr>
<tr class="odd">
<td>‘total’</td>
<td>‘single’</td>
<td>‘late’</td>
</tr>
<tr class="even">
<td>‘showed’</td>
<td>‘reported’</td>
<td>‘young’</td>
</tr>
<tr class="odd">
<td>‘data’</td>
<td>‘previously’</td>
<td>‘away’</td>
</tr>
<tr class="even">
<td>‘observed’</td>
<td>‘mice’</td>
<td>‘life’</td>
</tr>
<tr class="odd">
<td>‘functional’</td>
<td>‘20’</td>
<td>‘american’</td>
</tr>
<tr class="even">
<td>‘number’</td>
<td>‘\\u2003’</td>
<td>‘month’</td>
</tr>
<tr class="odd">
<td>‘based’</td>
<td>‘6’</td>
<td>‘large’</td>
</tr>
<tr class="even">
<td>‘\\u2018’</td>
<td>‘c’</td>
<td>‘company’</td>
</tr>
<tr class="odd">
<td>‘small’</td>
<td>‘study’</td>
<td>‘way’</td>
</tr>
<tr class="even">
<td>‘cells’</td>
<td>‘control’</td>
<td>‘black’</td>
</tr>
<tr class="odd">
<td>‘positive’</td>
<td>‘similar’</td>
<td>‘early’</td>
</tr>
<tr class="even">
<td>‘conditions’</td>
<td>‘studies’</td>
<td>‘east’</td>
</tr>
<tr class="odd">
<td>‘20’</td>
<td>‘expression’</td>
<td>‘real’</td>
</tr>
<tr class="even">
<td>‘data’</td>
<td>‘data’</td>
<td>‘3’</td>
</tr>
<tr class="odd">
<td>‘regions’</td>
<td>‘time’</td>
<td>‘11’</td>
</tr>
<tr class="even">
<td>‘data’</td>
<td>‘30’</td>
<td>‘state’</td>
</tr>
<tr class="odd">
<td>‘proteins’</td>
<td>‘fig’</td>
<td>‘20’</td>
</tr>
<tr class="even">
<td>‘new’</td>
<td>‘95’</td>
<td>‘world’</td>
</tr>
<tr class="odd">
<td>‘mice’</td>
<td>‘\\u2019’</td>
<td>‘net’</td>
</tr>
<tr class="even">
<td>‘relative’</td>
<td>‘model’</td>
<td>‘j.’</td>
</tr>
<tr class="odd">
<td>‘addition’</td>
<td>‘levels’</td>
<td>‘street’</td>
</tr>
<tr class="even">
<td>‘6’</td>
<td>‘primary’</td>
<td>‘end’</td>
</tr>
<tr class="odd">
<td>‘neurons’</td>
<td>‘samples’</td>
<td>‘think’</td>
</tr>
<tr class="even">
<td>‘studies’</td>
<td>‘large’</td>
<td>‘day’</td>
</tr>
<tr class="odd">
<td>‘c’</td>
<td>‘small’</td>
<td>‘long’</td>
</tr>
<tr class="even">
<td>‘cells’</td>
<td>‘lower’</td>
<td>‘state’</td>
</tr>
<tr class="odd">
<td>‘100’</td>
<td>’ ’</td>
<td>‘david’</td>
</tr>
<tr class="even">
<td>‘function’</td>
<td>‘increased’</td>
<td>‘best’</td>
</tr>
<tr class="odd">
<td>‘activity’</td>
<td>‘100’</td>
<td>‘robert’</td>
</tr>
<tr class="even">
<td>‘highly’</td>
<td>‘patients’</td>
<td>‘local’</td>
</tr>
<tr class="odd">
<td>‘experimental’</td>
<td>‘based’</td>
<td>‘city’</td>
</tr>
<tr class="even">
<td>‘standard’</td>
<td>‘figure’</td>
<td>‘million’</td>
</tr>
<tr class="odd">
<td>‘30’</td>
<td>‘blood’</td>
<td>‘5’</td>
</tr>
<tr class="even">
<td>‘levels’</td>
<td>‘50’</td>
<td>‘earns’</td>
</tr>
<tr class="odd">
<td>‘brain’</td>
<td>‘effect’</td>
<td>‘st.’</td>
</tr>
<tr class="even">
<td>‘rna’</td>
<td>‘normal’</td>
<td>‘president’</td>
</tr>
<tr class="odd">
<td>‘models’</td>
<td>‘standard’</td>
<td>‘world’</td>
</tr>
<tr class="even">
<td>‘identified’</td>
<td>‘conditions’</td>
<td>‘nearly’</td>
</tr>
<tr class="odd">
<td>‘binding’</td>
<td>‘level’</td>
<td>‘4’</td>
</tr>
<tr class="even">
<td>‘50’</td>
<td>‘important’</td>
<td>‘home’</td>
</tr>
</tbody>
</table>
</div>
<div id="fig:corpora_comparison_panels_bio_v_pmid" class="fignos">
<figure>
<img src="https://raw.githubusercontent.com/danich1/annorxiver/65ee4a556ab69f2308e5e4d9192905e8cfec3728/figure_generation/output/Figure_S3.png" data-tag="S3" alt="Figure S3: A. The significant differences in token frequencies for the corpora appear to be driven by the fields with the highest uptake of bioRxiv, as terms from neuroscience and genomics are relatively more abundant in bioRxiv. We plotted the 95% confidence interval for each reported token. B. Of the tokens that differ between bioRxiv and PMC, the most abundant in bioRxiv are “gene”, “genes” and “model” while the most abundant in PMC is “study.” Data for the information depicted in this figure are available at https://github.com/greenelab/annorxiver/blob/master/FIGURE_DATA_SOURCE.md#figure-s3." /><figcaption aria-hidden="true"><span>Figure S3:</span> <strong>A.</strong> The significant differences in token frequencies for the corpora appear to be driven by the fields with the highest uptake of bioRxiv, as terms from neuroscience and genomics are relatively more abundant in bioRxiv.
We plotted the 95% confidence interval for each reported token.
<strong>B.</strong> Of the tokens that differ between bioRxiv and PMC, the most abundant in bioRxiv are “gene”, “genes” and “model” while the most abundant in PMC is “study.”
Data for the information depicted in this figure are available at <a href="https://github.com/greenelab/annorxiver/blob/master/FIGURE_DATA_SOURCE.md#figure-s3">https://github.com/greenelab/annorxiver/blob/master/FIGURE_DATA_SOURCE.md#figure-s3</a>.</figcaption>
</figure>
</div>
<div id="fig:corpora_comparison_panels_pre_v_pub" class="fignos">
<figure>
<img src="https://raw.githubusercontent.com/danich1/annorxiver/65ee4a556ab69f2308e5e4d9192905e8cfec3728/figure_generation/output/Figure_S4.png" data-tag="S4" alt="Figure S4: A. The significant differences in token frequencies for preprints and their corresponding published version often appear to be associated with data availability and supplementary or additional materials. We plotted the 95% confidence interval for each reported token. B. The tokens with the largest absolute differences in abundance appear related to scientific figures and data availability. Data for the information depicted in this figure are available at https://github.com/greenelab/annorxiver/blob/master/FIGURE_DATA_SOURCE.md#figure-s4." /><figcaption aria-hidden="true"><span>Figure S4:</span> <strong>A.</strong> The significant differences in token frequencies for preprints and their corresponding published version often appear to be associated with data availability and supplementary or additional materials.
We plotted the 95% confidence interval for each reported token.
<strong>B.</strong> The tokens with the largest absolute differences in abundance appear related to scientific figures and data availability.
Data for the information depicted in this figure are available at <a href="https://github.com/greenelab/annorxiver/blob/master/FIGURE_DATA_SOURCE.md#figure-s4">https://github.com/greenelab/annorxiver/blob/master/FIGURE_DATA_SOURCE.md#figure-s4</a>.</figcaption>
</figure>
</div>
<div id="fig:knn_auc" class="fignos">
<figure>
<img src="https://raw.githubusercontent.com/danich1/annorxiver/65ee4a556ab69f2308e5e4d9192905e8cfec3728/figure_generation/output/Figure_S5.png" data-tag="S5" style="width:100.0%" alt="Figure S5: Both classifiers outperform the randomized baseline when predicting a paper’s journal endpoint. This bargraph shows each model’s accuracy in respect to predicting the training and test set. Data for the information depicted in this figure are available at https://github.com/greenelab/annorxiver/blob/master/FIGURE_DATA_SOURCE.md#figure-s5." /><figcaption aria-hidden="true"><span>Figure S5:</span> Both classifiers outperform the randomized baseline when predicting a paper’s journal endpoint.
This bargraph shows each model’s accuracy in respect to predicting the training and test set.
Data for the information depicted in this figure are available at <a href="https://github.com/greenelab/annorxiver/blob/master/FIGURE_DATA_SOURCE.md#figure-s5">https://github.com/greenelab/annorxiver/blob/master/FIGURE_DATA_SOURCE.md#figure-s5</a>.</figcaption>
</figure>
</div>
<div id="fig:filterd_polka_analysis" class="fignos">
<figure>
<img src="https://raw.githubusercontent.com/danich1/annorxiver/65ee4a556ab69f2308e5e4d9192905e8cfec3728/figure_generation/output/Figure_S6.png" data-tag="S6" style="width:100.0%" alt="Figure S6: A. The Preprints in Motion were published faster than other bioRxiv preprints, and the number of versions was not associated with an increase in time to publication. The x-axis shows the number of versions of a preprint posted on bioRxiv. The y-axis indicates the number of days that elapsed between the first version of a preprint posted on bioRxiv and the date at which the peer-reviewed publication appeared. The density of observations is depicted in the violin plot with an embedded boxplot. The red dots and red regression line represent Preprints in Motion. B. The Preprints in Motion collection were published faster than other bioRxiv preprints, and no dependence between the amount of linguistic change and time to publish was observed. The x-axis shows the Euclidean distance between document representations of the first version of a preprint and its peer-reviewed form. The y-axis shows the number of days elapsed between the first version of a preprint posted on bioRxiv and when a preprint is published. The color bar on the right represents the density of each hexbin in this plot, where more dense regions are shown in a brighter color. The red dots and red regression line represent Preprints in Motion. Data for the information depicted in this figure are available at https://github.com/greenelab/annorxiver/blob/master/FIGURE_DATA_SOURCE.md#figure-s6." /><figcaption aria-hidden="true"><span>Figure S6:</span> <strong>A.</strong> The Preprints in Motion were published faster than other bioRxiv preprints, and the number of versions was not associated with an increase in time to publication.
The x-axis shows the number of versions of a preprint posted on bioRxiv.
The y-axis indicates the number of days that elapsed between the first version of a preprint posted on bioRxiv and the date at which the peer-reviewed publication appeared.
The density of observations is depicted in the violin plot with an embedded boxplot.
The red dots and red regression line represent Preprints in Motion.
<strong>B.</strong> The Preprints in Motion collection were published faster than other bioRxiv preprints, and no dependence between the amount of linguistic change and time to publish was observed.
The x-axis shows the Euclidean distance between document representations of the first version of a preprint and its peer-reviewed form.
The y-axis shows the number of days elapsed between the first version of a preprint posted on bioRxiv and when a preprint is published.
The color bar on the right represents the density of each hexbin in this plot, where more dense regions are shown in a brighter color.
The red dots and red regression line represent Preprints in Motion.
Data for the information depicted in this figure are available at <a href="https://github.com/greenelab/annorxiver/blob/master/FIGURE_DATA_SOURCE.md#figure-s6">https://github.com/greenelab/annorxiver/blob/master/FIGURE_DATA_SOURCE.md#figure-s6</a>.</figcaption>
</figure>
</div>
<h1 id="response-to-reviewers">Response to Reviewers</h1>
<h2 id="reviewer-1">Reviewer #1</h2>
<blockquote>
<p>[identifies himself as Ross Mounce]</p>
<p>This manuscript ‘Linguistic Analysis of the bioRxiv Preprint Landscape’ presents an analysis of bioRxiv fulltexts and metadata, relative to journal-published versions of the same preprints (n= 17,952 pairs), and the New York Times Annotated Corpus.</p>
<p>It’s an interesting manuscript worthy of publication in PLOS Biology after a few relatively minor revisions.</p>
<p>I have left many comments directly on the manuscript via the dedicated manuscript website, using public Hypothes.is annotations: https://greenelab.github.io/annorxiver_manuscript/</p>
<p>I incorporate some but not all of these comments into this formal review supplied to the journal (PLOS Biology) who invited me to review this manuscript.</p>
<p>Unsurprisingly, biorxiv preprints and journal-published versions of biorxiv preprints are found to be linguistically different to the New York Times Annotated Corpus e.g. in average document length and to a lesser degree in average sentence length, and % in passive voice.</p>
<p>Luckily there are plenty more actually interesting results reported in this manuscript, not least that of 23,271 preprint-published pairs, 17,952 of those pairs (&gt;77%) had a published version present within the PMCOA corpus. I don’t think the authors quite realise the significance of this result. 77% is a very very high rate of open access. It could do with being discussed more within the manuscript e.g. relative to the overall (lower) rate of open access of <em>all</em> biomedical and life science research articles. What does this signify about preprint authors / ‘preprinters’?</p>
<p>I can think of a couple of hypotheses:</p>
<ol type="a">
<li><p>preprinters are perhaps more likely to have grant-funded research subject to an open access policy</p></li>
<li><p>perhaps preprinters are more publishing ‘savvy’ and want to achieve more impact/citations and thus strive harder to ensure that the eventual journal-published version of their work is open access (reflected in being in PMCOA).</p></li>
</ol>
</blockquote>
<p><strong>We appreciate the reviewer’s positive comments on our manuscript.
The reviewer is corrected that we did not realize that the finding of 77% of preprint-published pairs being present in PMCOA is a surprising discovery, and we thank this reviewer for bringing this to our attention.
We now include the following in our discussion section to emphasize this point:</strong></p>
<div class="sourceCode" id="cb1"><pre class="sourceCode diff"><code class="sourceCode diff"><span id="cb1-1"><a href="#cb1-1" aria-hidden="true" tabindex="-1"></a><span class="va">+ Over 77% of bioRxiv preprints with a corresponding publication in our snapshot were successfully detected within Pubmed Central’s Open Access Corpus (PMCOA). </span></span>
<span id="cb1-2"><a href="#cb1-2" aria-hidden="true" tabindex="-1"></a><span class="va">+ This suggests that most work from groups participating in the preprint ecosystem is now available in final form for literature mining and other applications.</span></span></code></pre></div>
<blockquote>
<p>If it were my choice I would cut the entire subsection ‘Document embeddings derived from bioRxiv reveal fields and subfields’. It is already known that document embeddings can reveal fields and subfields. Being ‘preprints’ or ‘biorxiv preprints’ rather than say published journal articles won’t change that. I found this section very uninteresting and extremely un-novel. It is descriptive and accurate, but in the context of an already long manuscript, I feel it is unnecessary.</p>
</blockquote>
<p><strong>We felt that this analysis also included other findings that were less obvious: namely the principal components that separated fields and the finding that certain fields like systems biology were spread across certain components that distinguished quantitative systems biology from cellular systems biology papers.
We think that this lays the groundwork for a number of future research efforts.
However, we agree with the reviewer that this manuscript does present a broad examination of the full text content of <em>bioRxiv</em> and is somewhat lengthy, so we did move this section into the supplement.</strong></p>
<blockquote>
<p>Aside from the manuscript, I have some brief comments on the actual web application.</p>
<p>I tried some palaeontology preprints (it’s a field i’m very familiar with). The results were rather mixed.
e.g. for https://greenelab.github.io/preprint-similarity-search/?doi=10.1101/2020.12.10.406678 (“The first dinosaur egg remains a mystery”), the most similar paper recommendations were excellent. However, the most similar journals suggested were surprisingly poor - many of these could obviously at a glance never publish this preprint (dinosaurs are not plants!) e.g. American Journal of Botany, World Archaeology, Journal of Phycology, The Holocene, Botanical Journal of the Linnean Society. Linnean Society of London</p>
<p>But I realise that PMCOA isn’t exactly great training date for interpreting palaeontology articles – fringe content from PMC’s perspective(?)</p>
</blockquote>
<p><strong>We agree that using PMCOA as a training set will limit the fields to which the website can be applied.
We felt that PMCOA was likely to be appropriate for much <em>bioRxiv</em> and <em>medRxiv</em> content, which are the servers our tool supports.
We have also implemented a system to automatically update to bring in new PMCOA papers.
If PMCOA begins to include more journals that publish articles in these fields or author-contributed manuscripts in these fields, then the tool would be more likely to identify appropriate matches.</strong></p>
<blockquote>
<p>Specific comments:</p>
<p>1.) https://hypothes.is/a/1ODs5NLbEeuhnEPjCpUFpA</p>
<p>I think this needs to be made more specific as [25] analysed a few different things.</p>
<p>Your statement here is true with respect to their analysis of abstract text “Over 50% of abstracts had changes that minorly altered, strengthened, or softened the main conclusions”</p>
<p>BUT</p>
<p>it is not true with respect to the panels and tables analysis in [25]:</p>
<p>“over 70% of 162 published preprints were classified with “no change” or superficial rearrangements to panels and 163 tables, confirming the previous conclusion”</p>
<p>thus perhaps you should consider writing something like:</p>
<p>an analysis of preprints posted at the beginning of 2020 revealed that over 50% underwent minor changes in the abstract text as they were published, but over 70% had ‘no change’ or only superficial rearrangements to panels and tables [25].</p>
</blockquote>
<p><strong>We agree with the reviewer that the proposed phrasing is better.
We now write:</strong></p>
<div class="sourceCode" id="cb2"><pre class="sourceCode diff"><code class="sourceCode diff"><span id="cb2-1"><a href="#cb2-1" aria-hidden="true" tabindex="-1"></a><span class="st">- Preprint repositories by definition do not perform in-depth peer review, which can result in posted preprints containing inconsistent results or conclusions [...]; however, an analysis of preprints posted at the beginning of 2020 revealed that most underwent minor changes as they were published [...].</span></span>
<span id="cb2-2"><a href="#cb2-2" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb2-3"><a href="#cb2-3" aria-hidden="true" tabindex="-1"></a><span class="va">+ Preprint repositories by definition do not perform in-depth peer review, which can result in posted preprints containing inconsistent results or conclusions [...]; however, an analysis of preprints posted at the beginning of 2020 revealed that over 50% underwent minor changes in the abstract text as they were published, but over 70% did not change or only had simple rearrangements to panels and tables [...].</span></span></code></pre></div>
<blockquote>
<p>2.) https://hypothes.is/a/djNirNLcEeu2YxNM5WBxvA</p>
<p>but to clarify, you did remove these from the analysis, right? It would just be good to clarify that. They are easy to identify and should just be removed. I can’t see how they would add anything but noise to this analysis. What is the total number of preprints after withdrawn preprints are removed from the sample?</p>
</blockquote>
<p><strong>In the version of the manuscript that the reviewer saw, we did not remove these preprints from our analysis as we felt their impact would be minimal.
Based on the reviewer’s comments, we have now rerun all of the analyses with these withdrawn articles removed.
This did not lead to substantive changes in the article or figures, but we agree it was the most rigorous analysis and are happy that the reviewer brought up this point.</strong></p>
<div class="sourceCode" id="cb3"><pre class="sourceCode diff"><code class="sourceCode diff"><span id="cb3-1"><a href="#cb3-1" aria-hidden="true" tabindex="-1"></a><span class="st">- As there were very few withdrawn preprints, we did not treat these as a special case.</span></span>
<span id="cb3-2"><a href="#cb3-2" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb3-3"><a href="#cb3-3" aria-hidden="true" tabindex="-1"></a><span class="va">+ We encountered a total of 72 withdrawn preprints within our snapshot. </span></span>
<span id="cb3-4"><a href="#cb3-4" aria-hidden="true" tabindex="-1"></a><span class="va">+ After removal, we were left with 97,951 preprints for our downstream analyses.</span></span></code></pre></div>
<blockquote>
<p>3.) https://hypothes.is/a/bmdkpNLeEeuZ0k8oLqyeWQ</p>
<p>actually, there need not necessarily be an embargo period. Many publishers now offer a zero-day embargo so that the author accepted manuscript can be deposited either at acceptance (before even journal publication!) or on the day of journal publication. Even if the journal normally tries to embargo the work, you can see some full text author manuscripts become immediately available well before the journal would normally permit them ‘out’ thanks to the Plan S Rights Retention Strategy e.g. this one here: https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7610590/</p>
<p>So what you should really say here is that full text works appear in PMC as either accepted author manuscripts (green open access) or via open access publishing at the journal (gold open access).</p>
<p>BTW, I resent calling it a ‘closed access’ [article?] if the accepted manuscript is fully freely available – that would seem to give undue primacy to the journal published version. It’s an article with different versions - one freely accessible at a repository e.g. PMC, without publisher branding and another behind a paywall at the publisher website with publisher branding</p>
</blockquote>
<p><strong>We agree that ‘closed access’ was imprecise phrasing.
We have updated this section in our manuscript.</strong></p>
<div class="sourceCode" id="cb4"><pre class="sourceCode diff"><code class="sourceCode diff"><span id="cb4-1"><a href="#cb4-1" aria-hidden="true" tabindex="-1"></a><span class="st">- PMC articles can be closed access ones from research funded by the NIH appearing after an embargo period or be published under Gold Open Access [...] publishing schemes.</span></span>
<span id="cb4-2"><a href="#cb4-2" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb4-3"><a href="#cb4-3" aria-hidden="true" tabindex="-1"></a><span class="va">+ Articles appear in PMC as either accepted author manuscripts (Green Open Access) or via open access publishing at the journal (Gold Open Access [...]).</span></span></code></pre></div>
<blockquote>
<p>4.) https://hypothes.is/a/-v_PbtLeEeu8Rw8afgzT5g</p>
<p>presuming a journal allows individual articles to be published with a CC BY licence under a so-called ‘hybrid-OA’ option, can a journal really NOT participate for those CC BY licenced articles? If biomedically relevant and CC BY licensed surely PMC takes that content at the article level and thus its debatable as to whether journals really have the power to actually 100% not participate.</p>
</blockquote>
<p><strong>We agree with the reviewer that this is an important distinction and have adjusted our text:</strong></p>
<div class="sourceCode" id="cb5"><pre class="sourceCode diff"><code class="sourceCode diff"><span id="cb5-1"><a href="#cb5-1" aria-hidden="true" tabindex="-1"></a><span class="st">- Individual journals have the option to fully participate in submitting articles to PMC, selectively participate sending only a few papers to PMC, only submit papers according to NIH&#39;s public access policy [...], or not participate at all.</span></span>
<span id="cb5-2"><a href="#cb5-2" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb5-3"><a href="#cb5-3" aria-hidden="true" tabindex="-1"></a><span class="va">+ Individual journals have the option to fully participate in submitting articles to PMC, selectively participate sending only a few papers to PMC, only submit papers according to NIH&#39;s public access policy [...], or not participate at all; however, individual articles published with the CC BY license may be incorporated.</span></span></code></pre></div>
<blockquote>
<p>5.) https://hypothes.is/a/vekhYNLfEeuxGm9i6MkLqw
it’s a real pity you chose not to compare preprints to author manuscripts. As your results demonstrated, lots of the word changes were just journal-style related e.g. “figure” -&gt; “fig.” . An analysis of just preprints matched to author manuscripts would get more closely and cleanly to what the textual difference between pre-peer-review and post-peer-review (without minor stylistic changes).</p>
</blockquote>
<p><strong>We agree that contrasting differences in bioRxiv to author-supplied vs journal-supplied manuscripts have the potential to remove some of the journal-related stylistic changes revealed in our token analysis.
We felt that this would be an interesting manuscript in its own right and that we would not be able to give it an appropriate treatment within this manuscript.
However, we leave this analysis for future endeavors.</strong></p>
<blockquote>
<p>6.) https://hypothes.is/a/YkqPKNLgEeucPGc1_W-jzA</p>
<p>minor typo: tagging surely</p>
</blockquote>
<p><strong>Thank you for pointing this out.
We updated text to fix this typo.</strong></p>
<div class="sourceCode" id="cb6"><pre class="sourceCode diff"><code class="sourceCode diff"><span id="cb6-1"><a href="#cb6-1" aria-hidden="true" tabindex="-1"></a><span class="st">- This collection contains over 1.8 million articles where 1.5 million of those articles have undergone manual entity tagged by library scientists [...].</span></span>
<span id="cb6-2"><a href="#cb6-2" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb6-3"><a href="#cb6-3" aria-hidden="true" tabindex="-1"></a><span class="va">+ This collection contains over 1.8 million articles where 1.5 million of those articles have undergone manual entity tagging by library scientists [...].</span></span></code></pre></div>
<blockquote>
<p>7.) https://hypothes.is/a/Bsv68tLkEeugHEfCZMUGag
From the perspective of a person (me!) interested in open access to (‘final’) peer-reviewed research outputs this is a super interesting result in itself, which should perhaps be remarked upon more in this manuscript.</p>
<p>It implies that over 77% (17,952/23,271) of biomedical preprints that are detectably linked to a journal published paper, that subsequent journal published paper became open access in the PMCOA corpus (regardless of specific means/route). That’s great news. The subset of works from biomedical researchers that do preprinting have a much higher level of open access (to the eventual journal published version) than biomedical research overall (including works that don’t have a preprint version)</p>
<p>See figure 3a from ‘Open access levels: a quantitative exploration using Web of Science and oaDOI data’ by Bosman and Kramer for a comparator looking at OA levels in biomedical and life science papers https://peerj.com/preprints/3520.pdf even in the ‘best’ OA performing subfield (Cell Biology) it doesnt reach 70%. 30% to 50% is more typical albeit looking at 2016 publications.</p>
<p>Put another way, we only ‘lose’ 23% of biomedical preprinted research to paywalled journals that do not allow a copy of the work to be made full text available in the PMCOA corpus, in reasonable time**. And in those cases we still have access to the preprint</p>
<p>** with no doubt many other caveats such as cases where the author could do it without help from the journal, but does not for some unknown reason</p>
</blockquote>
<p><strong>We appreciate that the reviewer pointed out this interesting finding in our results that we had missed, and we have updated our discussion accordingly.</strong></p>
<div class="sourceCode" id="cb7"><pre class="sourceCode diff"><code class="sourceCode diff"><span id="cb7-1"><a href="#cb7-1" aria-hidden="true" tabindex="-1"></a><span class="st">- The majority of research involving bioRxiv focuses on the metadata of preprints; however, the language contained within these preprints has not previously been systematically examined.</span></span>
<span id="cb7-2"><a href="#cb7-2" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb7-3"><a href="#cb7-3" aria-hidden="true" tabindex="-1"></a><span class="va">+ Over 77% of bioRxiv preprints with a corresponding publication in our snapshot were successfully detected within Pubmed Central&#39;s Open Access Corpus (PMCOA).</span></span>
<span id="cb7-4"><a href="#cb7-4" aria-hidden="true" tabindex="-1"></a><span class="va">+ This suggests that most work from groups participating in the preprint ecosystem is now available in final form for literature mining and other applications.</span></span>
<span id="cb7-5"><a href="#cb7-5" aria-hidden="true" tabindex="-1"></a><span class="va">+ Most research on bioRxiv preprints has examined their metadata; we examine the text content as well.</span></span></code></pre></div>
<blockquote>
<p>8.) https://hypothes.is/a/aWmt3tLlEeuh3M-Ku8ThvQ</p>
<p>To be clear 326 stopwords is the default setting?</p>
<p>Interestingly ‘ca’ is one of those 326 stopwords. I would have thought that one might actually be significant in a life sciences context e.g. calcium channels “Ca2+”</p>
</blockquote>
<p><strong>We used the 326 stopwords provided by default.
We agree that the stopwords are not precisely tuned for life sciences research.
We have adjusted our text to say:</strong></p>
<div class="sourceCode" id="cb8"><pre class="sourceCode diff"><code class="sourceCode diff"><span id="cb8-1"><a href="#cb8-1" aria-hidden="true" tabindex="-1"></a><span class="st">- We used spaCy&#39;s &quot;en_core_web_sm&quot; model [@spacy2] (version 2.2.3) to preprocess all corpora and filter out 326 spaCy-provided stopwords.</span></span>
<span id="cb8-2"><a href="#cb8-2" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb8-3"><a href="#cb8-3" aria-hidden="true" tabindex="-1"></a><span class="va">+ We used spaCy&#39;s &quot;en_core_web_sm&quot; model [@spacy2] (version 2.2.3) to preprocess all corpora and filter out 326 stopwords using spaCy&#39;s default settings.</span></span></code></pre></div>
<blockquote>
<p>9.) https://hypothes.is/a/dkIurtLmEeuqkU-qTN4dtg</p>
<p>I’m sure you’ve got this in the github, but just to make the manuscript more readily understandable without digging around in github, do you think you could provide as a supplementary file a list of those 100 most frequently occurring tokens, so that people can get a better feel for what the data is here?</p>
</blockquote>
<p><strong>We agree with the reviewer that this is a convenient table to have at hand.
This is now Supplementary Table 5.</strong></p>
<blockquote>
<p>10.) https://hypothes.is/a/-2AU8tLmEeu6swvY4jN5Mw</p>
<p>hmmm… not a problem of this manuscript, but that’s really not good enough from bioRxiv is it? Change one word of a long and complex title and suddenly ‘oh, we can’t do it’. A comment to suggest that bioRxiv could do better would be fun, no(?) i.e. look at author names AND title and if both are similar enough, then do auto-linking.</p>
<p>oh okay, you did actually do that. Nice :)</p>
</blockquote>
<p><strong>We are glad that you liked our document matching solution.</strong></p>
<blockquote>
<p>11.) https://hypothes.is/a/dhGbgNLnEeuYkkfUNqKwgA</p>
<p>I don’t suppose you could possibly be precise about this rather than just ‘a limited number’? Is it 5, 50, or 500?</p>
</blockquote>
<p><strong>We agree that this line is vague and have updated our text to be more explicit.</strong></p>
<div class="sourceCode" id="cb9"><pre class="sourceCode diff"><code class="sourceCode diff"><span id="cb9-1"><a href="#cb9-1" aria-hidden="true" tabindex="-1"></a><span class="st">- There were a limited number of cases in which authors appeared to post preprints after the publication date, which results in preprints receiving a negative time difference, as previously reported [...]</span></span>
<span id="cb9-2"><a href="#cb9-2" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb9-3"><a href="#cb9-3" aria-hidden="true" tabindex="-1"></a><span class="va">+ We encountered 123 cases where the preprint posting date was subsequent to the publication date, resulting in a negative time difference, as previously reported [...].</span></span></code></pre></div>
<blockquote>
<p>12.) https://hypothes.is/a/DzrzGtLtEeuJlcdOlUfUBA</p>
<p>I’m surprised to see no citation given to JANE: https://jane.biosemantics.org/</p>
<p>https://academic.oup.com/bioinformatics/article/24/5/727/202224</p>
<p>reviewed in https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6300233/</p>
<p>The ‘find journals’ functionality of JANE appears somewhat similar to the discovering similar journals functionality here.</p>
</blockquote>
<p><strong>Thank you for pointing this citation out.
We have incorporated it into our manuscript.</strong></p>
<div class="sourceCode" id="cb10"><pre class="sourceCode diff"><code class="sourceCode diff"><span id="cb10-1"><a href="#cb10-1" aria-hidden="true" tabindex="-1"></a><span class="st">- We developed a web application that places any bioRxiv or medRxiv preprint into the overall document landscape and identifies similar papers and journals.</span></span>
<span id="cb10-2"><a href="#cb10-2" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb10-3"><a href="#cb10-3" aria-hidden="true" tabindex="-1"></a><span class="va">+ We developed a web application that places any bioRxiv or medRxiv preprint into the overall document landscape and identifies topically similar papers and journals (similar to [..]).</span></span></code></pre></div>
<blockquote>
<p>13.) https://hypothes.is/a/eJ7-DtLpEeuh0A8OL8WAWw</p>
<p>This is the kind of journal-faff difference that I hypothesise would not be visible or less visible if one did an analysis of preprints vs author manuscripts.</p>
<p>There is change but is change from figure to ‘fig.’ to suit journal style actually helpful/valuable? In my opinion it is not!</p>
</blockquote>
<p><strong>We recognize the limitation of our design choice and agree it would be interesting to compare journal-supplied vs author-supplied manuscripts in a future study.</strong></p>
<blockquote>
<p>14.) https://hypothes.is/a/-ixA_NLqEeuv6YviR-lI0A</p>
<p>I would cut this entire subsection from the manuscript to make it shorter (or relegate it to a supplementary section).</p>
<p>Don’t we already know that if one uses full texts we can determine the subfield of the paper? It’s not that interesting in my opinion and not relevant to the main hypotheses of the paper – comparing between preprints and the journal published version.</p>
</blockquote>
<p><strong>We believe this section is needed to describe the landscape of the bioRxiv corpus; however, we agree the manuscript is long and has many results, so we moved this section to the supplement.</strong></p>
<blockquote>
<p>15.) https://hypothes.is/a/V2wj3NLqEeuZ9idQ44yS4Q</p>
<p>adjusting for recency? i.e. not sampling 2019 preprints? in figure C the line indicates (if I’m interpreting correctly) that overall only 46.55% are published but that’s because it includes very recent preprints that haven’t had time to be journal published yet. Just be explicit that you are adjusting for recency (i.e. excluding 2019 and newer preprints) when you say that most preprints are eventually published.</p>
</blockquote>
<p><strong>Initially, we incorporated all preprints (2013 to early 2020) when calculating the overall proportion of preprints published.
We agree that adjusting for recency is a better approach; however, we need to incorporate all preprints to have a fair comparison against Abdill et al. estimate.
The authors’ estimate incorporated all preprints at the time of their study.
Therefore, we updated our analysis to include two proportion estimates.
Our first estimate included all preprints posted onto bioRxiv, while the second estimate adjusts for recency.
These two estimates are found in Figure 2 in our updated manuscript (reproduced below), where the first estimate is the solid green line and our second estimate is the dashed line.</strong></p>
<figure>
<img src="https://raw.githubusercontent.com/danich1/annorxiver/cfa1b151ee91e36d72d6f6ec0cdd5378e6105fc4/biorxiv/article_distances/output/figures/publication_rate_reviewer_request.png" width="680" alt="updated overall proportion calculation" /><figcaption aria-hidden="true">updated overall proportion calculation</figcaption>
</figure>
<blockquote>
<p>16.) https://hypothes.is/a/N8cKltLrEeuZ-iPc5lrP5g</p>
<p>text changes relative to the journal published version? You might want to make that more explicit. Text changes alone is not adequately specific in my opinion.</p>
</blockquote>
<p><strong>We agree that text changes were a bit vague and have updated the title accordingly.</strong></p>
<div class="sourceCode" id="cb11"><pre class="sourceCode diff"><code class="sourceCode diff"><span id="cb11-1"><a href="#cb11-1" aria-hidden="true" tabindex="-1"></a><span class="st">- Preprints with more versions or more text changes took longer to publish</span></span>
<span id="cb11-2"><a href="#cb11-2" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb11-3"><a href="#cb11-3" aria-hidden="true" tabindex="-1"></a><span class="va">+ Preprints with more versions or more text changes relative to their published counterpart took longer to publish</span></span></code></pre></div>
<blockquote>
<p>17.) https://hypothes.is/a/z4r5ztLgEeupmPfOPmlVbw
think this is insufficient information.</p>
<p>It should be more clearly highlighted that the NYTAC is proprietary data and it may require a fee of $150-300 to be paid to access, if a non-member of the Linguistic Data Consortium. To say merely “is available upon request” and nothing else is not quite true to my eyes - please warn that it may require payment to access, depending on one’s institutional affiliation (or lack thereof).</p>
</blockquote>
<p><strong>We agree and have updated this text to make the fees accessing NYTAC more apparent.</strong></p>
<div class="sourceCode" id="cb12"><pre class="sourceCode diff"><code class="sourceCode diff"><span id="cb12-1"><a href="#cb12-1" aria-hidden="true" tabindex="-1"></a><span class="st">- Access to the New York Times Annotated Corpus (NYTAC) is available upon request with the Linguistic Data Consortium at [https://catalog.ldc.upenn.edu/LDC2008T19](https://catalog.ldc.upenn.edu/LDC2008T19).</span></span>
<span id="cb12-2"><a href="#cb12-2" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb12-3"><a href="#cb12-3" aria-hidden="true" tabindex="-1"></a><span class="va">+ New York Times Annotated Corpus (NYTAC) can be accessed from the Linguistic Data Consortium at [https://catalog.ldc.upenn.edu/LDC2008T19](https://catalog.ldc.upenn.edu/LDC2008T19) where there may be a \$150 to \$300 fee depending on membership status.</span></span></code></pre></div>
<blockquote>
<p>is using a proprietary data set that charges for access congruent with the PLOS data availability policy?</p>
<p>See: “Please note, if data have been obtained from a third-party source, we require that other researchers would be able to access the data set in the same manner as the authors” https://journals.plos.org/plosone/s/data-availability despite that URL indicating just PLOS ONE, the policy applies to all PLOS journals, unless otherwise noted.</p>
</blockquote>
<p><strong>We believe that using the NYTAC dataset does not violate PLOS’s data availability policy.
We accessed the dataset in accordance with this policy.</strong></p>
<h1 id="reviewer-2">Reviewer #2</h1>
<blockquote>
<p>Overall, I enjoyed this manuscript for offering a way to quantify the transition of preprints to manuscripts within the biological sciences. Further, the authors develop an approach that could also more generally be useful for classifying biomedical literature, and they even provide as an example a web-based program to find potential publication avenues.</p>
</blockquote>
<p><strong>We appreciate the reviewer’s positive sentiment about our work.</strong></p>
<blockquote>
<p>The methodological approach of the authors is quite unexpected. While I do not see a fundamental flaw in their approach, I would anticipate it to be biased toward the most frequent phrases. When performing computational research there is a risk to pursue analyses through well-intended “improvements” or “customizations” whenever the approach does not seem to yield the expected outcome. As some people could be tempted to interpret parts of the analysis of the authors as warning flags for above having happened, I would recommend adding some additional control analyses and explicit statements about their chosen rationales.</p>
<p>Particularly, I would be very curious about the discussion, or main text commenting on why the authors created a custom scheme of classifying documents and their similarity based on vectors of words instead of using existing approaches that provide vectors of documents - including doc2vec that is included in the software package that the authors used for word2vec. Do the results change according to the approach?</p>
</blockquote>
<p><strong>We agree with the reviewer that there are a number of possible approaches to both explore the linguistic landscape of bioRxiv and to predict the ultimate publication venue of preprints.
Our first goal was to examine the linguistic landscape, so we prioritized strategies that led to interpretable embedding spaces.
Our primary strategy was to use word embeddings, which we could use to explore topics and language.
We combined those same word embeddings to produce document embeddings as the average of the word embeddings.
This lets us use the same underlying embeddings through the manuscript from the exploration of language through the journal prediction.
We also agree with the reviewer that Doc2Vec would be likely to produce embeddings that would improve performance on the prediction task.<br />
For this revision, we directly compared both methods for their interpretability and predictive performance.
We find that the Word2Vec model’s first principal component (shown below) was more interpretable than the one from the Doc2Vec model (also below).
On the other hand, we find that the Doc2Vec model’s predictive accuracy is better for the journal classification task (also below).
On balance, we think including multiple embedding models within this paper, which aims to characterize the linguistic landscape of bioRxiv, would be confusing.
For this revision, we have elected to retain the Word2Vec model.
We debated changing the journal prediction server and results to use Doc2Vec, but also considering the feedback from reviewer #1 about the complexity of the paper decided against at this time.
We could easily imagine an improvement to the server in the future that moves to Doc2Vec.</strong></p>
<p><strong>Word2Vec 1st Prinicpal Component WordCloud</strong>
<img src="https://raw.githubusercontent.com/danich1/annorxiver/3bcac4ca2e9249359361525a7e5f3ae36c71d4a5/biorxiv/pca_association_experiment/output/word_pca_similarity/figure_pieces/pca_01_cossim_word_cloud.png" width="680" alt="Word2vec" /></p>
<p><strong>Doc2Vec 1st Prinicpal Component WordCloud</strong>
<img src="https://raw.githubusercontent.com/danich1/annorxiver/3bcac4ca2e9249359361525a7e5f3ae36c71d4a5/biorxiv/pca_association_experiment/output/word_pca_similarity/figure_pieces/pca_01_cossim_word_cloud_doc2vec.png" width="680" alt="Doc2vec" /></p>
<p><strong>Doc2vec vs Word2vec in Journal Recommendations</strong>
<img src="https://raw.githubusercontent.com/danich1/annorxiver/13d9789370900a1d04fb296457fd1faf2d76818a/pmc/journal_recommendation/output/figures/doc2vec_word2vec_comparison.png" width="680" alt="Doc2vec vs Word2vec Journal Recommendation" /></p>
<blockquote>
<p>Further, word2vec often seems to work even better when first trained on a larger corpus before then being applied or transferred to more specialized corpora. Personally, I also made this experience when following an example tutorial provided by the creators of the package that the authors used - which too suggests starting with existing pre-trained models. While the more restricted training done by the authors might have reduced the sensitivity of their approach (… which would likely only strengthen their claims), I would be curious whether there was an additional rationale for avoiding the former strategy that might be missed by readers (e.g.: different meanings such as “abstract” that has different meanings for scientists and non-scientists?).</p>
</blockquote>
<p><strong>We did consider using a pretraining step, but we were worried that then it might be hard to differentiate the effect of pretraining from the text content of bioRxiv.
Since our goal was to explore the linguistic landscape of bioRxiv itself, we elected not to do this.
We agree with the reviewer that if the goal is to enhance the predictive performance of the similarity search webserver, this would be a good step to take.
As with above, we may consider this in a future iteration of the server, but at this point we want to make sure the work continues to characterize the linguistic landscape of bioRxiv.</strong></p>
<blockquote>
<p>Likewise, I’m wondering why the authors used a Euclidean distance for word embeddings instead of a Cosine similarity (which if I recall correctly would also be default in the similarity module of the package which the authors used). Cosine similarity should also allow the authors to make statements about the similarity of words without imposing assumptions on similar text lengths or usage frequencies.</p>
</blockquote>
<p><strong>We used Euclidean distance because it satisfied the triangle inequality, which let us use the sklearn implementation of k-d tree for preprint similarity search.
This allowed us to perform search efficiently with a minimum of development time, and also provided a framework that we could use to shard the search across nodes in the event that we needed to further accelerate performance.
We didn’t find the distance metric to be a substantial driver of performance: preprints and their published counterparts have significantly lower distances regardless of the distance metric used (comparison figure provided below).</strong></p>
<figure>
<img src="https://github.com/greenelab/annorxiver/blob/ff456c5c5cb7cddd16400f6e5b74422e1b33d349/biorxiv/article_distances/output/figures/distance_metrics_top_k_precision.png" width="680" alt="Distance comparison" /><figcaption aria-hidden="true">Distance comparison</figcaption>
</figure>
<blockquote>
<p>Similarly, I was wondering how the “journal-based” approach, which the authors mention briefly against the influence of high publication frequency journals, was implemented. Further, if it could have been avoided by avoiding the Euclidean space.</p>
<p>The mapping of similarity seems to be based on individual pairs of text and as such it would seem vulnerable of shifting distributions (e.g.: if published articles were somewhat different from preprints, as implied in Figure 1A). I would suspect that the authors would be able to improve their performance even further by doing global matching between many pairs (… again see their adherence to a weaker approach as something that ultimately strengthens their findings). Again, a comment on the rationale of their chosen approach could convey additional non-evident considerations.</p>
</blockquote>
<p><strong>While a more optimal distance metric could make some difference in performance, it wouldn’t substantially change the issue of the background frequency whereby some mega journals have published orders of magnitude more manuscripts than others.
We feel that the two approaches together (nearest publications, nearest centroids) provide a reasonable solution that can be effectively integrated into a web interface.
The reviewer’s additional comment about implementing an explicit adjustment for the distributional shift is quite insightful and would likely be a path to improved performance.
We could imagine using something like Orthogonal Procrustes to align the preprint-publication vector spaces before performing search.
We also recently added the capability to automatically update the search indices, which makes implementing this more challenging (we’d want to check the alignments each time), so we didn’t implement this at this time but we agree with the reviewer that this is an important path to explore for improved performance in the future.</strong></p>
<blockquote>
<p>I love the web application!</p>
</blockquote>
<p><strong>We are happy you like the application.</strong></p>
<blockquote>
<p>No statistics are given for the enrichments in Figure 1B-E.</p>
</blockquote>
<p><strong>We report each token with a 95% confidence interval (CI).
The underlying frequency counts are sufficiently large that the confidence intervals are difficult to discern on the plot.
We have adjusted our plot style to make the confidence intervals more apparent.</strong></p>
<blockquote>
<p>I would welcome a supplemental analysis, that removes single letters and special characters from the analysis of Figure 1B-E as they might change the baseline.</p>
</blockquote>
<p><strong>We have added this analysis to our supplement (Supplemental Figures S3 and S4).
We reproduced these figures below.</strong></p>
<p>BioRxiv vs Pubmed Central
Supplemental Figure S3
<img src="https://raw.githubusercontent.com/greenelab/annorxiver/ff456c5c5cb7cddd16400f6e5b74422e1b33d349/biorxiv/corpora_comparison/output/figures/biorxiv_vs_pubmed_central_special_char_removed.png" width="680" /></p>
<p>Preprints vs their Published Counterparts
Supplemental Figure S4
<img src="https://raw.githubusercontent.com/greenelab/annorxiver/ff456c5c5cb7cddd16400f6e5b74422e1b33d349/biorxiv/corpora_comparison/output/figures/preprint_published_comparison_special_char_removed.png" width="680" /></p>
<blockquote>
<p>The word cloud of Figure 2B, C is somewhat nice as it shows the main words. However, this information could also be conveyed in the text. Would personally favor to quantitatively see loadings of first few principal components for different terms.</p>
</blockquote>
<p><strong>We have provided a table in our supplemental section (Supplemental Table 3 and 4).
We also provide these tables below:</strong></p>
<p>Supplementary Table S1: Top and bottom five cosine simularity scores between tokens and the PC1 axis.</p>
<table>
<thead>
<tr class="header">
<th>Cosine Simularity (PC1, word)</th>
<th>word</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td>0.6399154807185836</td>
<td>empirical</td>
</tr>
<tr class="even">
<td>0.5995356000266072</td>
<td>estimates</td>
</tr>
<tr class="odd">
<td>0.5918321530159384</td>
<td>choice</td>
</tr>
<tr class="even">
<td>0.5905550757923625</td>
<td>statistics</td>
</tr>
<tr class="odd">
<td>0.5832932491448216</td>
<td>performance</td>
</tr>
<tr class="even">
<td>0.5803836474390357</td>
<td>accuracy</td>
</tr>
<tr class="odd">
<td>0.5757250459195589</td>
<td>weighting</td>
</tr>
<tr class="even">
<td>0.5753027342288192</td>
<td>estimation</td>
</tr>
<tr class="odd">
<td>0.5730092178610916</td>
<td>uncertainty</td>
</tr>
<tr class="even">
<td>0.5720493442813257</td>
<td>task</td>
</tr>
<tr class="odd">
<td></td>
<td></td>
</tr>
<tr class="even">
<td>-0.4484093198386865</td>
<td>abrogated</td>
</tr>
<tr class="odd">
<td>-0.4490583645152233</td>
<td>transfected</td>
</tr>
<tr class="even">
<td>-0.4500847285921068</td>
<td>incubating</td>
</tr>
<tr class="odd">
<td>-0.4531550791501111</td>
<td>inhibited</td>
</tr>
<tr class="even">
<td>-0.4585422153514687</td>
<td>co-incubated</td>
</tr>
<tr class="odd">
<td>-0.4774721756292901</td>
<td>pre-incubated</td>
</tr>
<tr class="even">
<td>-0.4793057689825842</td>
<td>overexpressing</td>
</tr>
<tr class="odd">
<td>-0.4839313193713342</td>
<td>purified</td>
</tr>
<tr class="even">
<td>-0.4869885872803974</td>
<td>incubated</td>
</tr>
<tr class="odd">
<td>-0.5040798110023075</td>
<td>cultured</td>
</tr>
</tbody>
</table>
<p>Supplementary Table S2: Top and bottom five cosine simularity scores between tokens and the PC2 axis.</p>
<table>
<thead>
<tr class="header">
<th>Cosine Simularity (PC2, word)</th>
<th>word</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td>0.65930201597598</td>
<td>genomic</td>
</tr>
<tr class="even">
<td>0.6333515216782134</td>
<td>genome</td>
</tr>
<tr class="odd">
<td>0.5974018685580009</td>
<td>gene</td>
</tr>
<tr class="even">
<td>0.5796531207938461</td>
<td>genomes</td>
</tr>
<tr class="odd">
<td>0.5353687686155728</td>
<td>annotation</td>
</tr>
<tr class="even">
<td>0.5310140161149529</td>
<td>sequencing</td>
</tr>
<tr class="odd">
<td>0.5197350376908197</td>
<td>sequencesM.</td>
</tr>
<tr class="even">
<td>0.5181781615670665</td>
<td>genome,</td>
</tr>
<tr class="odd">
<td>0.5168781637087506</td>
<td>bioinformatic</td>
</tr>
<tr class="even">
<td>0.513853407439108</td>
<td>WGS</td>
</tr>
<tr class="odd">
<td></td>
<td></td>
</tr>
<tr class="even">
<td>-0.4589201401582101</td>
<td>duration</td>
</tr>
<tr class="odd">
<td>-0.4690482252758019</td>
<td>stimuli</td>
</tr>
<tr class="even">
<td>-0.4712875761979691</td>
<td>amplitudes</td>
</tr>
<tr class="odd">
<td>-0.4772723570301678</td>
<td>contralateral</td>
</tr>
<tr class="even">
<td>-0.4813219679071856</td>
<td>stimulation:</td>
</tr>
<tr class="odd">
<td>-0.4946709932017581</td>
<td>delay</td>
</tr>
<tr class="even">
<td>-0.5111990014804086</td>
<td>stimulus</td>
</tr>
<tr class="odd">
<td>-0.5251288188682695</td>
<td>amplitude</td>
</tr>
<tr class="even">
<td>-0.543586881182879</td>
<td>stimulation</td>
</tr>
<tr class="odd">
<td>-0.5467022203294039</td>
<td>evoked</td>
</tr>
</tbody>
</table>
<blockquote>
<p>The definition of “True matches” could be more explicit in within the main text as the preceding figure 3A could for some people set up a different anticipation.</p>
</blockquote>
<p><strong>We agree this term is too vague, and we have updated the text to be more explicit.</strong></p>
<div class="sourceCode" id="cb13"><pre class="sourceCode diff"><code class="sourceCode diff"><span id="cb13-1"><a href="#cb13-1" aria-hidden="true" tabindex="-1"></a><span class="st">- Approximately 98% of our 200 pairs with an embedding distance in the 0-25th and 25th-50th percentile bins were scored as true matches (Figure 3B).</span></span>
<span id="cb13-2"><a href="#cb13-2" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb13-3"><a href="#cb13-3" aria-hidden="true" tabindex="-1"></a><span class="va">+ Approximately 98% of our 200 pairs with an embedding distance in the 0-25th and 25th-50th percentile bins were successfully matched with their published counterpart (Figure 2B)</span></span></code></pre></div>
<blockquote>
<p>The association given in Figure 4A seems to mainly stem from a few papers with large distances. Would an association be present when using the rank-based Spearman correlation instead of a linear regression? Would, for visualization, a logarithmic relationship describe the data better than a linear one?</p>
</blockquote>
<p><strong>This figure depicts the median half-life publication time for each preprint category within bioRxiv, so we don’t think that it is driven by outliers within categories.
We have adjusted the figure legend to more clearly note what we plotted.</strong></p>
<blockquote>
<p>I believe that the analysis of Figure 4 B is quite clever as it would seem to address the thinkable concern of preprints with no delay and changes mainly stemming from those manuscripts that were already essentially accepted by manuscripts at the time of posting.</p>
</blockquote>
<p><strong>We are glad the reviewer found our analysis insightful.</strong></p>
<blockquote>
<p>The analysis remarks that for the “Preprints in Motion Collection” the relationship between textual distance and time to publication disappears, and supports this through Figure 6E. However, the background trend in figure 6E includes publications that have been published at a time that exceeds a year. Hence a more faithful comparison would be to censor the background data by a distribution of durations that would correspond to the distribution of durations that would be possible for the “Preprint in Motions Collection” (taking distribution corresponding to interval between their dates on bioRxiv and the time at which authors assessed whether manuscripts were published).</p>
</blockquote>
<p><strong>This point is very interesting!
The preprints in motion selection included preprints that were both posted within a specific time interval and then subsequently published.
We think the best comparison set would be with preprints posted on the same dates as preprint in motion preprints, but where the analysis is conducted on the set after enough years have passed for those that will be published to have been published.
Since we don’t have this set, we did conduct the analysis proposed by the reviewer.
The results of this analysis are available in Supplemental Figure S6 (reproduced below).</strong></p>
<figure>
<img src="https://raw.githubusercontent.com/danich1/annorxiver/65ee4a556ab69f2308e5e4d9192905e8cfec3728/figure_generation/output/Figure_S6.png" width="680" alt="Adjusct Background Analysis" /><figcaption aria-hidden="true">Adjusct Background Analysis</figcaption>
</figure>
<blockquote>
<p>Other:</p>
<p>Labels within figures could often be increased in size to improve readability.</p>
</blockquote>
<p><strong>We have updated the size of our labels for this manuscript.</strong></p>
<blockquote>
<p>The methods section briefly comments on some ambiguous cases for the matching. Would these cases be the result of modifications that defy a 1:1 mapping, e.g.: multiple stories getting fused, or one story getting split?</p>
</blockquote>
<p><strong>Out of our small set of disagreements, we encountered a variety of reasons for annotation mismatches.
Some of these cases involved entirely different papers, which are clearly false positives.
Other cases involved title, abstract or main text changes, while the remaining cases consisted of papers sharing similar research topics.
We provide a table of these disagreements below along with short description about each pair.</strong></p>
<table>
<colgroup>
<col style="width: 41%" />
<col style="width: 43%" />
<col style="width: 15%" />
</colgroup>
<thead>
<tr class="header">
<th>biorxiv_doi_url</th>
<th>pmcid_url</th>
<th>Description</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td>https://doi.org/10.1101/413450</td>
<td>https://www.ncbi.nlm.nih.gov/pmc/PMC2967545</td>
<td>Entirely different papers.</td>
</tr>
<tr class="even">
<td>https://doi.org/10.1101/776930</td>
<td>https://www.ncbi.nlm.nih.gov/pmc/PMC6210049</td>
<td>Entirely different papers.</td>
</tr>
<tr class="odd">
<td>https://doi.org/10.1101/2020.01.13.905521</td>
<td>https://www.ncbi.nlm.nih.gov/pmc/PMC4171638</td>
<td>Text changes but same paper.</td>
</tr>
<tr class="even">
<td>https://doi.org/10.1101/352963</td>
<td>https://www.ncbi.nlm.nih.gov/pmc/PMC6116183</td>
<td>Text changes but same paper.</td>
</tr>
<tr class="odd">
<td>https://doi.org/10.1101/513002</td>
<td>https://www.ncbi.nlm.nih.gov/pmc/PMC3545240</td>
<td>Similar aspects of research (liver studies) but different papers.</td>
</tr>
<tr class="even">
<td>https://doi.org/10.1101/680843</td>
<td>https://www.ncbi.nlm.nih.gov/pmc/PMC6379322</td>
<td>Similar aspects of research (taxonomy studies) but different papers.</td>
</tr>
<tr class="odd">
<td>https://doi.org/10.1101/074450</td>
<td>https://www.ncbi.nlm.nih.gov/pmc/PMC5776756</td>
<td>Significant text changes but arguably same paper.</td>
</tr>
<tr class="even">
<td>https://doi.org/10.1101/530758</td>
<td>https://www.ncbi.nlm.nih.gov/pmc/PMC6663035</td>
<td>Significant text changes but same paper.</td>
</tr>
</tbody>
</table>
<blockquote>
<p>The results of Figure 2A could possibly be strengthened by avoiding Principal Components and replacing them by UMAP projects to account for non-linearity.</p>
</blockquote>
<p><strong>We chose to use PCA for figure 2A as our goal was to visually highlight the concepts captured by our generated principal components.
We did also perform a UMAP embedding, which we included in <a href="https://github.com/greenelab/annorxiver/blob/b120488de2c197d93072678f80fce68d11d9e08d/biorxiv/word_vector_experiment/02_biorxiv_visualize_embeddings.ipynb">this notebook</a> but not the manuscript itself.</strong></p>
<blockquote>
<p>Although peripheral to the current manuscript, their approach and data would also seem capable to providing an update-able map of the biomedical sciences, by applying their approach of Figure 2 to the PMC corpus data which the authors access too. Such a map could be interesting for those trying to obtain an overview about biology. In case that the authors do not hold plans to publish this elsewhere, and in case that it would be less than a day of work, I would recommend adding such a map to the supplement or as a web service.</p>
</blockquote>
<p><strong>Our web application provides 2D visualization of PMC’s open-access corpus.
This visualization uses SAUCIE, an autoencoder designed for RNA-seq, instead of UMAP or PCA to generate the landscape.
We also constructed an auto-updater pipeline for this tool, incorporating new papers into our website and visualization every month.
We think that the map is now in place in our server, and that others could produce their own map using either our API or the underlying SAUCIE models.
Code to train our SAUCIE model can be found in <a href="https://github.com/greenelab/annorxiver/blob/master/pmc/journal_recommendation/03_journal_recommendation_figure_generator.ipynb">this notebook</a> and our fully trained model can be found using <a href="https://github.com/greenelab/preprint-similarity-search/tree/master/server/saucie_model">this link</a>.</strong></p>
<blockquote>
<p>Are the few publications in Figure 2A, which lie outside of the space that is generally occupied by their respective article categories, somewhat different when doing a superficial manual inspection (e.g.: misclassified by authors, or interdisciplinary research)</p>
</blockquote>
<p><strong>We sampled a select number of preprints from the neuroscience and bioinformatic category that were closer to the left side of the figure (PC1 &lt;= -2.5 and -2 &lt; PC2 &lt; 2).
We found that these outliers mainly consisted of interdisciplinary research (e.g., a bioinformatic paper analyzing fluorescence micrographs or a cell biology approach used to explore a neuroscience concept).
We provide a table below of preprint DOIs that fall into this situation.</strong></p>
<figure>
<img src="https://raw.githubusercontent.com/greenelab/annorxiver/51e602d9c4863e1314ed5f01bc483901894ed705/biorxiv/pca_association_experiment/output/pca_plots/svg_files/scatterplot_files/pca01_v_pca02_reversed.svg" alt="PCA plot" /><figcaption aria-hidden="true">PCA plot</figcaption>
</figure>
<table>
<thead>
<tr class="header">
<th>doi</th>
<th>document category</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td>10.1101/075440</td>
<td>bioinformatics</td>
</tr>
<tr class="even">
<td>10.1101/806216</td>
<td>bioinformatics</td>
</tr>
<tr class="odd">
<td>10.1101/696625</td>
<td>bioinformatics</td>
</tr>
<tr class="even">
<td>10.1101/835181</td>
<td>bioinformatics</td>
</tr>
<tr class="odd">
<td>10.1101/583187</td>
<td>bioinformatics</td>
</tr>
<tr class="even">
<td>10.1101/610196</td>
<td>neuroscience</td>
</tr>
<tr class="odd">
<td>10.1101/2020.01.08.898080</td>
<td>neuroscience</td>
</tr>
<tr class="even">
<td>10.1101/664557</td>
<td>neuroscience</td>
</tr>
<tr class="odd">
<td>10.1101/655498</td>
<td>neuroscience</td>
</tr>
<tr class="even">
<td>10.1101/244111</td>
<td>neuroscience</td>
</tr>
</tbody>
</table>
<blockquote>
<p>Adding a few words to “examining the top five and bottom five preprints” could avoid misunderstanding (e.g.: while I suspect that it is the position in Figure 2A, I was first thinking about the most/least successful/downloaded…)</p>
</blockquote>
<p><strong>We have updated the text to be more explicit.</strong></p>
<div class="sourceCode" id="cb14"><pre class="sourceCode diff"><code class="sourceCode diff"><span id="cb14-1"><a href="#cb14-1" aria-hidden="true" tabindex="-1"></a><span class="st">- Examining the top five and bottom five preprints within the systems biology field reinforces PC1&#39;s dichotomous theme (Table ...).</span></span>
<span id="cb14-2"><a href="#cb14-2" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb14-3"><a href="#cb14-3" aria-hidden="true" tabindex="-1"></a><span class="va">+ Examining the top five highest-scoring and bottom five lowest-scoring systems biology preprints along PC1 reinforces its dichotomous theme (Supplementary Table ...).</span></span></code></pre></div>
<blockquote>
<p>The vector representation of words and documents should allow the authors to quantify shifts that appear between preprints and published manuscripts. Though not necessary from my perspective, many interesting analyses could be done in vector space (e.g.: does language get more positive, or start to refer to more established concepts…?). Maybe there is something small that could be done. Alternatively, the discussion could possibly be extended to demonstrate the implications of vector space, and thus their own work, for future research into preprints and peer review.</p>
</blockquote>
<p><strong>Thank you for pointing out potential extensions to our vector space.
We incorporated these suggestions into our discussion/conclusion section.</strong></p>
<div class="sourceCode" id="cb15"><pre class="sourceCode diff"><code class="sourceCode diff"><span id="cb15-1"><a href="#cb15-1" aria-hidden="true" tabindex="-1"></a><span class="va">+ This embedding space could also be used to quantify sentiment trends or other linguistic features.</span></span>
<span id="cb15-2"><a href="#cb15-2" aria-hidden="true" tabindex="-1"></a><span class="va">+ Furthermore, methodologies for uncovering latent scientific knowledge [...] may be applicable in this embedding space.</span></span></code></pre></div>
<blockquote>
<p>Along above, the discussion could be extended toward prior uses of Word2vec in the studies of science, such as Tshitoyan et al. Nature 2019.</p>
</blockquote>
<p><strong>We have added this citation into our discussion.</strong></p>
<div class="sourceCode" id="cb16"><pre class="sourceCode diff"><code class="sourceCode diff"><span id="cb16-1"><a href="#cb16-1" aria-hidden="true" tabindex="-1"></a><span class="va">+ Furthermore, methodologies for uncovering latent scientific knowledge [...] may be applicable in this embedding space.</span></span></code></pre></div>
<blockquote>
<p>Repeating the link to the web app in the main text would be convenient.</p>
</blockquote>
<p><strong>We have updated the text to include the web app link throughout our website section.</strong></p>
<blockquote>
<p>Seeing Figure 6D and 6E, I would enjoy the authors showing or discussing more explicitly, whether textual differences increase with the number of revisions (and/or if there were some more complex changes such as reversions to earlier versions).</p>
</blockquote>
<p><strong>We agree with the reviewer that this would be interesting to examine.
We performed a linear regression analysis to examine relationships between preprint version counts and the amount of change using all preprint-published pairs within bioRxiv.
We found a small positive slope between version count and document distance (see below), but given the caveats involved with respect to small sample size at the extremes we elected not to include this analysis in the revised manuscript.</strong></p>
<figure>
<img src="https://raw.githubusercontent.com/danich1/annorxiver/bc7761c4db132f1287205680a1e2e5f220a85755/biorxiv/publication_delay_experiment/output/version_count_doc_distances.png" width="680" alt="Version Count vs Document distance Linear regression" /><figcaption aria-hidden="true">Version Count vs Document distance Linear regression</figcaption>
</figure>
<h1 id="reviewer-3">Reviewer #3</h1>
<blockquote>
<p>This study asks an important question: (how) do preprints change between their initial release on a preprint server and their eventual publication in a peer-reviewed journal? While the analysis of the linguistic changes doesn’t reveal anything particularly exciting (mostly typesetting and references to supplementary information included in response to reviewer requests), this is an incredibly useful result in demonstrating that preprints are typically of high quality, which has broad implications for how researchers and their work are assessed in career, funding, and publishing decisions. The authors have developed some very promising deliverables based on document embeddings that should be broadly applicable to readers, authors, journal editors, and other stakeholders navigating the complex landscape of preprinted and published literature.</p>
</blockquote>
<p><strong>We appreciate the reviewer’s positive comments on the value of our manuscript.
We agree that the linguistic changes aren’t particularly exciting and with the reviewer’s sentiment that that finding, in itself, is exciting.</strong></p>
<blockquote>
<p>Major Comments:</p>
<p>The method for discovering unannotated preprint-publication relationships is very neat, but I imagine it’s rather unwieldy to match a novel publication against the full-text bioRxiv corpus in downstream applications (e.g., bioRxiv’s automation)–could this be optimized by reducing the search space to preprints that share some or all of the same authors, within a reasonable date range, and/or only considering paper/preprint metadata (e.g., abstract, title, references)? Such an approach might also enable annotation of preprints that are eventually published as non-OA peer reviewed articles for which such metadata are available.</p>
</blockquote>
<p><strong>We investigated this based on the reviewer’s question.
Our results suggest that it is likely to be feasible to identify preprint-published pairs using abstracts alone.
We generated document embeddings using solely abstracts and calculated distances between known preprint-published pairs and preprints with a randomly sampled article from the same journal.
We found that the ranking by abstract distances was slightly better than full text (figure below) for matching preprints with their published pair.
This indicates that abstracts can be used to establish preprint and published links, and while the evidence is relatively weak (as the difference between full text and abstract is small) it suggests that perhaps abstracts undergo less change than full text.</strong></p>
<figure>
<img src="https://raw.githubusercontent.com/greenelab/annorxiver/ff456c5c5cb7cddd16400f6e5b74422e1b33d349/biorxiv/article_distances/output/figures/abstract_vs_full_text_top_k_precision.png" width="680" alt="Abstracts vs Full Text" /><figcaption aria-hidden="true">Abstracts vs Full Text</figcaption>
</figure>
<blockquote>
<p>Section “Building Classifiers to Detect Linguistically Similar Journal Venues and Published Articles”:</p>
<p>“Specific journals publish articles in a focused topic area, while others publish articles that cover many topics. Likewise, some journals have a publication rate of at most hundreds of papers per year, while others publish at a rate of at least ten thousand papers per year. Accounting for these characteristics, we designed two approaches - one centered on manuscripts and another centered on journals.” &lt;&lt; this could use some unpacking and/or reorganizing of details found later in this section–as I understand it, the variation in journals’ topical breadth motivates the development of a manuscript-focused classifier (so that topically similar papers appearing in generalist journals do not get obscured) and the variation in journals’ publication rates motivates a journal-focused classifier (so that high-output journals do not overwhelm more selective or less popular journals).</p>
</blockquote>
<p><strong>We agree with the reviewer that this section was unduly dense, and we have revised the manuscript to more clearly unpack this explanation.</strong></p>
<div class="sourceCode" id="cb17"><pre class="sourceCode diff"><code class="sourceCode diff"><span id="cb17-1"><a href="#cb17-1" aria-hidden="true" tabindex="-1"></a><span class="va">+ Training models to identify which journal publishes similar articles is challenging as not all journals are the same. </span></span>
<span id="cb17-2"><a href="#cb17-2" aria-hidden="true" tabindex="-1"></a><span class="va">+ Some journals have a publication rate of at most hundreds of papers per year, while others publish at a rate of at least ten thousand papers per year.</span></span>
<span id="cb17-3"><a href="#cb17-3" aria-hidden="true" tabindex="-1"></a><span class="va">+ Furthermore, some journals focus on publishing articles within a concentrated topic area, while others cover many dispersive topics.</span></span>
<span id="cb17-4"><a href="#cb17-4" aria-hidden="true" tabindex="-1"></a><span class="va">+ Therefore, we designed two approaches to account for these characteristics.</span></span>
<span id="cb17-5"><a href="#cb17-5" aria-hidden="true" tabindex="-1"></a><span class="va">+ Our first approach focuses on articles that account for a journal&#39;s variation of publication topics.</span></span>
<span id="cb17-6"><a href="#cb17-6" aria-hidden="true" tabindex="-1"></a><span class="va">+ This approach allows for topically similar papers to be retrieved independently of their respective journal.</span></span>
<span id="cb17-7"><a href="#cb17-7" aria-hidden="true" tabindex="-1"></a><span class="va">+ Our second approach is centered on journals to account for varying publication rates.</span></span>
<span id="cb17-8"><a href="#cb17-8" aria-hidden="true" tabindex="-1"></a><span class="va">+ This approach allows more selective or less popular journals to have equal representation to their high publishing counterparts.</span></span></code></pre></div>
<blockquote>
<p>I’m also curious how often these two classifiers agree–are the top matching papers typically found in the top matching journals? In cases where the two classifiers tend to disagree, are there any common characteristics of the preprints the application is trying to classify?</p>
</blockquote>
<p><strong>We evaluated our classifier agreement by calculating the overlap coefficient, which is designed to measure the overlap between two sets.
We randomly sampled 1700 out of 20232 known preprint-published pairs from our test dataset.
We generated ten recommendations from our centroid classifier and ten unique journal recommendations from our paper-paper model for every sampled pair (as the paper classifier can return papers from the same journal, this means we are examining ten or more manuscripts until we reach ten unique journals).
This resulted in an average overlap coefficient of 0.21.
Along with this calculation, we generated baselines for each model.
Our first baseline was designed for our journal centroid model.
We randomly sampled ten journals without replacement for each preprint-published pair.
We compared this random listing against our original journal centroid recommendation list and found an average score of 0.0184.
Our other baseline was designed for our paper-paper classifier.
We randomly sampled without filtering ten unique journals for each preprint-published pair and compared this sample to the original paper-paper recommendation list.
This baseline overlap coefficient was 0.009.
Our takeaway from this analysis is that both approaches agree much more than they would due to random overlap, but the overlap coefficient remains modest.
Because of the relatively large number of discrepancies between the resulting sets, we were not able to identify a practical way to answer the characteristics of preprints that led to differences vs common predictions.</strong></p>
<blockquote>
<p>Minor Comments (by section):</p>
<p>Introduction:</p>
<p>The references of text mining on biomedical corpora should include Desai et al (2018) [https://www.biorxiv.org/content/10.1101/333922v1.abstract], which describes a similar recommendation engine.</p>
</blockquote>
<p><strong>We agree and have added this reference.</strong></p>
<div class="sourceCode" id="cb18"><pre class="sourceCode diff"><code class="sourceCode diff"><span id="cb18-1"><a href="#cb18-1" aria-hidden="true" tabindex="-1"></a><span class="st">- Textual analysis uses linguistic, statistical, and machine learning techniques to analyze and extract information from text [...].</span></span>
<span id="cb18-2"><a href="#cb18-2" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb18-3"><a href="#cb18-3" aria-hidden="true" tabindex="-1"></a><span class="va">+ Textual analysis uses linguistic, statistical, and machine learning techniques to analyze and extract information from text [...].</span></span></code></pre></div>
<blockquote>
<p>Section “Comparing Corpora”:</p>
<p>Inconsistent formatting of “spaCy”</p>
</blockquote>
<p><strong>We updated our manuscript to make sure spaCy is formatted correctly.</strong></p>
<div class="sourceCode" id="cb19"><pre class="sourceCode diff"><code class="sourceCode diff"><span id="cb19-1"><a href="#cb19-1" aria-hidden="true" tabindex="-1"></a><span class="st">- Spacy is a lightweight and easy-to-use python package designed to preprocess and filter text [@spacy2].</span></span>
<span id="cb19-2"><a href="#cb19-2" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb19-3"><a href="#cb19-3" aria-hidden="true" tabindex="-1"></a><span class="va">+ SpaCy is a lightweight and easy-to-use python package designed to preprocess and filter text [@spacy2].</span></span></code></pre></div>
<blockquote>
<p>Define “stopwords,” since many readers may be unfamiliar with this term</p>
</blockquote>
<p><strong>We added an explanation of “stopwords” to the manuscript.</strong></p>
<div class="sourceCode" id="cb20"><pre class="sourceCode diff"><code class="sourceCode diff"><span id="cb20-1"><a href="#cb20-1" aria-hidden="true" tabindex="-1"></a><span class="va">+ All corpora contain multiple words that do not have any meaning (e.g. conjunctions, prepositions, etc.) or occur with a high frequency.</span></span>
<span id="cb20-2"><a href="#cb20-2" aria-hidden="true" tabindex="-1"></a><span class="va">+ These words are termed stopwords and are often removed to improve text processing pipelines.</span></span></code></pre></div>
<blockquote>
<p>Section “Constructing a Document Representation for Life Sciences Text”:</p>
<p>This switches back to using “words” instead of “tokens” as in the previous section</p>
</blockquote>
<p><strong>Thank you for pointing out this inconsistency.
We have carefully edited our manuscript to make sure “tokens” was consistently used instead of “words”.</strong></p>
<blockquote>
<p>Section “Measuring Time Duration for Preprint Publication Process”:</p>
<p>Does this include the new preprint-publication pairs discovered in the previous section, or only those annotated in the data provided by the bioRxiv API?</p>
</blockquote>
<p><strong>This section only contains preprints pairs that have been established before our document matching approach.</strong></p>
<blockquote>
<p>Section “Preprints with more versions or more text changes took longer to publish”:</p>
<p>Fig. 4: can the longer publication times for scicomm/education papers (Fig 4a) be explained by a tendency to go through more versions (Fig 4b)?</p>
</blockquote>
<p><strong>We investigated this.
Unfortunately, we were not able to provide a detailed answer to this question as most published articles in this category weren’t contained in Pubmed Central’s Open Access Corpus (PMCOA).
We do provide a table of preprints that have a matching counterpart in PMCOA in our supplemental files, which provides the ingredients for further investigations.</strong></p>
<blockquote>
<p>It might be worthwhile to explore what happens post-publication to papers that go through more preprint revisions and take longer to publish, as this could have practical implications for authors as they decide when/if to submit/revise their preprints. Do these papers ultimately receive more citations, end up in journals with higher impact factors, or receive more attention on social media?</p>
</blockquote>
<p><strong>We agree that this is an interesting question.
We thought that this work would be outside the scope of this manuscript, but we wanted to make it as easy as possible for this to be tackled in the future.
We provide a supplemental file (published_preprints_information.xlsx) containing preprints and corresponding publication to enable these future studies.</strong></p>
<blockquote>
<p>Section “Preprints with similar document embeddings share publication venues”:</p>
<p>From personal experience, converting bioRxiv PDFs to text sometimes introduces weird noise and artifacts. Since bioRxiv and medRxiv both offer full-text HTML for many (if not all?) articles, is it possible to modify the application to use this cleaner data source?</p>
</blockquote>
<p><strong>At the time of submission, we recognized that using XML was better than solely relying on a pdf parser.
We have now updated the webserver to attempt to retrieve the XML version first, then resort to the pdf parser if the XML version is unavailable.</strong></p>
<blockquote>
<p>Section “Contextualizing the Preprints in Motion Collection”:</p>
<p>Figure description for Fig 6E is mislabeled as D</p>
</blockquote>
<p><strong>We updated our text to fix this label mismatch.</strong></p>
<blockquote>
<p>There are several casually/awkwardly-worded or grammatically incorrect sentences throughout that could use some finesse:</p>
<p>Introduction:</p>
<p>“We hypothesize that preprints and biomedical text are pretty similar…”</p>
<p>Measuring Time Duration for Preprint Publication Process:</p>
<p>“Preprints that are published can take varying amounts of time to be published.”</p>
<p>“We accomplish this by first randomly sampled with replacement a pair of preprints…”</p>
<p>Building Classifiers to Detect Linguistically Similar Journal Venues and Published Articles:</p>
<p>“Preprints are more likely to be published in journals that contained similar content to work in question.”</p>
<p>Web Application for Discovering Similar Preprints and Journals:</p>
<p>“The application downloads a pdf version of any preprint hosted on the bioRxiv or medRxiv server uses PyMuPDF to extract text from the downloaded pdf and feeds the extracted text into our CBOW model to construct a document embedding representation.”</p>
<p>Preprints with more versions or more text changes took longer to publish:</p>
<p>“Each new version adds additional 51 days before a preprint is published.”</p>
</blockquote>
<p><strong>We have updated these sentences in the manuscript.
We thank the reviewer for providing these corrections.</strong></p>
<div class="sourceCode" id="cb21"><pre class="sourceCode diff"><code class="sourceCode diff"><span id="cb21-1"><a href="#cb21-1" aria-hidden="true" tabindex="-1"></a><span class="st">- We hypothesize that preprints and biomedical text are pretty similar, especially when controlling for the differential uptake of preprints across fields.</span></span>
<span id="cb21-2"><a href="#cb21-2" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb21-3"><a href="#cb21-3" aria-hidden="true" tabindex="-1"></a><span class="va">+ We hypothesize that preprints and biomedical text will appear to have similar characteristics, especially when controlling for the differential uptake of preprints across fields.</span></span></code></pre></div>
<div class="sourceCode" id="cb22"><pre class="sourceCode diff"><code class="sourceCode diff"><span id="cb22-1"><a href="#cb22-1" aria-hidden="true" tabindex="-1"></a><span class="st">- Preprints that are published can take varying amounts of time to be published.</span></span>
<span id="cb22-2"><a href="#cb22-2" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb22-3"><a href="#cb22-3" aria-hidden="true" tabindex="-1"></a><span class="va">+ Preprints can take varying amounts of time to be published.</span></span></code></pre></div>
<div class="sourceCode" id="cb23"><pre class="sourceCode diff"><code class="sourceCode diff"><span id="cb23-1"><a href="#cb23-1" aria-hidden="true" tabindex="-1"></a><span class="st">- We accomplish this by first randomly sampled with replacement a pair of preprints from the Bioinformatics topic area as this was well represented within bioRxiv and contains a diverse set of research articles.</span></span>
<span id="cb23-2"><a href="#cb23-2" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb23-3"><a href="#cb23-3" aria-hidden="true" tabindex="-1"></a><span class="va">+ We first randomly sampled with replacement a pair of preprints from the Bioinformatics topic area as this was well represented within bioRxiv and contains a diverse set of research articles.</span></span></code></pre></div>
<div class="sourceCode" id="cb24"><pre class="sourceCode diff"><code class="sourceCode diff"><span id="cb24-1"><a href="#cb24-1" aria-hidden="true" tabindex="-1"></a><span class="st">- Preprints are more likely to be published in journals that contained similar content to work in question.</span></span>
<span id="cb24-2"><a href="#cb24-2" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb24-3"><a href="#cb24-3" aria-hidden="true" tabindex="-1"></a><span class="va">+ Preprints are more likely to be published in journals that publish articles with similar content.</span></span></code></pre></div>
<div class="sourceCode" id="cb25"><pre class="sourceCode diff"><code class="sourceCode diff"><span id="cb25-1"><a href="#cb25-1" aria-hidden="true" tabindex="-1"></a><span class="st">- The application downloads a pdf version of any preprint hosted on the bioRxiv or medRxiv server uses PyMuPDF [...] to extract text from the downloaded pdf and feeds the extracted text into our CBOW model to construct a document embedding representation.</span></span>
<span id="cb25-2"><a href="#cb25-2" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb25-3"><a href="#cb25-3" aria-hidden="true" tabindex="-1"></a><span class="va">+ Our application attempts to download the full text xml version of any preprint hosted on the bioRxiv or medRxiv server and uses the lxml package (version num) to extract text.</span></span>
<span id="cb25-4"><a href="#cb25-4" aria-hidden="true" tabindex="-1"></a><span class="va">+ If the xml version isn&#39;t available our application defaults to downloading the pdf version and uses PyMuPDF [...] to extract text from the pdf. </span></span>
<span id="cb25-5"><a href="#cb25-5" aria-hidden="true" tabindex="-1"></a><span class="va">+ The extracted text is fed into our CBOW model to construct a document embedding representation.</span></span></code></pre></div>
<div class="sourceCode" id="cb26"><pre class="sourceCode diff"><code class="sourceCode diff"><span id="cb26-1"><a href="#cb26-1" aria-hidden="true" tabindex="-1"></a><span class="st">- Each new version adds additional 51 days before a preprint is published.</span></span>
<span id="cb26-2"><a href="#cb26-2" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb26-3"><a href="#cb26-3" aria-hidden="true" tabindex="-1"></a><span class="va">+ Every additional preprint version was associated with an increase of 51 days before a preprint was published.</span></span></code></pre></div>
<!-- default theme -->

<style>
  /* import google fonts */
  @import url("https://fonts.googleapis.com/css?family=Open+Sans:400,600,700");
  @import url("https://fonts.googleapis.com/css?family=Source+Code+Pro");

  /* -------------------------------------------------- */
  /* global */
  /* -------------------------------------------------- */

  /* all elements */
  * {
    /* force sans-serif font unless specified otherwise */
    font-family: "Open Sans", "Helvetica", sans-serif;

    /* prevent text inflation on some mobile browsers */
    -webkit-text-size-adjust: none !important;
    -moz-text-size-adjust: none !important;
    -o-text-size-adjust: none !important;
    text-size-adjust: none !important;
  }

  @media only screen {
    /* "page" element */
    body {
      position: relative;
      box-sizing: border-box;
      font-size: 12pt;
      line-height: 1.5;
      max-width: 8.5in;
      margin: 20px auto;
      padding: 40px;
      border-radius: 5px;
      border: solid 1px #bdbdbd;
      box-shadow: 0 0 20px rgba(0, 0, 0, 0.05);
      background: #ffffff;
    }
  }

  /* when on screen < 8.5in wide */
  @media only screen and (max-width: 8.5in) {
    /* "page" element */
    body {
      padding: 20px;
      margin: 0;
      border-radius: 0;
      border: none;
      box-shadow: 0 0 20px rgba(0, 0, 0, 0.05) inset;
      background: none;
    }
  }

  /* -------------------------------------------------- */
  /* headings */
  /* -------------------------------------------------- */

  /* all headings */
  h1,
  h2,
  h3,
  h4,
  h5,
  h6 {
    margin: 20px 0;
    padding: 0;
    font-weight: bold;
  }

  /* biggest heading */
  h1 {
    margin: 40px 0;
    text-align: center;
  }

  /* second biggest heading */
  h2 {
    margin-top: 30px;
    padding-bottom: 5px;
    border-bottom: solid 1px #bdbdbd;
  }

  /* heading font sizes */
  h1 {
    font-size: 2em;
  }
  h2 {
    font-size: 1.5em;
  }
  h3 {
    font-size: 1.35em;
  }
  h4 {
    font-size: 1.25em;
  }
  h5 {
    font-size: 1.15em;
  }
  h6 {
    font-size: 1em;
  }

  /* -------------------------------------------------- */
  /* manuscript header */
  /* -------------------------------------------------- */

  /* manuscript title */
  header > h1 {
    margin: 0;
  }

  /* manuscript title caption text (ie "automatically generated on") */
  header + p {
    text-align: center;
    margin-top: 10px;
  }

  /* -------------------------------------------------- */
  /* text elements */
  /* -------------------------------------------------- */

  /* links */
  a {
    color: #2196f3;
    overflow-wrap: break-word;
  }

  /* superscripts and subscripts */
  sub,
  sup {
    /* prevent from affecting line height */
    line-height: 0;
  }

  /* unordered and ordered lists*/
  ul,
  ol {
    padding-left: 20px;
  }

  /* class for styling text semibold */
  .semibold {
    font-weight: 600;
  }

  /* class for styling elements horizontally left aligned */
  .left {
    display: block;
    text-align: left;
    margin-left: auto;
    margin-right: 0;
    justify-content: left;
  }

  /* class for styling elements horizontally centered */
  .center {
    display: block;
    text-align: center;
    margin-left: auto;
    margin-right: auto;
    justify-content: center;
  }

  /* class for styling elements horizontally right aligned */
  .right {
    display: block;
    text-align: right;
    margin-left: 0;
    margin-right: auto;
    justify-content: right;
  }

  /* -------------------------------------------------- */
  /* section elements */
  /* -------------------------------------------------- */

  /* horizontal divider line */
  hr {
    border: none;
    height: 1px;
    background: #bdbdbd;
  }

  /* paragraphs, horizontal dividers, figures, tables, code */
  p,
  hr,
  figure,
  table,
  pre {
    /* treat all as "paragraphs", with consistent vertical margins */
    margin-top: 20px;
    margin-bottom: 20px;
  }

  /* -------------------------------------------------- */
  /* figures */
  /* -------------------------------------------------- */

  /* figure */
  figure {
    max-width: 100%;
    margin-left: auto;
    margin-right: auto;
  }

  /* figure caption */
  figcaption {
    padding: 0;
    padding-top: 10px;
  }

  /* figure image element */
  figure > img,
  figure > svg {
    max-width: 100%;
    display: block;
    margin-left: auto;
    margin-right: auto;
  }

  /* figure auto-number */
  img + figcaption > span:first-of-type,
  svg + figcaption > span:first-of-type {
    font-weight: bold;
    margin-right: 5px;
  }

  /* -------------------------------------------------- */
  /* tables */
  /* -------------------------------------------------- */

  /* table */
  table {
    border-collapse: collapse;
    border-spacing: 0;
    width: 100%;
    margin-left: auto;
    margin-right: auto;
  }

  /* table cells */
  th,
  td {
    border: solid 1px #bdbdbd;
    padding: 10px;
    /* squash table if too wide for page by forcing line breaks */
    overflow-wrap: break-word;
    word-break: break-word;
  }

  /* header row and even rows */
  th,
  tr:nth-child(2n) {
    background-color: #fafafa;
  }

  /* odd rows */
  tr:nth-child(2n + 1) {
    background-color: #ffffff;
  }

  /* table caption */
  caption {
    text-align: left;
    padding: 0;
    padding-bottom: 10px;
  }

  /* table auto-number */
  table > caption > span:first-of-type {
    font-weight: bold;
    margin-right: 5px;
  }

  /* -------------------------------------------------- */
  /* code */
  /* -------------------------------------------------- */

  /* multi-line code block */
  pre {
    padding: 10px;
    background-color: #eeeeee;
    color: #000000;
    border-radius: 5px;
    break-inside: avoid;
    text-align: left;
  }

  /* inline code, ie code within normal text */
  :not(pre) > code {
    padding: 0 4px;
    background-color: #eeeeee;
    color: #000000;
    border-radius: 5px;
  }

  /* code text */
  /* apply all children, to reach syntax highlighting sub-elements */
  code,
  code * {
    /* force monospace font */
    font-family: "Source Code Pro", "Courier New", monospace;
  }

  /* -------------------------------------------------- */
  /* quotes */
  /* -------------------------------------------------- */

  /* quoted text */
  blockquote {
    margin: 0;
    padding: 0;
    border-left: 4px solid #bdbdbd;
    padding-left: 16px;
    break-inside: avoid;
  }

  /* -------------------------------------------------- */
  /* banners */
  /* -------------------------------------------------- */

  /* info banners */
  .banner {
    box-sizing: border-box;
    display: block;
    position: relative;
    width: 100%;
    margin-top: 20px;
    margin-bottom: 20px;
    padding: 20px;
    text-align: center;
  }

  /* paragraph in banner */
  .banner > p {
    margin: 0;
  }

  /* -------------------------------------------------- */
  /* highlight colors */
  /* -------------------------------------------------- */

  .white {
    background: #ffffff;
  }
  .lightgrey {
    background: #eeeeee;
  }
  .grey {
    background: #757575;
  }
  .darkgrey {
    background: #424242;
  }
  .black {
    background: #000000;
  }
  .lightred {
    background: #ffcdd2;
  }
  .lightyellow {
    background: #ffecb3;
  }
  .lightgreen {
    background: #dcedc8;
  }
  .lightblue {
    background: #e3f2fd;
  }
  .lightpurple {
    background: #f3e5f5;
  }
  .red {
    background: #f44336;
  }
  .orange {
    background: #ff9800;
  }
  .yellow {
    background: #ffeb3b;
  }
  .green {
    background: #4caf50;
  }
  .blue {
    background: #2196f3;
  }
  .purple {
    background: #9c27b0;
  }
  .white,
  .lightgrey,
  .lightred,
  .lightyellow,
  .lightgreen,
  .lightblue,
  .lightpurple,
  .orange,
  .yellow,
  .white a,
  .lightgrey a,
  .lightred a,
  .lightyellow a,
  .lightgreen a,
  .lightblue a,
  .lightpurple a,
  .orange a,
  .yellow a {
    color: #000000;
  }
  .grey,
  .darkgrey,
  .black,
  .red,
  .green,
  .blue,
  .purple,
  .grey a,
  .darkgrey a,
  .black a,
  .red a,
  .green a,
  .blue a,
  .purple a {
    color: #ffffff;
  }

  /* -------------------------------------------------- */
  /* buttons */
  /* -------------------------------------------------- */

  /* class for styling links like buttons */
  .button {
    display: inline-flex;
    justify-content: center;
    align-items: center;
    margin: 5px;
    padding: 10px 20px;
    font-size: 0.75em;
    font-weight: 600;
    text-transform: uppercase;
    text-decoration: none;
    letter-spacing: 1px;
    background: none;
    color: #2196f3;
    border: solid 1px #bdbdbd;
    border-radius: 5px;
  }

  /* buttons when hovered */
  .button:hover:not([disabled]),
  .icon_button:hover:not([disabled]) {
    cursor: pointer;
    background: #f5f5f5;
  }

  /* buttons when disabled */
  .button[disabled],
  .icon_button[disabled] {
    opacity: 0.35;
    pointer-events: none;
  }

  /* class for styling buttons containg only single icon */
  .icon_button {
    display: inline-flex;
    justify-content: center;
    align-items: center;
    text-decoration: none;
    margin: 0;
    padding: 0;
    background: none;
    border-radius: 5px;
    border: none;
    width: 20px;
    height: 20px;
    min-width: 20px;
    min-height: 20px;
  }

  /* icon button inner svg image */
  .icon_button > svg {
    height: 16px;
  }

  /* -------------------------------------------------- */
  /* icons */
  /* -------------------------------------------------- */

  /* class for styling icons inline with text */
  .inline_icon {
    height: 1em;
    position: relative;
    top: 0.125em;
  }

  /* -------------------------------------------------- */
  /* references */
  /* -------------------------------------------------- */

  .csl-entry {
    margin-top: 15px;
    margin-bottom: 15px;
  }

  /* -------------------------------------------------- */
  /* print control */
  /* -------------------------------------------------- */

  @media print {
    @page {
      /* suggested printing margin */
      margin: 0.5in;
    }

    /* document and "page" elements */
    html,
    body {
      margin: 0;
      padding: 0;
      width: 100%;
      height: 100%;
    }

    /* "page" element */
    body {
      font-size: 11pt !important;
      line-height: 1.35;
    }

    /* all headings */
    h1,
    h2,
    h3,
    h4,
    h5,
    h6 {
      margin: 15px 0;
    }

    /* figures and tables */
    figure,
    table {
      font-size: 0.85em;
    }

    /* table cells */
    th,
    td {
      padding: 5px;
    }

    /* shrink font awesome icons */
    i.fas,
    i.fab,
    i.far,
    i.fal {
      transform: scale(0.85);
    }

    /* decrease banner margins */
    .banner {
      margin-top: 15px;
      margin-bottom: 15px;
      padding: 15px;
    }

    /* class for centering an element vertically on its own page */
    .page_center {
      margin: auto;
      width: 100%;
      height: 100%;
      display: flex;
      align-items: center;
      vertical-align: middle;
      break-before: page;
      break-after: page;
    }

    /* always insert a page break before the element */
    .page_break_before {
      break-before: page;
    }

    /* always insert a page break after the element */
    .page_break_after {
      break-after: page;
    }

    /* avoid page break before the element */
    .page_break_before_avoid {
      break-before: avoid;
    }

    /* avoid page break after the element */
    .page_break_after_avoid {
      break-after: avoid;
    }

    /* avoid page break inside the element */
    .page_break_inside_avoid {
      break-inside: avoid;
    }
  }

  /* -------------------------------------------------- */
  /* override pandoc css quirks */
  /* -------------------------------------------------- */

  .sourceCode {
    /* prevent unsightly overflow in wide code blocks */
    overflow: auto !important;
  }

  div.sourceCode {
    /* prevent background fill on top-most code block  container */
    background: none !important;
  }

  .sourceCode * {
    /* force consistent line spacing */
    line-height: 1.5 !important;
  }

  div.sourceCode {
    /* style code block margins same as <pre> element */
    margin-top: 20px;
    margin-bottom: 20px;
  }

  /* -------------------------------------------------- */
  /* tablenos */
  /* -------------------------------------------------- */

  /* tablenos wrapper */
  .tablenos {
    width: 100%;
    margin: 20px 0;
  }

  .tablenos > table {
    /* move margins from table to table_wrapper to allow margin collapsing */
    margin: 0;
  }

  @media only screen {
    /* tablenos wrapper */
    .tablenos {
      /* show scrollbar on tables if necessary to prevent overflow */
      overflow-x: auto !important;
    }

    .tablenos th,
    .tablenos td {
      overflow-wrap: unset !important;
      word-break: unset !important;
    }

    /* table in wrapper */
    .tablenos table,
    .tablenos table * {
      /* don't break table words */
      overflow-wrap: normal !important;
    }
  }
</style>
<!-- 
    Plugin Core

    Functions needed for and shared across all first-party plugins.
-->

<script>
  // get element that is target of hash (from link element or url)
  function getHashTarget(link) {
    const hash = link ? link.hash : window.location.hash;
    const id = hash.slice(1);
    let target = document.querySelector(`[id="${id}"]`);
    if (!target) return;

    // if figure or table, modify target to get expected element
    if (id.indexOf("fig:") === 0) target = target.querySelector("figure");
    if (id.indexOf("tbl:") === 0) target = target.querySelector("table");

    return target;
  }

  // get position/dimensions of element or viewport
  function getRectInView(element) {
    let rect = {};
    rect.left = 0;
    rect.top = 0;
    rect.right = document.documentElement.clientWidth;
    rect.bottom = document.documentElement.clientHeight;
    let style = {};

    if (element instanceof HTMLElement) {
      rect = element.getBoundingClientRect();
      style = window.getComputedStyle(element);
    }

    const margin = {};
    margin.left = parseFloat(style.marginLeftWidth) || 0;
    margin.top = parseFloat(style.marginTopWidth) || 0;
    margin.right = parseFloat(style.marginRightWidth) || 0;
    margin.bottom = parseFloat(style.marginBottomWidth) || 0;

    const border = {};
    border.left = parseFloat(style.borderLeftWidth) || 0;
    border.top = parseFloat(style.borderTopWidth) || 0;
    border.right = parseFloat(style.borderRightWidth) || 0;
    border.bottom = parseFloat(style.borderBottomWidth) || 0;

    const newRect = {};
    newRect.left = rect.left + margin.left + border.left;
    newRect.top = rect.top + margin.top + border.top;
    newRect.right = rect.right + margin.right + border.right;
    newRect.bottom = rect.bottom + margin.bottom + border.bottom;
    newRect.width = newRect.right - newRect.left;
    newRect.height = newRect.bottom - newRect.top;

    return newRect;
  }

  // get position of element relative to page
  function getRectInPage(element) {
    const rect = getRectInView(element);
    const body = getRectInView(document.body);

    const newRect = {};
    newRect.left = rect.left - body.left;
    newRect.top = rect.top - body.top;
    newRect.right = rect.right - body.left;
    newRect.bottom = rect.bottom - body.top;
    newRect.width = rect.width;
    newRect.height = rect.height;

    return newRect;
  }

  // get closest element before specified element that matches query
  function firstBefore(element, query) {
    while (element && element !== document.body && !element.matches(query))
      element = element.previousElementSibling || element.parentNode;

    return element;
  }

  // check if element is part of collapsed heading
  function isCollapsed(element) {
    while (element && element !== document.body) {
      if (element.dataset.collapsed === "true") return true;
      element = element.parentNode;
    }
    return false;
  }

  // expand any collapsed parent containers of element if necessary
  function expandElement(element) {
    if (isCollapsed(element)) {
      // accordion plugin
      const heading = firstBefore(element, "h2");
      if (heading) heading.click();
      // details/summary HTML element
      const summary = firstBefore(element, "summary");
      if (summary) summary.click();
    }
  }

  // scroll to and focus element
  function goToElement(element, offset) {
    // expand accordion section if collapsed
    expandElement(element);
    const y =
      getRectInView(element).top -
      getRectInView(document.documentElement).top -
      (offset || 0);

    // trigger any function listening for "onscroll" event
    window.dispatchEvent(new Event("scroll"));
    window.scrollTo(0, y);
    document.activeElement.blur();
    element.focus();
  }

  // get list of elements after a start element up to element matching query
  function nextUntil(element, query, exclude) {
    const elements = [];
    while (((element = element.nextElementSibling), element)) {
      if (element.matches(query)) break;
      if (!element.matches(exclude)) elements.push(element);
    }
    return elements;
  }
</script>
<!--
  Accordion Plugin

  Allows sections of content under h2 headings to be collapsible.
-->

<script type="module">
  // whether to always start expanded ('false'), always start collapsed
  // ('true'), or start collapsed when screen small ('auto')
  const startCollapsed = "auto";

  // start script
  function start() {
    // run through each <h2> heading
    const headings = document.querySelectorAll("h2");
    for (const heading of headings) {
      addArrow(heading);

      // start expanded/collapsed based on option
      if (
        startCollapsed === "true" ||
        (startCollapsed === "auto" && isSmallScreen()) ||
        heading.dataset.collapsed === "true"
      )
        collapseHeading(heading);
      else expandElement(heading);
    }

    // attach hash change listener to window
    window.addEventListener("hashchange", onHashChange);
  }

  // when hash (eg manuscript.html#introduction) changes
  function onHashChange() {
    const target = getHashTarget();
    if (target) goToElement(target);
  }

  // add arrow to heading
  function addArrow(heading) {
    // add arrow button
    const arrow = document.createElement("button");
    arrow.innerHTML = document.querySelector(".icon_angle_down").innerHTML;
    arrow.classList.add("icon_button", "accordion_arrow");
    heading.insertBefore(arrow, heading.firstChild);

    // attach click listener to heading and button
    heading.addEventListener("click", onHeadingClick);
    arrow.addEventListener("click", onArrowClick);
  }

  // determine if on mobile-like device with small screen
  function isSmallScreen() {
    return Math.min(window.innerWidth, window.innerHeight) < 480;
  }

  // when <h2> heading is clicked
  function onHeadingClick(event) {
    // only collapse if <h2> itself is target of click (eg, user did
    // not click on anchor within <h2>)
    if (event.target === this) toggleCollapse(this);
  }

  // when arrow button is clicked
  function onArrowClick() {
    toggleCollapse(this.parentNode);
  }

  // collapse section if expanded, expand if collapsed
  function toggleCollapse(heading) {
    if (heading.dataset.collapsed === "false") collapseHeading(heading);
    else expandElement(heading);
  }

  // elements to exclude from collapse, such as table of contents panel,
  // hypothesis panel, etc
  const exclude = "#toc_panel, div.annotator-frame, #lightbox_overlay";

  // collapse section
  function collapseHeading(heading) {
    heading.setAttribute("data-collapsed", "true");
    const children = getChildren(heading);
    for (const child of children) child.setAttribute("data-collapsed", "true");
  }

  // expand section
  function expandElement(heading) {
    heading.setAttribute("data-collapsed", "false");
    const children = getChildren(heading);
    for (const child of children) child.setAttribute("data-collapsed", "false");
  }

  // get list of elements between this <h2> and next <h2> or <h1>
  // ("children" of the <h2> section)
  function getChildren(heading) {
    return nextUntil(heading, "h2, h1", exclude);
  }

  // start script when document is finished loading
  window.addEventListener("load", start);
</script>

<!-- angle down icon -->

<template class="icon_angle_down">
  <!-- modified from: https://fontawesome.com/icons/angle-down -->
  <svg width="16" height="16" viewBox="0 0 448 512">
    <path
      fill="currentColor"
      d="M207.029 381.476L12.686 187.132c-9.373-9.373-9.373-24.569 0-33.941l22.667-22.667c9.357-9.357 24.522-9.375 33.901-.04L224 284.505l154.745-154.021c9.379-9.335 24.544-9.317 33.901.04l22.667 22.667c9.373 9.373 9.373 24.569 0 33.941L240.971 381.476c-9.373 9.372-24.569 9.372-33.942 0z"
    ></path>
  </svg>
</template>

<style>
  @media only screen {
    /* accordion arrow button */
    .accordion_arrow {
      margin-right: 10px;
    }

    /* arrow icon when <h2> data-collapsed attribute true */
    h2[data-collapsed="true"] > .accordion_arrow > svg {
      transform: rotate(-90deg);
    }

    /* all elements (except <h2>'s) when data-collapsed attribute true */
    *:not(h2)[data-collapsed="true"] {
      display: none;
    }

    /* accordion arrow button when hovered and <h2>'s when hovered */
    .accordion_arrow:hover,
    h2[data-collapsed="true"]:hover,
    h2[data-collapsed="false"]:hover {
      cursor: pointer;
    }
  }

  /* always hide accordion arrow button on print */
  @media only print {
    .accordion_arrow {
      display: none;
    }
  }
</style>
<!--
  Anchors Plugin

  Adds an anchor next to each of a certain type of element that provides a
  human-readable url to that specific item/position in the document (e.g.
  "manuscript.html#abstract"). It also makes it such that scrolling out of view
  of a target removes its identifier from the url.
-->

<script type="module">
  // which types of elements to add anchors next to, in "document.querySelector"
  // format
  const typesQuery =
    'h1, h2, h3, div[id^="fig:"], div[id^="tbl:"], span[id^="eq:"]';

  // start script
  function start() {
    // add anchor to each element of specified types
    const elements = document.querySelectorAll(typesQuery);
    for (const element of elements) addAnchor(element);

    // attach scroll listener to window
    window.addEventListener("scroll", onScroll);
  }

  // when window is scrolled
  function onScroll() {
    // if url has hash and user has scrolled out of view of hash
    // target, remove hash from url
    const tolerance = 100;
    const target = getHashTarget();
    if (target) {
      if (
        target.getBoundingClientRect().top > window.innerHeight + tolerance ||
        target.getBoundingClientRect().bottom < 0 - tolerance
      )
        history.pushState(null, null, " ");
    }
  }

  // add anchor to element
  function addAnchor(element) {
    let addTo; // element to add anchor button to

    // if figure or table, modify withId and addTo to get expected
    // elements
    if (element.id.indexOf("fig:") === 0) {
      addTo = element.querySelector("figcaption");
    } else if (element.id.indexOf("tbl:") === 0) {
      addTo = element.querySelector("caption");
    } else if (element.id.indexOf("eq:") === 0) {
      addTo = element.querySelector(".eqnos-number");
    }

    addTo = addTo || element;
    const id = element.id || null;

    // do not add anchor if element doesn't have assigned id.
    // id is generated by pandoc and is assumed to be unique and
    // human-readable
    if (!id) return;

    // create anchor button
    const anchor = document.createElement("a");
    anchor.innerHTML = document.querySelector(".icon_link").innerHTML;
    anchor.title = "Link to this part of the document";
    anchor.classList.add("icon_button", "anchor");
    anchor.dataset.ignore = "true";
    anchor.href = "#" + id;
    addTo.appendChild(anchor);
  }

  // start script when document is finished loading
  window.addEventListener("load", start);
</script>

<!-- link icon -->

<template class="icon_link">
  <!-- modified from: https://fontawesome.com/icons/link -->
  <svg width="16" height="16" viewBox="0 0 512 512">
    <path
      fill="currentColor"
      d="M326.612 185.391c59.747 59.809 58.927 155.698.36 214.59-.11.12-.24.25-.36.37l-67.2 67.2c-59.27 59.27-155.699 59.262-214.96 0-59.27-59.26-59.27-155.7 0-214.96l37.106-37.106c9.84-9.84 26.786-3.3 27.294 10.606.648 17.722 3.826 35.527 9.69 52.721 1.986 5.822.567 12.262-3.783 16.612l-13.087 13.087c-28.026 28.026-28.905 73.66-1.155 101.96 28.024 28.579 74.086 28.749 102.325.51l67.2-67.19c28.191-28.191 28.073-73.757 0-101.83-3.701-3.694-7.429-6.564-10.341-8.569a16.037 16.037 0 0 1-6.947-12.606c-.396-10.567 3.348-21.456 11.698-29.806l21.054-21.055c5.521-5.521 14.182-6.199 20.584-1.731a152.482 152.482 0 0 1 20.522 17.197zM467.547 44.449c-59.261-59.262-155.69-59.27-214.96 0l-67.2 67.2c-.12.12-.25.25-.36.37-58.566 58.892-59.387 154.781.36 214.59a152.454 152.454 0 0 0 20.521 17.196c6.402 4.468 15.064 3.789 20.584-1.731l21.054-21.055c8.35-8.35 12.094-19.239 11.698-29.806a16.037 16.037 0 0 0-6.947-12.606c-2.912-2.005-6.64-4.875-10.341-8.569-28.073-28.073-28.191-73.639 0-101.83l67.2-67.19c28.239-28.239 74.3-28.069 102.325.51 27.75 28.3 26.872 73.934-1.155 101.96l-13.087 13.087c-4.35 4.35-5.769 10.79-3.783 16.612 5.864 17.194 9.042 34.999 9.69 52.721.509 13.906 17.454 20.446 27.294 10.606l37.106-37.106c59.271-59.259 59.271-155.699.001-214.959z"
    ></path>
  </svg>
</template>

<style>
  @media only screen {
    /* anchor button */
    .anchor {
      opacity: 0;
      margin-left: 5px;
    }

    /* anchor buttons within <h2>'s */
    h2 .anchor {
      margin-left: 10px;
    }

    /* anchor buttons when hovered/focused and anything containing an anchor button when hovered */
    *:hover > .anchor,
    .anchor:hover,
    .anchor:focus {
      opacity: 1;
    }

    /* anchor button when hovered */
    .anchor:hover {
      cursor: pointer;
    }
  }

  /* always show anchor button on devices with no mouse/hover ability */
  @media (hover: none) {
    .anchor {
      opacity: 1;
    }
  }

  /* always hide anchor button on print */
  @media only print {
    .anchor {
      display: none;
    }
  }
</style>
<!-- 
    Attributes Plugin

    Allows arbitrary HTML attributes to be attached to (almost) any element.
    Place an HTML comment inside or next to the desired element with the content:
    $attribute="value"
-->

<script type="module">
  // start script
  function start() {
    // get list of comments in document
    const comments = findComments();

    for (const comment of comments)
      if (comment.parentElement)
        addAttributes(comment.parentElement, comment.nodeValue.trim());
  }

  // add html attributes to specified element based on string of
  // html attributes and values
  function addAttributes(element, text) {
    // regex's for finding attribute/value pairs in the format of
    // attribute="value" or attribute='value
    const regex2 = /\$([a-zA-Z\-]+)?=\"(.+?)\"/;
    const regex1 = /\$([a-zA-Z\-]+)?=\'(.+?)\'/;

    // loop through attribute/value pairs
    let match;
    while ((match = text.match(regex2) || text.match(regex1))) {
      // get attribute and value from regex capture groups
      let attribute = match[1];
      let value = match[2];

      // remove from string
      text = text.substring(match.index + match[0].length);

      if (!attribute || !value) break;

      // set attribute of parent element
      try {
        element.setAttribute(attribute, value);
      } catch (error) {
        console.log(error);
      }

      // special case for colspan
      if (attribute === "colspan") removeTableCells(element, value);
    }
  }

  // get list of comment elements in document
  function findComments() {
    const comments = [];

    // iterate over comment nodes in document
    function acceptNode(node) {
      return NodeFilter.FILTER_ACCEPT;
    }
    const iterator = document.createNodeIterator(
      document.body,
      NodeFilter.SHOW_COMMENT,
      acceptNode
    );
    let node;
    while ((node = iterator.nextNode())) comments.push(node);

    return comments;
  }

  // remove certain number of cells after specified cell
  function removeTableCells(cell, number) {
    number = parseInt(number);
    if (!number) return;

    // remove elements
    for (; number > 1; number--) {
      if (cell.nextElementSibling) cell.nextElementSibling.remove();
    }
  }

  // start script on DOMContentLoaded instead of load to ensure this plugins
  // runs before other plugins
  window.addEventListener("DOMContentLoaded", start);
</script>
<!--
  Jump to First Plugin

  Adds a button next to each reference entry, figure, and table that jumps the
  page to the first occurrence of a link to that item in the manuscript.
-->

<script type="module">
  // whether to add buttons next to reference entries
  const references = "true";
  // whether to add buttons next to figures
  const figures = "true";
  // whether to add buttons next to tables
  const tables = "true";

  // start script
  function start() {
    if (references !== "false")
      makeButtons(`div[id^="ref-"]`, ".csl-left-margin", "reference");
    if (figures !== "false")
      makeButtons(`div[id^="fig:"]`, "figcaption", "figure");
    if (tables !== "false") makeButtons(`div[id^="tbl:"]`, "caption", "table");
  }

  // when jump button clicked
  function onButtonClick() {
    const first = getFirstOccurrence(this.dataset.id);
    if (!first) return;

    // update url hash so navigating "back" in history will return user to button
    window.location.hash = this.dataset.id;
    // scroll to link
    const timeout = function () {
      goToElement(first, window.innerHeight * 0.5);
    };
    window.setTimeout(timeout, 0);
  }

  // get first occurrence of link to item in document
  function getFirstOccurrence(id) {
    let query = "a";
    query += '[href="#' + id + '"]';
    // exclude buttons, anchor links, toc links, etc
    query += ":not(.button):not(.icon_button):not(.anchor):not(.toc_link)";
    return document.querySelector(query);
  }

  // add button next to each reference entry, figure, or table
  function makeButtons(query, containerQuery, subject) {
    const elements = document.querySelectorAll(query);
    for (const element of elements) {
      const id = element.id;
      const buttonContainer = element.querySelector(containerQuery);
      const first = getFirstOccurrence(id);

      // if can't find link to reference or place to put button, ignore
      if (!first || !buttonContainer) continue;

      // make jump button
      let button = document.createElement("button");
      button.classList.add("icon_button", "jump_arrow");
      button.title = `Jump to the first occurrence of this ${subject} in the document`;
      const icon = document.querySelector(".icon_angle_double_up");
      button.innerHTML = icon.innerHTML;
      button.dataset.id = id;
      button.dataset.ignore = "true";
      button.addEventListener("click", onButtonClick);
      buttonContainer.prepend(button);
    }
  }

  // start script when document is finished loading
  window.addEventListener("load", start);
</script>

<!-- angle double up icon -->

<template class="icon_angle_double_up">
  <!-- modified from: https://fontawesome.com/icons/angle-double-up -->
  <svg width="16" height="16" viewBox="0 0 320 512">
    <path
      fill="currentColor"
      d="M177 255.7l136 136c9.4 9.4 9.4 24.6 0 33.9l-22.6 22.6c-9.4 9.4-24.6 9.4-33.9 0L160 351.9l-96.4 96.4c-9.4 9.4-24.6 9.4-33.9 0L7 425.7c-9.4-9.4-9.4-24.6 0-33.9l136-136c9.4-9.5 24.6-9.5 34-.1zm-34-192L7 199.7c-9.4 9.4-9.4 24.6 0 33.9l22.6 22.6c9.4 9.4 24.6 9.4 33.9 0l96.4-96.4 96.4 96.4c9.4 9.4 24.6 9.4 33.9 0l22.6-22.6c9.4-9.4 9.4-24.6 0-33.9l-136-136c-9.2-9.4-24.4-9.4-33.8 0z"
    ></path>
  </svg>
</template>

<style>
  @media only screen {
    /* jump button */
    .jump_arrow {
      position: relative;
      top: 0.125em;
      margin-right: 5px;
    }
  }

  /* always hide jump button on print */
  @media only print {
    .jump_arrow {
      display: none;
    }
  }
</style>
<!-- 
    Lightbox Plugin

    Makes it such that when a user clicks on an image, the image fills the
    screen and the user can pan/drag/zoom the image and navigate between other
    images in the document.
-->

<script type="module">
  // list of possible zoom/scale factors
  const zooms =
    "0.1, 0.25, 0.333333, 0.5, 0.666666, 0.75, 1, 1.25, 1.5, 1.75, 2, 2.5, 3, 3.5, 4, 5, 6, 7, 8";
  // whether to fit image to view ('fit'), display at 100% and shrink if
  // necessary ('shrink'), or always display at 100% ('100')
  const defaultZoom = "fit";
  // whether to zoom in/out toward center of view ('true') or mouse ('false')
  const centerZoom = "false";

  // start script
  function start() {
    // run through each <img> element
    const imgs = document.querySelectorAll("figure > img");
    let count = 1;
    for (const img of imgs) {
      img.classList.add("lightbox_document_img");
      img.dataset.number = count;
      img.dataset.total = imgs.length;
      img.addEventListener("click", openLightbox);
      count++;
    }

    // attach mouse and key listeners to window
    window.addEventListener("mousemove", onWindowMouseMove);
    window.addEventListener("keyup", onKeyUp);
  }

  // when mouse is moved anywhere in window
  function onWindowMouseMove(event) {
    window.mouseX = event.clientX;
    window.mouseY = event.clientY;
  }

  // when key pressed
  function onKeyUp(event) {
    if (!event || !event.key) return;

    switch (event.key) {
      // trigger click of prev button
      case "ArrowLeft":
        const prevButton = document.getElementById("lightbox_prev_button");
        if (prevButton) prevButton.click();
        break;
      // trigger click of next button
      case "ArrowRight":
        const nextButton = document.getElementById("lightbox_next_button");
        if (nextButton) nextButton.click();
        break;
      // close on esc
      case "Escape":
        closeLightbox();
        break;
    }
  }

  // open lightbox
  function openLightbox() {
    const lightbox = makeLightbox(this);
    if (!lightbox) return;

    blurBody(lightbox);
    document.body.appendChild(lightbox);
  }

  // make lightbox
  function makeLightbox(img) {
    // delete lightbox if it exists, start fresh
    closeLightbox();

    // create screen overlay containing lightbox
    const overlay = document.createElement("div");
    overlay.id = "lightbox_overlay";

    // create image info boxes
    const numberInfo = document.createElement("div");
    const zoomInfo = document.createElement("div");
    numberInfo.id = "lightbox_number_info";
    zoomInfo.id = "lightbox_zoom_info";

    // create container for image
    const imageContainer = document.createElement("div");
    imageContainer.id = "lightbox_image_container";
    const lightboxImg = makeLightboxImg(
      img,
      imageContainer,
      numberInfo,
      zoomInfo
    );
    imageContainer.appendChild(lightboxImg);

    // create bottom container for caption and navigation buttons
    const bottomContainer = document.createElement("div");
    bottomContainer.id = "lightbox_bottom_container";
    const caption = makeCaption(img);
    const prevButton = makePrevButton(img);
    const nextButton = makeNextButton(img);
    bottomContainer.appendChild(prevButton);
    bottomContainer.appendChild(caption);
    bottomContainer.appendChild(nextButton);

    // attach top middle and bottom to overlay
    overlay.appendChild(numberInfo);
    overlay.appendChild(zoomInfo);
    overlay.appendChild(imageContainer);
    overlay.appendChild(bottomContainer);

    return overlay;
  }

  // make <img> object that is intuitively draggable and zoomable
  function makeLightboxImg(sourceImg, container, numberInfoBox, zoomInfoBox) {
    // create copy of source <img>
    const img = sourceImg.cloneNode(true);
    img.classList.remove("lightbox_document_img");
    img.removeAttribute("id");
    img.removeAttribute("width");
    img.removeAttribute("height");
    img.style.position = "unset";
    img.style.margin = "0";
    img.style.padding = "0";
    img.style.width = "";
    img.style.height = "";
    img.style.minWidth = "";
    img.style.minHeight = "";
    img.style.maxWidth = "";
    img.style.maxHeight = "";
    img.id = "lightbox_img";

    // build sorted list of zoomSteps
    const zoomSteps = zooms.split(/[^0-9.]/).map((step) => parseFloat(step));
    zoomSteps.sort((a, b) => a - b);

    // <img> object property variables
    let zoom = 1;
    let translateX = 0;
    let translateY = 0;
    let clickMouseX = undefined;
    let clickMouseY = undefined;
    let clickTranslateX = undefined;
    let clickTranslateY = undefined;

    updateNumberInfo();

    // update image numbers displayed in info box
    function updateNumberInfo() {
      numberInfoBox.innerHTML =
        sourceImg.dataset.number + " of " + sourceImg.dataset.total;
    }

    // update zoom displayed in info box
    function updateZoomInfo() {
      let zoomInfo = zoom * 100;
      if (!Number.isInteger(zoomInfo)) zoomInfo = zoomInfo.toFixed(2);
      zoomInfoBox.innerHTML = zoomInfo + "%";
    }

    // move to closest zoom step above current zoom
    const zoomIn = function () {
      for (const zoomStep of zoomSteps) {
        if (zoomStep > zoom) {
          zoom = zoomStep;
          break;
        }
      }
      updateTransform();
    };

    // move to closest zoom step above current zoom
    const zoomOut = function () {
      zoomSteps.reverse();
      for (const zoomStep of zoomSteps) {
        if (zoomStep < zoom) {
          zoom = zoomStep;
          break;
        }
      }
      zoomSteps.reverse();

      updateTransform();
    };

    // update display of <img> based on scale/translate properties
    const updateTransform = function () {
      // set transform
      img.style.transform =
        "translate(" +
        (translateX || 0) +
        "px," +
        (translateY || 0) +
        "px) scale(" +
        (zoom || 1) +
        ")";

      // get new width/height after scale
      const rect = img.getBoundingClientRect();
      // limit translate
      translateX = Math.max(translateX, -rect.width / 2);
      translateX = Math.min(translateX, rect.width / 2);
      translateY = Math.max(translateY, -rect.height / 2);
      translateY = Math.min(translateY, rect.height / 2);

      // set transform
      img.style.transform =
        "translate(" +
        (translateX || 0) +
        "px," +
        (translateY || 0) +
        "px) scale(" +
        (zoom || 1) +
        ")";

      updateZoomInfo();
    };

    // fit <img> to container
    const fit = function () {
      // no x/y offset, 100% zoom by default
      translateX = 0;
      translateY = 0;
      zoom = 1;

      // widths of <img> and container
      const imgWidth = img.naturalWidth;
      const imgHeight = img.naturalHeight;
      const containerWidth = parseFloat(
        window.getComputedStyle(container).width
      );
      const containerHeight = parseFloat(
        window.getComputedStyle(container).height
      );

      // how much zooming is needed to fit <img> to container
      const xRatio = imgWidth / containerWidth;
      const yRatio = imgHeight / containerHeight;
      const maxRatio = Math.max(xRatio, yRatio);
      const newZoom = 1 / maxRatio;

      // fit <img> to container according to option
      if (defaultZoom === "shrink") {
        if (maxRatio > 1) zoom = newZoom;
      } else if (defaultZoom === "fit") zoom = newZoom;

      updateTransform();
    };

    // when mouse wheel is rolled anywhere in container
    const onContainerWheel = function (event) {
      if (!event) return;

      // let ctrl + mouse wheel to zoom behave as normal
      if (event.ctrlKey) return;

      // prevent normal scroll behavior
      event.preventDefault();
      event.stopPropagation();

      // point around which to scale img
      const viewRect = container.getBoundingClientRect();
      const viewX = (viewRect.left + viewRect.right) / 2;
      const viewY = (viewRect.top + viewRect.bottom) / 2;
      const originX = centerZoom === "true" ? viewX : mouseX;
      const originY = centerZoom === "true" ? viewY : mouseY;

      // get point on image under origin
      const oldRect = img.getBoundingClientRect();
      const oldPercentX = (originX - oldRect.left) / oldRect.width;
      const oldPercentY = (originY - oldRect.top) / oldRect.height;

      // increment/decrement zoom
      if (event.deltaY < 0) zoomIn();
      if (event.deltaY > 0) zoomOut();

      // get offset between previous image point and origin
      const newRect = img.getBoundingClientRect();
      const offsetX = originX - (newRect.left + newRect.width * oldPercentX);
      const offsetY = originY - (newRect.top + newRect.height * oldPercentY);

      // translate image to keep image point under origin
      translateX += offsetX;
      translateY += offsetY;

      // perform translate
      updateTransform();
    };

    // when container is clicked
    function onContainerClick(event) {
      // if container itself is target of click, and not other
      // element above it
      if (event.target === this) closeLightbox();
    }

    // when mouse button is pressed on image
    const onImageMouseDown = function (event) {
      // store original mouse position relative to image
      clickMouseX = window.mouseX;
      clickMouseY = window.mouseY;
      clickTranslateX = translateX;
      clickTranslateY = translateY;
      event.stopPropagation();
      event.preventDefault();
    };

    // when mouse button is released anywhere in window
    const onWindowMouseUp = function (event) {
      // reset original mouse position
      clickMouseX = undefined;
      clickMouseY = undefined;
      clickTranslateX = undefined;
      clickTranslateY = undefined;

      // remove global listener if lightbox removed from document
      if (!document.body.contains(container))
        window.removeEventListener("mouseup", onWindowMouseUp);
    };

    // when mouse is moved anywhere in window
    const onWindowMouseMove = function (event) {
      if (
        clickMouseX === undefined ||
        clickMouseY === undefined ||
        clickTranslateX === undefined ||
        clickTranslateY === undefined
      )
        return;

      // offset image based on original and current mouse position
      translateX = clickTranslateX + window.mouseX - clickMouseX;
      translateY = clickTranslateY + window.mouseY - clickMouseY;
      updateTransform();
      event.preventDefault();

      // remove global listener if lightbox removed from document
      if (!document.body.contains(container))
        window.removeEventListener("mousemove", onWindowMouseMove);
    };

    // when window is resized
    const onWindowResize = function (event) {
      fit();

      // remove global listener if lightbox removed from document
      if (!document.body.contains(container))
        window.removeEventListener("resize", onWindowResize);
    };

    // attach the necessary event listeners
    img.addEventListener("dblclick", fit);
    img.addEventListener("mousedown", onImageMouseDown);
    container.addEventListener("wheel", onContainerWheel);
    container.addEventListener("mousedown", onContainerClick);
    container.addEventListener("touchstart", onContainerClick);
    window.addEventListener("mouseup", onWindowMouseUp);
    window.addEventListener("mousemove", onWindowMouseMove);
    window.addEventListener("resize", onWindowResize);

    // run fit() after lightbox atttached to document and <img> Loaded
    // so needed container and img dimensions available
    img.addEventListener("load", fit);

    return img;
  }

  // make caption
  function makeCaption(img) {
    const caption = document.createElement("div");
    caption.id = "lightbox_caption";
    const captionSource = img.nextElementSibling;
    if (captionSource.tagName.toLowerCase() === "figcaption") {
      const captionCopy = makeCopy(captionSource);
      caption.innerHTML = captionCopy.innerHTML;
    }

    caption.addEventListener("touchstart", function (event) {
      event.stopPropagation();
    });

    return caption;
  }

  // make carbon copy of html dom element
  function makeCopy(source) {
    const sourceCopy = source.cloneNode(true);

    // delete elements marked with ignore (eg anchor and jump buttons)
    const deleteFromCopy = sourceCopy.querySelectorAll('[data-ignore="true"]');
    for (const element of deleteFromCopy) element.remove();

    // delete certain element attributes
    const attributes = [
      "id",
      "data-collapsed",
      "data-selected",
      "data-highlighted",
      "data-glow",
    ];
    for (const attribute of attributes) {
      sourceCopy.removeAttribute(attribute);
      const elements = sourceCopy.querySelectorAll("[" + attribute + "]");
      for (const element of elements) element.removeAttribute(attribute);
    }

    return sourceCopy;
  }

  // make button to jump to previous image in document
  function makePrevButton(img) {
    const prevButton = document.createElement("button");
    prevButton.id = "lightbox_prev_button";
    prevButton.title = "Jump to the previous image in the document [←]";
    prevButton.classList.add("icon_button", "lightbox_button");
    prevButton.innerHTML = document.querySelector(".icon_caret_left").innerHTML;

    // attach click listeners to button
    prevButton.addEventListener("click", function () {
      getPrevImg(img).click();
    });

    return prevButton;
  }

  // make button to jump to next image in document
  function makeNextButton(img) {
    const nextButton = document.createElement("button");
    nextButton.id = "lightbox_next_button";
    nextButton.title = "Jump to the next image in the document [→]";
    nextButton.classList.add("icon_button", "lightbox_button");
    nextButton.innerHTML = document.querySelector(
      ".icon_caret_right"
    ).innerHTML;

    // attach click listeners to button
    nextButton.addEventListener("click", function () {
      getNextImg(img).click();
    });

    return nextButton;
  }

  // get previous image in document
  function getPrevImg(img) {
    const imgs = document.querySelectorAll(".lightbox_document_img");

    // find index of provided img
    let index;
    for (index = 0; index < imgs.length; index++) {
      if (imgs[index] === img) break;
    }

    // wrap index to other side if < 1
    if (index - 1 >= 0) index--;
    else index = imgs.length - 1;
    return imgs[index];
  }

  // get next image in document
  function getNextImg(img) {
    const imgs = document.querySelectorAll(".lightbox_document_img");

    // find index of provided img
    let index;
    for (index = 0; index < imgs.length; index++) {
      if (imgs[index] === img) break;
    }

    // wrap index to other side if > total
    if (index + 1 <= imgs.length - 1) index++;
    else index = 0;
    return imgs[index];
  }

  // close lightbox
  function closeLightbox() {
    focusBody();

    const lightbox = document.getElementById("lightbox_overlay");
    if (lightbox) lightbox.remove();
  }

  // make all elements behind lightbox non-focusable
  function blurBody(overlay) {
    const all = document.querySelectorAll("*");
    for (const element of all) element.tabIndex = -1;
    document.body.classList.add("body_no_scroll");
  }

  // make all elements focusable again
  function focusBody() {
    const all = document.querySelectorAll("*");
    for (const element of all) element.removeAttribute("tabIndex");
    document.body.classList.remove("body_no_scroll");
  }

  // start script when document is finished loading
  window.addEventListener("load", start);
</script>

<!-- caret left icon -->

<template class="icon_caret_left">
  <!-- modified from: https://fontawesome.com/icons/caret-left -->
  <svg width="16" height="16" viewBox="0 0 192 512">
    <path
      fill="currentColor"
      d="M192 127.338v257.324c0 17.818-21.543 26.741-34.142 14.142L29.196 270.142c-7.81-7.81-7.81-20.474 0-28.284l128.662-128.662c12.599-12.6 34.142-3.676 34.142 14.142z"
    ></path>
  </svg>
</template>

<!-- caret right icon -->

<template class="icon_caret_right">
  <!-- modified from: https://fontawesome.com/icons/caret-right -->
  <svg width="16" height="16" viewBox="0 0 192 512">
    <path
      fill="currentColor"
      d="M0 384.662V127.338c0-17.818 21.543-26.741 34.142-14.142l128.662 128.662c7.81 7.81 7.81 20.474 0 28.284L34.142 398.804C21.543 411.404 0 402.48 0 384.662z"
    ></path>
  </svg>
</template>

<style>
  @media only screen {
    /* regular <img> in document when hovered */
    img.lightbox_document_img:hover {
      cursor: pointer;
    }

    .body_no_scroll {
      overflow: hidden !important;
    }

    /* screen overlay */
    #lightbox_overlay {
      display: flex;
      flex-direction: column;
      position: fixed;
      left: 0;
      top: 0;
      right: 0;
      bottom: 0;
      background: rgba(0, 0, 0, 0.75);
      z-index: 3;
    }

    /* middle area containing lightbox image */
    #lightbox_image_container {
      flex-grow: 1;
      display: flex;
      justify-content: center;
      align-items: center;
      overflow: hidden;
      position: relative;
      padding: 20px;
    }

    /* bottom area containing caption */
    #lightbox_bottom_container {
      display: flex;
      justify-content: center;
      align-items: center;
      height: 100px;
      min-height: 100px;
      max-height: 100px;
      background: rgba(0, 0, 0, 0.5);
    }

    /* image number info text box */
    #lightbox_number_info {
      position: absolute;
      color: #ffffff;
      font-weight: 600;
      left: 2px;
      top: 0;
      z-index: 4;
    }

    /* zoom info text box */
    #lightbox_zoom_info {
      position: absolute;
      color: #ffffff;
      font-weight: 600;
      right: 2px;
      top: 0;
      z-index: 4;
    }

    /* copy of image caption */
    #lightbox_caption {
      box-sizing: border-box;
      display: inline-block;
      width: 100%;
      max-height: 100%;
      padding: 10px 0;
      text-align: center;
      overflow-y: auto;
      color: #ffffff;
    }

    /* navigation previous/next button */
    .lightbox_button {
      width: 100px;
      height: 100%;
      min-width: 100px;
      min-height: 100%;
      color: #ffffff;
    }

    /* navigation previous/next button when hovered */
    .lightbox_button:hover {
      background: none !important;
    }

    /* navigation button icon */
    .lightbox_button > svg {
      height: 25px;
    }

    /* figure auto-number */
    #lightbox_caption > span:first-of-type {
      font-weight: bold;
      margin-right: 5px;
    }

    /* lightbox image when hovered */
    #lightbox_img:hover {
      cursor: grab;
    }

    /* lightbox image when grabbed */
    #lightbox_img:active {
      cursor: grabbing;
    }
  }

  /* when on screen < 480px wide */
  @media only screen and (max-width: 480px) {
    /* make navigation buttons skinnier on small screens to make more room for caption text */
    .lightbox_button {
      width: 50px;
      min-width: 50px;
    }
  }

  /* always hide lightbox on print */
  @media only print {
    #lightbox_overlay {
      display: none;
    }
  }
</style>
<!-- 
  Link Highlight Plugin

  Makes it such that when a user hovers or focuses a link, other links that have
  the same target will be highlighted. It also makes it such that when clicking
  a link, the target of the link (eg reference, figure, table) is briefly
  highlighted.
-->

<script type="module">
  // whether to also highlight links that go to external urls
  const externalLinks = "false";
  // whether user must click off to unhighlight instead of just
  // un-hovering
  const clickUnhighlight = "false";
  // whether to also highlight links that are unique
  const highlightUnique = "true";

  // start script
  function start() {
    const links = getLinks();
    for (const link of links) {
      // attach mouse and focus listeners to link
      link.addEventListener("mouseenter", onLinkFocus);
      link.addEventListener("focus", onLinkFocus);
      link.addEventListener("mouseleave", onLinkUnhover);
    }

    // attach click and hash change listeners to window
    window.addEventListener("click", onClick);
    window.addEventListener("touchstart", onClick);
    window.addEventListener("hashchange", onHashChange);

    // run hash change on window load in case user has navigated
    // directly to hash
    onHashChange();
  }

  // when link is focused (tabbed to) or hovered
  function onLinkFocus() {
    highlight(this);
  }

  // when link is unhovered
  function onLinkUnhover() {
    if (clickUnhighlight !== "true") unhighlightAll();
  }

  // when the mouse is clicked anywhere in window
  function onClick(event) {
    unhighlightAll();
  }

  // when hash (eg manuscript.html#introduction) changes
  function onHashChange() {
    const target = getHashTarget();
    if (target) glowElement(target);
  }

  // start glow sequence on an element
  function glowElement(element) {
    const startGlow = function () {
      onGlowEnd();
      element.dataset.glow = "true";
      element.addEventListener("animationend", onGlowEnd);
    };
    const onGlowEnd = function () {
      element.removeAttribute("data-glow");
      element.removeEventListener("animationend", onGlowEnd);
    };
    startGlow();
  }

  // highlight link and all others with same target
  function highlight(link) {
    // force unhighlight all to start fresh
    unhighlightAll();

    // get links with same target
    if (!link) return;
    const sameLinks = getSameLinks(link);

    // if link unique and option is off, exit and don't highlight
    if (sameLinks.length <= 1 && highlightUnique !== "true") return;

    // highlight all same links, and "select" (special highlight) this
    // one
    for (const sameLink of sameLinks) {
      if (sameLink === link) sameLink.setAttribute("data-selected", "true");
      else sameLink.setAttribute("data-highlighted", "true");
    }
  }

  // unhighlight all links
  function unhighlightAll() {
    const links = getLinks();
    for (const link of links) {
      link.setAttribute("data-selected", "false");
      link.setAttribute("data-highlighted", "false");
    }
  }

  // get links with same target
  function getSameLinks(link) {
    const results = [];
    const links = getLinks();
    for (const otherLink of links) {
      if (otherLink.getAttribute("href") === link.getAttribute("href"))
        results.push(otherLink);
    }
    return results;
  }

  // get all links of types we wish to handle
  function getLinks() {
    let query = "a";
    if (externalLinks !== "true") query += '[href^="#"]';
    // exclude buttons, anchor links, toc links, etc
    query += ":not(.button):not(.icon_button):not(.anchor):not(.toc_link)";
    return document.querySelectorAll(query);
  }

  // start script when document is finished loading
  window.addEventListener("load", start);
</script>

<style>
  @media only screen {
    /* anything with data-highlighted attribute true */
    [data-highlighted="true"] {
      background: #ffeb3b;
    }

    /* anything with data-selected attribute true */
    [data-selected="true"] {
      background: #ff8a65 !important;
    }

    /* animation definition for glow */
    @keyframes highlight_glow {
      0% {
        background: none;
      }
      10% {
        background: #bbdefb;
      }
      100% {
        background: none;
      }
    }

    /* anything with data-glow attribute true */
    [data-glow="true"] {
      animation: highlight_glow 2s;
    }
  }
</style>
<!--
  Table of Contents Plugin

  Provides a "table of contents" (toc) panel on the side of the document that
  allows the user to conveniently navigate between sections of the document.
-->

<script type="module">
  // which types of elements to add links for, in "document.querySelector" format
  const typesQuery = "h1, h2, h3";
  // whether toc starts open. use 'true' or 'false', or 'auto' to
  // use 'true' behavior when screen wide enough and 'false' when not
  const startOpen = "false";
  // whether toc closes when clicking on toc link. use 'true' or
  // 'false', or 'auto' to use 'false' behavior when screen wide
  // enough and 'true' when not
  const clickClose = "auto";
  // if list item is more than this many characters, text will be
  // truncated
  const charLimit = "50";
  // whether or not to show bullets next to each toc item
  const bullets = "false";

  // start script
  function start() {
    // make toc panel and populate with entries (links to document
    // sections)
    const panel = makePanel();
    if (!panel) return;
    makeEntries(panel);
    // attach panel to document after making entries, so 'toc' heading
    // in panel isn't included in toc
    document.body.insertBefore(panel, document.body.firstChild);

    // initial panel state
    if (startOpen === "true" || (startOpen === "auto" && !isSmallScreen()))
      openPanel();
    else closePanel();

    // attach click, scroll, and hash change listeners to window
    window.addEventListener("click", onClick);
    window.addEventListener("scroll", onScroll);
    window.addEventListener("hashchange", onScroll);
    window.addEventListener("keyup", onKeyUp);
    onScroll();

    // add class to push document body down out of way of toc button
    document.body.classList.add("toc_body_nudge");
  }

  // determine if screen wide enough to fit toc panel
  function isSmallScreen() {
    // in default theme:
    // 816px = 8.5in = width of "page" (<body>) element
    // 260px = min width of toc panel (*2 for both sides of <body>)
    return window.innerWidth < 816 + 260 * 2;
  }

  // when mouse is clicked anywhere in window
  function onClick() {
    if (isSmallScreen()) closePanel();
  }

  // when window is scrolled or hash changed
  function onScroll() {
    highlightViewed();
  }

  // when key pressed
  function onKeyUp(event) {
    if (!event || !event.key) return;

    // close on esc
    if (event.key === "Escape") closePanel();
  }

  // find entry of currently viewed document section in toc and highlight
  function highlightViewed() {
    const firstId = getFirstInView(typesQuery);

    // get toc entries (links), unhighlight all, then highlight viewed
    const list = document.getElementById("toc_list");
    if (!firstId || !list) return;
    const links = list.querySelectorAll("a");
    for (const link of links) link.dataset.viewing = "false";
    const link = list.querySelector('a[href="#' + firstId + '"]');
    if (!link) return;
    link.dataset.viewing = "true";
  }

  // get first or previous toc listed element in top half of view
  function getFirstInView(query) {
    // get all elements matching query and with id
    const elements = document.querySelectorAll(query);
    const elementsWithIds = [];
    for (const element of elements) {
      if (element.id) elementsWithIds.push(element);
    }

    // get first or previous element in top half of view
    for (let i = 0; i < elementsWithIds.length; i++) {
      const element = elementsWithIds[i];
      const prevElement = elementsWithIds[Math.max(0, i - 1)];
      if (element.getBoundingClientRect().top >= 0) {
        if (element.getBoundingClientRect().top < window.innerHeight / 2)
          return element.id;
        else return prevElement.id;
      }
    }
  }

  // make panel
  function makePanel() {
    // create panel
    const panel = document.createElement("div");
    panel.id = "toc_panel";
    if (bullets === "true") panel.dataset.bullets = "true";

    // create header
    const header = document.createElement("div");
    header.id = "toc_header";

    // create toc button
    const button = document.createElement("button");
    button.id = "toc_button";
    button.innerHTML = document.querySelector(".icon_th_list").innerHTML;
    button.title = "Table of Contents";
    button.classList.add("icon_button");

    // create header text
    const text = document.createElement("h4");
    text.innerHTML = "Table of Contents";

    // create container for toc list
    const list = document.createElement("div");
    list.id = "toc_list";

    // attach click listeners
    panel.addEventListener("click", onPanelClick);
    header.addEventListener("click", onHeaderClick);
    button.addEventListener("click", onButtonClick);

    // attach elements
    header.appendChild(button);
    header.appendChild(text);
    panel.appendChild(header);
    panel.appendChild(list);

    return panel;
  }

  // create toc entries (links) to each element of the specified types
  function makeEntries(panel) {
    const elements = document.querySelectorAll(typesQuery);
    for (const element of elements) {
      // do not add link if element doesn't have assigned id
      if (!element.id) continue;

      // create link/list item
      const link = document.createElement("a");
      link.classList.add("toc_link");
      switch (element.tagName.toLowerCase()) {
        case "h1":
          link.dataset.level = "1";
          break;
        case "h2":
          link.dataset.level = "2";
          break;
        case "h3":
          link.dataset.level = "3";
          break;
        case "h4":
          link.dataset.level = "4";
          break;
      }
      link.title = element.innerText;
      let text = element.innerText;
      if (text.length > charLimit) text = text.slice(0, charLimit) + "...";
      link.innerHTML = text;
      link.href = "#" + element.id;
      link.addEventListener("click", onLinkClick);

      // attach link
      panel.querySelector("#toc_list").appendChild(link);
    }
  }

  // when panel is clicked
  function onPanelClick(event) {
    // stop click from propagating to window/document and closing panel
    event.stopPropagation();
  }

  // when header itself is clicked
  function onHeaderClick(event) {
    togglePanel();
  }

  // when button is clicked
  function onButtonClick(event) {
    togglePanel();
    // stop header underneath button from also being clicked
    event.stopPropagation();
  }

  // when link is clicked
  function onLinkClick(event) {
    if (clickClose === "true" || (clickClose === "auto" && isSmallScreen()))
      closePanel();
    else openPanel();
  }

  // open panel if closed, close if opened
  function togglePanel() {
    const panel = document.getElementById("toc_panel");
    if (!panel) return;

    if (panel.dataset.open === "true") closePanel();
    else openPanel();
  }

  // open panel
  function openPanel() {
    const panel = document.getElementById("toc_panel");
    if (panel) panel.dataset.open = "true";
  }

  // close panel
  function closePanel() {
    const panel = document.getElementById("toc_panel");
    if (panel) panel.dataset.open = "false";
  }

  // start script when document is finished loading
  window.addEventListener("load", start);
</script>

<!-- th list icon -->

<template class="icon_th_list">
  <!-- modified from: https://fontawesome.com/icons/th-list -->
  <svg width="16" height="16" viewBox="0 0 512 512" tabindex="-1">
    <path
      fill="currentColor"
      d="M96 96c0 26.51-21.49 48-48 48S0 122.51 0 96s21.49-48 48-48 48 21.49 48 48zM48 208c-26.51 0-48 21.49-48 48s21.49 48 48 48 48-21.49 48-48-21.49-48-48-48zm0 160c-26.51 0-48 21.49-48 48s21.49 48 48 48 48-21.49 48-48-21.49-48-48-48zm96-236h352c8.837 0 16-7.163 16-16V76c0-8.837-7.163-16-16-16H144c-8.837 0-16 7.163-16 16v40c0 8.837 7.163 16 16 16zm0 160h352c8.837 0 16-7.163 16-16v-40c0-8.837-7.163-16-16-16H144c-8.837 0-16 7.163-16 16v40c0 8.837 7.163 16 16 16zm0 160h352c8.837 0 16-7.163 16-16v-40c0-8.837-7.163-16-16-16H144c-8.837 0-16 7.163-16 16v40c0 8.837 7.163 16 16 16z"
      tabindex="-1"
    ></path>
  </svg>
</template>

<style>
  @media only screen {
    /* toc panel */
    #toc_panel {
      box-sizing: border-box;
      position: fixed;
      top: 0;
      left: 0;
      background: #ffffff;
      box-shadow: 0 0 20px rgba(0, 0, 0, 0.05);
      z-index: 2;
    }

    /* toc panel when closed */
    #toc_panel[data-open="false"] {
      min-width: 60px;
      width: 60px;
      height: 60px;
      border-right: solid 1px #bdbdbd;
      border-bottom: solid 1px #bdbdbd;
    }

    /* toc panel when open */
    #toc_panel[data-open="true"] {
      min-width: 260px;
      max-width: 480px;
      /* keep panel edge consistent distance away from "page" edge */
      width: calc(((100vw - 8.5in) / 2) - 30px - 40px);
      bottom: 0;
      border-right: solid 1px #bdbdbd;
    }

    /* toc panel header */
    #toc_header {
      box-sizing: border-box;
      display: flex;
      flex-direction: row;
      align-items: center;
      height: 60px;
      margin: 0;
      padding: 20px;
    }

    /* toc panel header when hovered */
    #toc_header:hover {
      cursor: pointer;
    }

    /* toc panel header when panel open */
    #toc_panel[data-open="true"] > #toc_header {
      border-bottom: solid 1px #bdbdbd;
    }

    /* toc open/close header button */
    #toc_button {
      margin-right: 20px;
    }

    /* hide toc list and header text when closed */
    #toc_panel[data-open="false"] > #toc_header > *:not(#toc_button),
    #toc_panel[data-open="false"] > #toc_list {
      display: none;
    }

    /* toc list of entries */
    #toc_list {
      box-sizing: border-box;
      width: 100%;
      padding: 20px;
      position: absolute;
      top: calc(60px + 1px);
      bottom: 0;
      overflow: auto;
    }

    /* toc entry, link to section in document */
    .toc_link {
      display: block;
      padding: 5px;
      position: relative;
      font-weight: 600;
      text-decoration: none;
    }

    /* toc entry when hovered or when "viewed" */
    .toc_link:hover,
    .toc_link[data-viewing="true"] {
      background: #f5f5f5;
    }

    /* toc entry, level 1 indentation */
    .toc_link[data-level="1"] {
      margin-left: 0;
    }

    /* toc entry, level 2 indentation */
    .toc_link[data-level="2"] {
      margin-left: 20px;
    }

    /* toc entry, level 3 indentation */
    .toc_link[data-level="3"] {
      margin-left: 40px;
    }

    /* toc entry, level 4 indentation */
    .toc_link[data-level="4"] {
      margin-left: 60px;
    }

    /* toc entry bullets */
    #toc_panel[data-bullets="true"] .toc_link[data-level]:before {
      position: absolute;
      left: -15px;
      top: -1px;
      font-size: 1.5em;
    }

    /* toc entry, level 2 bullet */
    #toc_panel[data-bullets="true"] .toc_link[data-level="2"]:before {
      content: "\2022";
    }

    /* toc entry, level 3 bullet */
    #toc_panel[data-bullets="true"] .toc_link[data-level="3"]:before {
      content: "\25AB";
    }

    /* toc entry, level 4 bullet */
    #toc_panel[data-bullets="true"] .toc_link[data-level="4"]:before {
      content: "-";
    }
  }

  /* when on screen < 8.5in wide */
  @media only screen and (max-width: 8.5in) {
    /* push <body> ("page") element down to make room for toc icon */
    .toc_body_nudge {
      padding-top: 60px;
    }

    /* toc icon when panel closed and not hovered */
    #toc_panel[data-open="false"]:not(:hover) {
      background: rgba(255, 255, 255, 0.75);
    }
  }

  /* always hide toc panel on print */
  @media only print {
    #toc_panel {
      display: none;
    }
  }
</style>
<!-- 
  Tooltips Plugin

  Makes it such that when the user hovers or focuses a link to a citation or
  figure, a tooltip appears with a preview of the reference content, along with
  arrows to navigate between instances of the same reference in the document.
-->

<script type="module">
  // whether user must click off to close tooltip instead of just un-hovering
  const clickClose = "false";
  // delay (in ms) between opening and closing tooltip
  const delay = "100";

  // start script
  function start() {
    const links = getLinks();
    for (const link of links) {
      // attach hover and focus listeners to link
      link.addEventListener("mouseover", onLinkHover);
      link.addEventListener("mouseleave", onLinkUnhover);
      link.addEventListener("focus", onLinkFocus);
      link.addEventListener("touchend", onLinkTouch);
    }

    // attach mouse, key, and resize listeners to window
    window.addEventListener("mousedown", onClick);
    window.addEventListener("touchstart", onClick);
    window.addEventListener("keyup", onKeyUp);
    window.addEventListener("resize", onResize);
  }

  // when link is hovered
  function onLinkHover() {
    // function to open tooltip
    const delayOpenTooltip = function () {
      openTooltip(this);
    }.bind(this);

    // run open function after delay
    this.openTooltipTimer = window.setTimeout(delayOpenTooltip, delay);
  }

  // when mouse leaves link
  function onLinkUnhover() {
    // cancel opening tooltip
    window.clearTimeout(this.openTooltipTimer);

    // don't close on unhover if option specifies
    if (clickClose === "true") return;

    // function to close tooltip
    const delayCloseTooltip = function () {
      // if tooltip open and if mouse isn't over tooltip, close
      const tooltip = document.getElementById("tooltip");
      if (tooltip && !tooltip.matches(":hover")) closeTooltip();
    };

    // run close function after delay
    this.closeTooltipTimer = window.setTimeout(delayCloseTooltip, delay);
  }

  // when link is focused (tabbed to)
  function onLinkFocus(event) {
    openTooltip(this);
  }

  // when link is touched on touch screen
  function onLinkTouch(event) {
    // attempt to force hover state on first tap always, and trigger
    // regular link click (and navigation) on second tap
    if (event.target === document.activeElement) event.target.click();
    else {
      document.activeElement.blur();
      event.target.focus();
    }
    if (event.cancelable) event.preventDefault();
    event.stopPropagation();
    return false;
  }

  // when mouse is clicked anywhere in window
  function onClick(event) {
    closeTooltip();
  }

  // when key pressed
  function onKeyUp(event) {
    if (!event || !event.key) return;

    switch (event.key) {
      // trigger click of prev button
      case "ArrowLeft":
        const prevButton = document.getElementById("tooltip_prev_button");
        if (prevButton) prevButton.click();
        break;
      // trigger click of next button
      case "ArrowRight":
        const nextButton = document.getElementById("tooltip_next_button");
        if (nextButton) nextButton.click();
        break;
      // close on esc
      case "Escape":
        closeTooltip();
        break;
    }
  }

  // when window is resized or zoomed
  function onResize() {
    closeTooltip();
  }

  // get all links of types we wish to handle
  function getLinks() {
    const queries = [];
    // exclude buttons, anchor links, toc links, etc
    const exclude =
      ":not(.button):not(.icon_button):not(.anchor):not(.toc_link)";
    queries.push('a[href^="#ref-"]' + exclude); // citation links
    queries.push('a[href^="#fig:"]' + exclude); // figure links
    const query = queries.join(", ");
    return document.querySelectorAll(query);
  }

  // get links with same target, get index of link in set, get total
  // same links
  function getSameLinks(link) {
    const sameLinks = [];
    const links = getLinks();
    for (const otherLink of links) {
      if (otherLink.getAttribute("href") === link.getAttribute("href"))
        sameLinks.push(otherLink);
    }

    return {
      elements: sameLinks,
      index: sameLinks.indexOf(link),
      total: sameLinks.length,
    };
  }

  // open tooltip
  function openTooltip(link) {
    // delete tooltip if it exists, start fresh
    closeTooltip();

    // make tooltip element
    const tooltip = makeTooltip(link);

    // if source couldn't be found and tooltip not made, exit
    if (!tooltip) return;

    // make navbar elements
    const navBar = makeNavBar(link);
    if (navBar) tooltip.firstElementChild.appendChild(navBar);

    // attach tooltip to page
    document.body.appendChild(tooltip);

    // position tooltip
    const position = function () {
      positionTooltip(link);
    };
    position();

    // if tooltip contains images, position again after they've loaded
    const imgs = tooltip.querySelectorAll("img");
    for (const img of imgs) img.addEventListener("load", position);
  }

  // close (delete) tooltip
  function closeTooltip() {
    const tooltip = document.getElementById("tooltip");
    if (tooltip) tooltip.remove();
  }

  // make tooltip
  function makeTooltip(link) {
    // get target element that link points to
    const source = getSource(link);

    // if source can't be found, exit
    if (!source) return;

    // create new tooltip
    const tooltip = document.createElement("div");
    tooltip.id = "tooltip";
    const tooltipContent = document.createElement("div");
    tooltipContent.id = "tooltip_content";
    tooltip.appendChild(tooltipContent);

    // make copy of source node and put in tooltip
    const sourceCopy = makeCopy(source);
    tooltipContent.appendChild(sourceCopy);

    // attach mouse event listeners
    tooltip.addEventListener("click", onTooltipClick);
    tooltip.addEventListener("mousedown", onTooltipClick);
    tooltip.addEventListener("touchstart", onTooltipClick);
    tooltip.addEventListener("mouseleave", onTooltipUnhover);

    // (for interaction with lightbox plugin)
    // transfer click on tooltip copied img to original img
    const sourceImg = source.querySelector("img");
    const sourceCopyImg = sourceCopy.querySelector("img");
    if (sourceImg && sourceCopyImg) {
      const clickImg = function () {
        sourceImg.click();
        closeTooltip();
      };
      sourceCopyImg.addEventListener("click", clickImg);
    }

    return tooltip;
  }

  // make carbon copy of html dom element
  function makeCopy(source) {
    const sourceCopy = source.cloneNode(true);

    // delete elements marked with ignore (eg anchor and jump buttons)
    const deleteFromCopy = sourceCopy.querySelectorAll('[data-ignore="true"]');
    for (const element of deleteFromCopy) element.remove();

    // delete certain element attributes
    const attributes = [
      "id",
      "data-collapsed",
      "data-selected",
      "data-highlighted",
      "data-glow",
      "class",
    ];
    for (const attribute of attributes) {
      sourceCopy.removeAttribute(attribute);
      const elements = sourceCopy.querySelectorAll("[" + attribute + "]");
      for (const element of elements) element.removeAttribute(attribute);
    }

    return sourceCopy;
  }

  // when tooltip is clicked
  function onTooltipClick(event) {
    // when user clicks on tooltip, stop click from transferring
    // outside of tooltip (eg, click off to close tooltip, or eg click
    // off to unhighlight same refs)
    event.stopPropagation();
  }

  // when tooltip is unhovered
  function onTooltipUnhover(event) {
    if (clickClose === "true") return;

    // make sure new mouse/touch/focus no longer over tooltip or any
    // element within it
    const tooltip = document.getElementById("tooltip");
    if (!tooltip) return;
    if (this.contains(event.relatedTarget)) return;

    closeTooltip();
  }

  // make nav bar to go betwen prev/next instances of same reference
  function makeNavBar(link) {
    // find other links to the same source
    const sameLinks = getSameLinks(link);

    // don't show nav bar when singular reference
    if (sameLinks.total <= 1) return;

    // find prev/next links with same target
    const prevLink = getPrevLink(link, sameLinks);
    const nextLink = getNextLink(link, sameLinks);

    // create nav bar
    const navBar = document.createElement("div");
    navBar.id = "tooltip_nav_bar";
    const text = sameLinks.index + 1 + " of " + sameLinks.total;

    // create nav bar prev/next buttons
    const prevButton = document.createElement("button");
    const nextButton = document.createElement("button");
    prevButton.id = "tooltip_prev_button";
    nextButton.id = "tooltip_next_button";
    prevButton.title =
      "Jump to the previous occurence of this item in the document [←]";
    nextButton.title =
      "Jump to the next occurence of this item in the document [→]";
    prevButton.classList.add("icon_button");
    nextButton.classList.add("icon_button");
    prevButton.innerHTML = document.querySelector(".icon_caret_left").innerHTML;
    nextButton.innerHTML =
      document.querySelector(".icon_caret_right").innerHTML;
    navBar.appendChild(prevButton);
    navBar.appendChild(document.createTextNode(text));
    navBar.appendChild(nextButton);

    // attach click listeners to buttons
    prevButton.addEventListener("click", function () {
      onPrevNextClick(link, prevLink);
    });
    nextButton.addEventListener("click", function () {
      onPrevNextClick(link, nextLink);
    });

    return navBar;
  }

  // get previous link with same target
  function getPrevLink(link, sameLinks) {
    if (!sameLinks) sameLinks = getSameLinks(link);
    // wrap index to other side if < 1
    let index;
    if (sameLinks.index - 1 >= 0) index = sameLinks.index - 1;
    else index = sameLinks.total - 1;
    return sameLinks.elements[index];
  }

  // get next link with same target
  function getNextLink(link, sameLinks) {
    if (!sameLinks) sameLinks = getSameLinks(link);
    // wrap index to other side if > total
    let index;
    if (sameLinks.index + 1 <= sameLinks.total - 1) index = sameLinks.index + 1;
    else index = 0;
    return sameLinks.elements[index];
  }

  // get element that is target of link or url hash
  function getSource(link) {
    const hash = link ? link.hash : window.location.hash;
    const id = hash.slice(1);
    let target = document.querySelector('[id="' + id + '"]');
    if (!target) return;

    // if ref or figure, modify target to get expected element
    if (id.indexOf("ref-") === 0) target = target.querySelector(":nth-child(2)");
    else if (id.indexOf("fig:") === 0) target = target.querySelector("figure");

    return target;
  }

  // when prev/next arrow button is clicked
  function onPrevNextClick(link, prevNextLink) {
    if (link && prevNextLink)
      goToElement(prevNextLink, window.innerHeight * 0.5);
  }

  // scroll to and focus element
  function goToElement(element, offset) {
    // expand accordion section if collapsed
    expandElement(element);
    const y =
      getRectInView(element).top -
      getRectInView(document.documentElement).top -
      (offset || 0);
    // trigger any function listening for "onscroll" event
    window.dispatchEvent(new Event("scroll"));
    window.scrollTo(0, y);
    document.activeElement.blur();
    element.focus();
  }

  // determine position to place tooltip based on link position in
  // viewport and tooltip size
  function positionTooltip(link, left, top) {
    const tooltipElement = document.getElementById("tooltip");
    if (!tooltipElement) return;

    // get convenient vars for position/dimensions of
    // link/tooltip/page/view
    link = getRectInPage(link);
    const tooltip = getRectInPage(tooltipElement);
    const view = getRectInPage();

    // horizontal positioning
    if (left)
      // use explicit value
      left = left;
    else if (link.left + tooltip.width < view.right)
      // fit tooltip to right of link
      left = link.left;
    else if (link.right - tooltip.width > view.left)
      // fit tooltip to left of link
      left = link.right - tooltip.width;
    // center tooltip in view
    else left = (view.right - view.left) / 2 - tooltip.width / 2;

    // vertical positioning
    if (top)
      // use explicit value
      top = top;
    else if (link.top - tooltip.height > view.top)
      // fit tooltip above link
      top = link.top - tooltip.height;
    else if (link.bottom + tooltip.height < view.bottom)
      // fit tooltip below link
      top = link.bottom;
    else {
      // center tooltip in view
      top = view.top + view.height / 2 - tooltip.height / 2;
      // nudge off of link to left/right if possible
      if (link.right + tooltip.width < view.right) left = link.right;
      else if (link.left - tooltip.width > view.left)
        left = link.left - tooltip.width;
    }

    tooltipElement.style.left = left + "px";
    tooltipElement.style.top = top + "px";
  }

  // start script when document is finished loading
  window.addEventListener("load", start);
</script>

<!-- caret left icon -->

<template class="icon_caret_left">
  <!-- modified from: https://fontawesome.com/icons/caret-left -->
  <svg width="16" height="16" viewBox="0 0 192 512">
    <path
      fill="currentColor"
      d="M192 127.338v257.324c0 17.818-21.543 26.741-34.142 14.142L29.196 270.142c-7.81-7.81-7.81-20.474 0-28.284l128.662-128.662c12.599-12.6 34.142-3.676 34.142 14.142z"
    ></path>
  </svg>
</template>

<!-- caret right icon -->

<template class="icon_caret_right">
  <!-- modified from: https://fontawesome.com/icons/caret-right -->
  <svg width="16" height="16" viewBox="0 0 192 512">
    <path
      fill="currentColor"
      d="M0 384.662V127.338c0-17.818 21.543-26.741 34.142-14.142l128.662 128.662c7.81 7.81 7.81 20.474 0 28.284L34.142 398.804C21.543 411.404 0 402.48 0 384.662z"
    ></path>
  </svg>
</template>

<style>
  @media only screen {
    /* tooltip container */
    #tooltip {
      position: absolute;
      width: 50%;
      min-width: 240px;
      max-width: 75%;
      z-index: 1;
    }

    /* tooltip content */
    #tooltip_content {
      margin-bottom: 5px;
      padding: 20px;
      border-radius: 5px;
      border: solid 1px #bdbdbd;
      box-shadow: 0 0 20px rgba(0, 0, 0, 0.05);
      background: #ffffff;
      overflow-wrap: break-word;
    }

    /* tooltip copy of paragraphs and figures */
    #tooltip_content > p,
    #tooltip_content > figure {
      margin: 0;
      max-height: 320px;
      overflow-y: auto;
    }

    /* tooltip copy of <img> */
    #tooltip_content > figure > img,
    #tooltip_content > figure > svg {
      max-height: 260px;
    }

    /* navigation bar */
    #tooltip_nav_bar {
      margin-top: 10px;
      text-align: center;
    }

    /* navigation bar previous/next buton */
    #tooltip_nav_bar > .icon_button {
      position: relative;
      top: 3px;
    }

    /* navigation bar previous button */
    #tooltip_nav_bar > .icon_button:first-of-type {
      margin-right: 5px;
    }

    /* navigation bar next button */
    #tooltip_nav_bar > .icon_button:last-of-type {
      margin-left: 5px;
    }
  }

  /* always hide tooltip on print */
  @media only print {
    #tooltip {
      display: none;
    }
  }
</style>
<!--
  Analytics Plugin (third-party) 
  
  Copy and paste code from Google Analytics or similar service here.
-->
<!-- 
  Annotations Plugin

  Allows public annotation of the  manuscript. See https://web.hypothes.is/.
-->

<script type="module">
  // configuration
  window.hypothesisConfig = function () {
    return {
      branding: {
        accentColor: "#2196f3",
        appBackgroundColor: "#f8f8f8",
        ctaBackgroundColor: "#f8f8f8",
        ctaTextColor: "#000000",
        selectionFontFamily: "Open Sans, Helvetica, sans serif",
        annotationFontFamily: "Open Sans, Helvetica, sans serif",
      },
    };
  };

  // hypothesis client script
  const embed = "https://hypothes.is/embed.js";
  // hypothesis annotation count query url
  const query = "https://api.hypothes.is/api/search?limit=0&url=";

  // start script
  function start() {
    const button = makeButton();
    document.body.insertBefore(button, document.body.firstChild);
    insertCount(button);
  }

  // make button
  function makeButton() {
    // create button
    const button = document.createElement("button");
    button.id = "hypothesis_button";
    button.innerHTML = document.querySelector(".icon_hypothesis").innerHTML;
    button.title = "Hypothesis annotations";
    button.classList.add("icon_button");

    function onClick(event) {
      onButtonClick(event, button);
    }

    // attach click listeners
    button.addEventListener("click", onClick);

    return button;
  }

  // insert annotations count
  async function insertCount(button) {
    // get annotation count from Hypothesis based on url
    let count = "-";
    try {
      const canonical = document.querySelector('link[rel="canonical"]');
      const location = window.location;
      const url = encodeURIComponent((canonical || location).href);
      const response = await fetch(query + url);
      const json = await response.json();
      count = json.total || "-";
    } catch (error) {
      console.log(error);
    }

    // put count into button
    const counter = document.createElement("span");
    counter.id = "hypothesis_count";
    counter.innerHTML = count;
    button.title = "View " + count + " Hypothesis annotations";
    button.append(counter);
  }

  // when button is clicked
  function onButtonClick(event, button) {
    const script = document.createElement("script");
    script.src = embed;
    document.body.append(script);
    button.remove();
  }

  // start script when document is finished loading
  window.addEventListener("load", start);
</script>

<!-- hypothesis icon -->

<template class="icon_hypothesis">
  <!-- modified from: https://simpleicons.org/icons/hypothesis.svg / https://git.io/Jf1VB -->
  <svg width="16" height="16" viewBox="0 0 24 24" tabindex="-1">
    <path
      fill="currentColor"
      d="M3.43 0C2.5 0 1.72 .768 1.72 1.72V18.86C1.72 19.8 2.5 20.57 3.43 20.57H9.38L12 24L14.62 20.57H20.57C21.5 20.57 22.29 19.8 22.29 18.86V1.72C22.29 .77 21.5 0 20.57 0H3.43M5.14 3.43H7.72V9.43S8.58 7.72 10.28 7.72C12 7.72 13.74 8.57 13.74 11.24V17.14H11.16V12C11.16 10.61 10.28 10.07 9.43 10.29C8.57 10.5 7.72 11.41 7.72 13.29V17.14H5.14V3.43M18 13.72C18.95 13.72 19.72 14.5 19.72 15.42A1.71 1.71 0 0 1 18 17.13A1.71 1.71 0 0 1 16.29 15.42C16.29 14.5 17.05 13.71 18 13.71Z"
      tabindex="-1"
    ></path>
  </svg>
</template>

<style>
  /* hypothesis activation button */
  #hypothesis_button {
    box-sizing: border-box;
    position: fixed;
    top: 0;
    right: 0;
    width: 60px;
    height: 60px;
    background: #ffffff;
    border-radius: 0;
    border-left: solid 1px #bdbdbd;
    border-bottom: solid 1px #bdbdbd;
    box-shadow: 0 0 20px rgba(0, 0, 0, 0.05);
    z-index: 2;
  }

  /* hypothesis button svg */
  #hypothesis_button > svg {
    position: relative;
    top: -4px;
  }

  /* hypothesis annotation count */
  #hypothesis_count {
    position: absolute;
    left: 0;
    right: 0;
    bottom: 5px;
  }

  /* side panel */
  .annotator-frame {
    width: 280px !important;
  }

  /* match highlight color to rest of theme */
  .annotator-highlights-always-on .annotator-hl {
    background-color: #ffeb3b !important;
  }

  /* match focused color to rest of theme */
  .annotator-hl.annotator-hl-focused {
    background-color: #ff8a65 !important;
  }

  /* match bucket bar color to rest of theme */
  .annotator-bucket-bar {
    background: #f5f5f5 !important;
  }

  /* always hide button, toolbar, and tooltip on print */
  @media only print {
    #hypothesis_button {
      display: none;
    }

    .annotator-frame {
      display: none !important;
    }

    hypothesis-adder {
      display: none !important;
    }
  }
</style>
<!-- 
  Mathjax Plugin (third-party) 

  Allows the proper rendering of math/equations written in LaTeX.
  See https://www.mathjax.org/.
-->

<script type="text/x-mathjax-config">
  // configuration
  MathJax.Hub.Config({
    "CommonHTML": { linebreaks: { automatic: true } },
    "HTML-CSS": { linebreaks: { automatic: true } },
    "SVG": { linebreaks: { automatic: true } },
    "fast-preview": { disabled: true }
  });
</script>

<script
  src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.5/MathJax.js?config=TeX-MML-AM_CHTML"
  integrity="sha512-0doc9hKxR3PYwso42RD1p5ySZpzzuDiOwMrdCEh2WdJZCjcmFKc/wEnL+z8fBQrnHoiNWbo+3fiGkOYXBdQp4A=="
  crossorigin="anonymous"
></script>

<style>
  /* mathjax containers */
  .math.display > span:not(.MathJax_Preview) {
    /* turn inline element (no dimensions) into block (allows fixed width and thus scrolling) */
    display: flex !important;
    overflow-x: auto !important;
    overflow-y: hidden !important;
    justify-content: center;
    align-items: center;
    margin: 0 !important;
  }

  /* right click menu */
  .MathJax_Menu {
    border-radius: 5px !important;
    border: solid 1px #bdbdbd !important;
    box-shadow: none !important;
  }

  /* equation auto-number */
  span[id^="eq:"] > span.math.display + span {
    font-weight: 600;
  }

  /* equation */
  span[id^="eq:"] > span.math.display > span {
    /* nudge to make room for equation auto-number and anchor */
    margin-right: 60px !important;
  }
</style>
<!-- 
  Scite Plugin (third party)

  Adds a Scite badge next to each citation in the references section.
  See https://scite.ai/.
-->

<script
  async
  type="application/javascript"
  src="https://cdn.scite.ai/badge/scite-badge-latest.min.js"
></script>

<script type="module">
  // start script
  function start() {
    // if printing, exit and don't run badges
    if (window.matchMedia("print").matches) return;

    // get citation elements
    const query = ".references div[id*='ref-']";
    const elements = document.querySelectorAll(query);
    for (const element of elements) addBadge(element);
  }

  // add badge in citation element
  function addBadge(element) {
    // get citation text
    const text = element.innerText;
    // find doi string, eg "DOI: xxxxx/xxxxx"
    const match = text.match(/DOI:\s+(\S*)/) || {};
    const doi = match[1];
    if (!doi) return;
    // create cite badge and put after citation
    const badge = document.createElement("div");
    badge.innerHTML = "Loading Scite badge...";
    badge.classList.add("scite-badge");
    badge.setAttribute("data-doi", doi);
    badge.setAttribute("data-layout", "horizontal");
    badge.setAttribute("data-show-zero", "true");
    element.append(badge);
    // modify preceding citation <p> element
    const p = element.querySelector("p");
    if (p) p.style.marginBottom = "0";
  }

  // start script when document is finished loading
  window.addEventListener("load", start);
</script>

<style>
  .scite-badge {
    text-indent: 0;
    margin-top: 10px;
  }

  @media print {
    .scite-badge {
      display: none;
    }
  }
</style>
</body>
</html>
